
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>
<A NAME="16_03_09at13_22_41">&nbsp;</A>
////////////////////////////////////////////////////////////////////
RUN NUMBER 1 (16/03/09 at 13:22:41)
////////////////////////////////////////////////////////////////////

<A NAME="16_03_09at13_22_41_run_information">&nbsp;</A>

Project information:
--------------------
	NbSamples      =  4
	DataType       =  STANDARD
	GenotypicData  =  1
	GameticPhase   =  0
	RecessiveData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Infer haplotype definition from distance matrix
        Alllowed level of missing data = 0.2

 Active Tasks:
 -------------

    Analysis of Molecular Variance:
    -------------------------------
        No. of Permutations = 1000

    Population pairwise Fst values:
    -------------------------------
        Compute pairwise differences
        No. of permutations for significance = 1000
        No. of permutations for Mantel test  = 500

       Distance matrix:
            Compute distance matrix
            Molecular distance : Pairwise differences
            Gamma a value      = 0


    Correlation between distance matrices.
    --------------------------------------------------------

        No. of Permutations for significance = 500
<A NAME="16_03_09at13_22_41_gen_struct">&nbsp;</A>
===============================================================================
== GENETIC STRUCTURE ANALYSIS
===============================================================================


Number of usable loci for distance computation : 13
Allowed level of missing data                  : 0.2

List of usable loci :
---------------------
   1     2     3     4     5     6     7     8     9    10    11    12    13  

List of loci with too much missing data :
-----------------------------------------

NONE

<A NAME="16_03_09at13_22_41_amova">&nbsp;</A>

=================================================================
  AMOVA ANALYSIS
=================================================================


---------------------------
Genetic structure to test :
---------------------------

No. of Groups = 1


	[[Structure]]

		StructureName = "Aclarkii sizeclasses Structure"
		NbGroups = 1
		Group={
		"1"
		"2"
		"3"
		"4"
		}

---------------------------


Distance method: Pairwise differences

--------------------------
AMOVA design and results :
--------------------------

Reference: <i>Weir, B.S. and Cockerham, C.C. 1984.</i>
           <i>Excoffier, L., Smouse, P., and Quattro, J. 1992.</i>
           <i>Weir, B. S., 1996.</i>
----------------------------------------------------------------------
 Source of                  Sum of      Variance         Percentage
 variation      d.f.        squares     components       of variation
----------------------------------------------------------------------
 Among
 populations      3         10.702       -0.00192 Va            -0.05

 Among
 individuals
 within
 populations    374       1467.094        0.02352 Vb             0.60

 Within
 individuals    378       1465.000        3.87566 Vc            99.45
----------------------------------------------------------------------
 Total          755       2942.796        3.89727
----------------------------------------------------------------------
 Fixation Indices
      FIS :      0.00603
      FST :     -0.00049
      FIT :      0.00554
----------------------------------------------------------------------

Significance tests (1023 permutations)
------------------

Vb and FIS : P(rand. value > obs. value)  =  0.25122
             P(rand. value = obs. value)  =  0.00978
                                  P-value =  0.26100+-0.01301

Va and FST : P(rand. value > obs. value)  =  1.00000
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  1.00000+-0.00000

Vc and FIT : P(rand. value < obs. value)  =  0.23851
             P(rand. value = obs. value)  =  0.01369
                                  P-value =  0.25220+-0.01358

<A NAME="16_03_09at13_22_41_amova_AMOVA_FIS">&nbsp;</A>

-------------------------------------------------------
Population specific FIS indices (1023 permutations)
---------------------------------------------------
  Pop#      Name       FIS    P(Rand FIS>=Obs FIS)
---------------------------------------------------
     1         1  -0.00100            0.554252
     2         2   0.02086            0.122190
     3         3   0.00174            0.470186
     4         4   0.00121            0.530792
---------------------------------------------------

<A NAME="16_03_09at13_22_41_pairw_diff">&nbsp;</A>

======================================================================================
== Comparisons of pairs of population samples
======================================================================================

List of labels for population samples used below:
-------------------------------------------------

Label  	Population name
-----  	---------------
  1:	1
  2:	2
  3:	3
  4:	4

------------------------
Population pairwise FSTs
------------------------


Distance method: Pairwise differences
                     1         2         3         4
           1   0.00000
           2  -0.00116   0.00000
           3   0.00019  -0.00197   0.00000
           4  -0.00090   0.00173   0.00081   0.00000


------------
FST P values
------------

Number of permutations : 1023

                       1                 2                 3                 4
           1           *
           2   0.87012+-0.0115           *
           3   0.39551+-0.0135   0.99121+-0.0024           *
           4   0.70605+-0.0136   0.15430+-0.0139   0.25684+-0.0166           *


------------
Matrix of significant Fst P values
Significance Level=0.0500
------------

Number of permutations : 1023

                     1         2         3         4
           1                   -         -         -
           2         -                   -         -
           3         -         -                   -
           4         -         -         -          


---------------------------------------
Population average pairwise differences
----------------------------------------

Above diagonal    : Average number of pairwise differences between populations (PiXY)
Diagonal elements : Average number of pairwise differences within population (PiX)
Below diagonal    : Corrected average pairwise difference (PiXY-(PiX+PiY)/2)

Distance method: Pairwise differences
                       1          2          3          4
            1    7.74878    7.75686    7.80250    7.77931
            2   -0.00898    7.78290    7.80269    7.81693
            3    0.00148   -0.01539    7.85326    7.84513
            4   -0.00716    0.01340    0.00642    7.82416


-----------
PXY P value
-----------

                         1           2           3

             2     0.73021
             3     0.26979     0.99316
             4     0.39980     0.28935     0.52395


---------------------
Corrected PXY P value
---------------------

                           1            2            3

              2      0.87097
              3      0.39785      0.99218
              4      0.71457      0.15738      0.25318


********************
Locus by locus AMOVA:
********************

Distance method for locus-by-locus analysis: Pairwise differences

List of loci with only one allele:
----------------------------------


<A NAME="16_03_09at13_22_41Loc_by_Loc_AMOVA">&nbsp;</A>

----------------------------------------
AMOVA Results for polymorphic loci only:
----------------------------------------

       Among populations:                                     Among Individuals:                                    Within Individuals:                                 Fixation indices:
  Locus            SSD   d.f.             Va    % variation            SSD   d.f.             Vb    % variation            SSD   d.f.             Vc    % variation       FIS   P-value       FST   P-value       FIT   P-value
      1        0.55427      3       -0.00071       -0.23627      115.76159    368        0.01212        4.01795      108.00000    372        0.29032       96.21832   0.04008   0.12805  -0.00236   1.00000   0.03782   0.14076
      2        0.41102      3       -0.00058       -0.26649       90.05831    371        0.02604       12.04788       71.50000    375        0.19067       88.21861   0.12016   0.00000  -0.00266   1.00000   0.11781   0.00098
      3        0.20668      3        0.00011        0.21334       18.32390    372       -0.00064       -1.27408       19.00000    376        0.05053      101.06074  -0.01277   0.64027   0.00213   0.49267  -0.01061   0.63148
      4        1.40150      3        0.00018        0.04164      161.68603    373       -0.00475       -1.08304      167.00000    377        0.44297      101.04140  -0.01083   0.72630   0.00042   0.94526  -0.01041   0.73314
      5        0.39553      3       -0.00136       -0.35902      141.29914    371        0.00176        0.46684      141.50000    375        0.37733       99.89219   0.00465   0.46041  -0.00359   1.00000   0.00108   0.47898
      6        1.89460      3        0.00100        0.22573      165.70166    370        0.00400        0.89950      164.50000    374        0.43984       98.87477   0.00902   0.30303   0.00226   0.55718   0.01125   0.28935
      7        1.33656      3        0.00025        0.05982      145.26453    362       -0.01179       -2.85253      155.50000    366        0.42486      102.79270  -0.02854   0.91105   0.00060   0.89345  -0.02793   0.92473
      8        0.93781      3       -0.00004       -0.01224      118.88267    372        0.01019        3.29317      112.50000    376        0.29920       96.71908   0.03293   0.19941  -0.00012   0.82502   0.03281   0.20723
      9        0.34480      3       -0.00132       -0.38118      134.56393    374        0.01125        3.23925      127.50000    378        0.33730       97.14193   0.03227   0.16716  -0.00381   1.00000   0.02858   0.18573
     10        0.56011      3       -0.00091       -0.26075      123.81767    356       -0.00318       -0.90899      127.50000    360        0.35417      101.16974  -0.00907   0.62757  -0.00261   1.00000  -0.01170   0.65298
     11        0.26715      3       -0.00011       -0.10227       40.70507    374        0.00415        3.97222       38.00000    378        0.10053       96.13005   0.03968   0.22483  -0.00102   0.86315   0.03870   0.24340
     12        1.62730      3        0.00136        0.48910      108.48543    373        0.01346        4.82870       99.50000    377        0.26393       94.68219   0.04852   0.14467   0.00489   0.20821   0.05318   0.14663
     13        1.25360      3        0.00023        0.06345      139.27440    371        0.01037        2.83848      133.00000    375        0.35467       97.09807   0.02840   0.16618   0.00063   0.81329   0.02902   0.16813

====================================================
Global AMOVA results as a weighted average over loci
====================================================

----------------------------------------------------
AMOVA design and results (average over 13 loci):
----------------------------------------------------

------------------------------------------------------------------------
 Source of               Sum of           Variance            Percentage
 variation               squares          components          variation
------------------------------------------------------------------------
 Among
 populations              11.191            -0.00189            -0.04729

 Among
 individuals
 within
 populations            1503.824             0.07299             1.82585

 Within
 individuals            1465.000             3.92632            98.22144
------------------------------------------------------------------------
 Total                  2980.015             3.99742
------------------------------------------------------------------------


Average F-Statistics over all loci
 Fixation Indices
      FIS :      0.01825
      FST :     -0.00047
      FIT :      0.01779
<A NAME="16_03_09at13_22_41_comp_sum_bootstrap">&nbsp;</A>

FIT bootstrap percentile values (over 20000 bootstraps)
-------------------------------------------------------

 0.05 %    : -0.00711
 0.1 %     : -0.00611
 0.25 %    : -0.00504
 0.5 %     : -0.00320
 1 %       : -0.00138
 2.5 %     : 0.00149
 5 %       : 0.00384
 10 %      : 0.00674
 50 %      : 0.01775
 90 %      : 0.03040
 95 %      : 0.03451
 97.5 %    : 0.03808
 99 %      : 0.04235
 99.5 %    : 0.04543
 99.75 %   : 0.04837
 99.9 %    : 0.05272
 99.95 %   : 0.05654



FST bootstrap percentile values (over 20000 bootstraps)
-------------------------------------------------------

 0.05 %    : -0.00279
 0.1 %     : -0.00265
 0.25 %    : -0.00246
 0.5 %     : -0.00234
 1 %       : -0.00218
 2.5 %     : -0.00193
 5 %       : -0.00168
 10 %      : -0.00141
 50 %      : -0.00047
 90 %      : 0.00045
 95 %      : 0.00070
 97.5 %    : 0.00093
 99 %      : 0.00122
 99.5 %    : 0.00139
 99.75 %   : 0.00158
 99.9 %    : 0.00193
 99.95 %   : 0.00201



FIS bootstrap percentile values (over 20000 bootstraps)
-------------------------------------------------------

 0.05 %    : -0.00673
 0.1 %     : -0.00579
 0.25 %    : -0.00468
 0.5 %     : -0.00295
 1 %       : -0.00109
 2.5 %     : 0.00183
 5 %       : 0.00426
 10 %      : 0.00720
 50 %      : 0.01822
 90 %      : 0.03093
 95 %      : 0.03501
 97.5 %    : 0.03856
 99 %      : 0.04304
 99.5 %    : 0.04606
 99.75 %   : 0.04897
 99.9 %    : 0.05264
 99.95 %   : 0.05826



------------------
Significance tests (1023 permutations)
------------------

Vb and FIS : P(rand. value > obs. value)  =  0.01466
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.01466

Va and FST : P(rand. value > obs. value)  =  1.00000
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  1.00000

Vc and FIT : P(rand. value < obs. value)  =  0.01369
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.01369

<A NAME="16_03_09at13_22_41_comp_sum_LBL_AMOVA_FIS">&nbsp;</A>


-----------------------------------------------------------------------
Population specific FIS indices per polymorphic locus (absolute values)
-------------------------------------------------------------
  Locus   Average FIS         1         2         3         4
-------------------------------------------------------------
      1       0.04008   0.00906   0.14605   0.04724  -0.13515
      2       0.12016   0.09970   0.00366   0.27571   0.09165
      3      -0.01277  -0.04308  -0.01873   0.06082  -0.05765
      4      -0.01083  -0.00167   0.00857  -0.02350  -0.04276
      5       0.00465  -0.01527   0.00711   0.02112   0.01124
      6       0.00902   0.03203   0.01346  -0.04073   0.04865
      7      -0.02854  -0.04281  -0.04859  -0.03873   0.05963
      8       0.03293  -0.02125   0.08950   0.01653   0.07280
      9       0.03227   0.03516   0.06584   0.05118  -0.07282
     10      -0.00907  -0.00000  -0.03342  -0.10032   0.18531
     11       0.03968   0.12819  -0.03166  -0.00478   0.12757
     12       0.04852   0.03884   0.13567   0.03883  -0.08736
     13       0.02840   0.02523   0.01962   0.05594  -0.00025
-------------------------------------------------------------

---------------------------------------------------------------------------------------------------
Population specific FIS indices per polymorphic locus (relative values (%) compared to average FIS)
-------------------------------------------------------------
  Locus   Average FIS         1         2         3         4
-------------------------------------------------------------
      1       0.04008   -77.391   264.362    17.841  -437.167
      2       0.12016   -17.023   -96.952   129.457   -23.722
      3      -0.01277   237.381    46.710  -576.369   351.516
      4      -0.01083   -84.610  -179.125   116.862   294.673
      5       0.00465  -428.355    52.939   353.974   141.662
      6       0.00902   255.327    49.276  -551.814   439.621
      7      -0.02854    49.988    70.234    35.701  -308.928
      8       0.03293  -164.550   171.807   -49.790   121.088
      9       0.03227     8.946   104.042    58.615  -325.667
     10      -0.00907  -100.000   268.650  1006.563 -2143.980
     11       0.03968   223.059  -179.791  -112.058   221.489
     12       0.04852   -19.960   179.584   -19.973  -280.032
     13       0.02840   -11.155   -30.929    96.937  -100.863
-------------------------------------------------------------


////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 1 (16/03/09 at 13:23:13))
Total computing time for this run : 0h 0m 31s 937 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

