
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>
<A NAME="08_01_09at11_38_25">&nbsp;</A>
////////////////////////////////////////////////////////////////////
RUN NUMBER 1 (08/01/09 at 11:38:25)
////////////////////////////////////////////////////////////////////

<A NAME="08_01_09at11_38_25_run_information">&nbsp;</A>

Project information:
--------------------
	NbSamples      =  20
	DataType       =  STANDARD
	GenotypicData  =  1
	GameticPhase   =  0
	RecessiveData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Infer haplotype definition from distance matrix
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise differences
        GammaA Value       = 0.632
        Theta estimators   : 

    Linkage disequilibrium test:
    ----------------------------
        No. of permutations          = 10000
        No. of Initial conditions    = 2
        Print Histogram and table : No

    Hardy-Weinberg equilibrium test:
    --------------------------------
        No. of steps in Markov chain      = 1000000
        No. of Dememorisation Steps       = 100000
        Required precision on Probability = 0
        Test association at the Locus level

    Analysis of Molecular Variance:
    -------------------------------
        No. of Permutations = 1000

    Population pairwise Fst values:
    -------------------------------
        Compute pairwise differences
        No. of permutations for significance = 1000
        No. of permutations for Mantel test  = 500

       Distance matrix:
            Compute distance matrix
            Molecular distance : Pairwise differences
            Gamma a value      = 0

    Exact test of population differentiation:
    -----------------------------------------
        No. of steps in Markov chain      = 100000
        No. of Dememorisation Steps       = 10000
        Required precision on Probability = 0
        Print Histogram and table : Yes
        Significance level        = 0.05


    Correlation between distance matrices.
    --------------------------------------------------------

        No. of Permutations for significance = 500
<A NAME="08_01_09at11_38_25_gen_struct">&nbsp;</A>
===============================================================================
== GENETIC STRUCTURE ANALYSIS
===============================================================================


Number of usable loci for distance computation : 1
Allowed level of missing data                  : 0.05

List of usable loci :
---------------------
   1  

List of loci with too much missing data :
-----------------------------------------

   2     3     4  

<A NAME="08_01_09at11_38_25_amova">&nbsp;</A>

=================================================================
  AMOVA ANALYSIS
=================================================================


---------------------------
Genetic structure to test :
---------------------------

No. of Groups = 2


	[[Structure]]

		StructureName = "New Edited Structure"
		NbGroups = 2
		Group={
		"2_Handumon"
		"7_Santander"
		"8_Boljoon"
		"9_Argao"
		"10_Carcar"
		"11_Minglanilla"
		"13_Danao"
		"14_Sogod"
		"15_Tabogon"
		"16_Daanbantayan"
		"17_Malapascua"
		}

		Group={
		"18_Pintuyan"
		"19_PadreBurgos"
		"20_Maasin"
		"22_InopacanIsland"
		"22_InopacanMainland"
		"23_Baybay"
		"24_Albuera"
		"25_Ormoc"
		"27_Palompon"
		}

---------------------------


Distance method: Pairwise differences

--------------------------
AMOVA design and results :
--------------------------

Reference: <i>Weir, B.S. and Cockerham, C.C. 1984.</i>
           <i>Excoffier, L., Smouse, P., and Quattro, J. 1992.</i>
           <i>Weir, B. S., 1996.</i>
----------------------------------------------------------------------
 Source of                  Sum of      Variance         Percentage
 variation      d.f.        squares     components       of variation
----------------------------------------------------------------------
 Among
 groups           1          0.311        0.00002 Va             0.01

 Among
 populations
 within
 groups          18          5.459        0.00080 Vb             0.29

 Within
 populations    710        194.745        0.27429 Vc            99.70
----------------------------------------------------------------------
 Total          729        200.515        0.27510
----------------------------------------------------------------------
 Fixation Indices
      FST :      0.00296
      FSC :      0.00289
      FCT :      0.00007
----------------------------------------------------------------------

Significance tests (1023 permutations)
------------------

Vc and FST : P(rand. value < obs. value)  =  0.27077
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.27077+-0.01208

Vb and FSC : P(rand. value > obs. value)  =  0.27175
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.27175+-0.01562

Va and FCT : P(rand. value > obs. value)  =  0.39687
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.39687+-0.01298

<A NAME="08_01_09at11_38_25_pairw_diff">&nbsp;</A>

======================================================================================
== Comparisons of pairs of population samples
======================================================================================

List of labels for population samples used below:
-------------------------------------------------

Label  	Population name
-----  	---------------
  1:	2_Handumon
  2:	7_Santander
  3:	8_Boljoon
  4:	9_Argao
  5:	10_Carcar
  6:	11_Minglanilla
  7:	13_Danao
  8:	14_Sogod
  9:	15_Tabogon
  10:	16_Daanbantayan
  11:	17_Malapascua
  12:	18_Pintuyan
  13:	19_PadreBurgos
  14:	20_Maasin
  15:	22_InopacanIsland
  16:	22_InopacanMainland
  17:	23_Baybay
  18:	24_Albuera
  19:	25_Ormoc
  20:	27_Palompon

------------------------
Population pairwise FSTs
------------------------


Distance method: Pairwise differences
                     1         2         3         4         5         6         7         8         9        10        11        12        13        14        15        16        17        18        19        20
           1   0.00000
           2  -0.02292   0.00000
           3  -0.03798  -0.00890   0.00000
           4  -0.04148  -0.01369  -0.02138   0.00000
           5  -0.03829   0.02006  -0.00923  -0.00068   0.00000
           6  -0.04963   0.00896  -0.01036  -0.01029  -0.02015   0.00000
           7  -0.04138  -0.00154  -0.02673  -0.01988  -0.01475  -0.01475   0.00000
           8   0.01694   0.06285   0.00643   0.02163   0.05509   0.05115   0.01641   0.00000
           9  -0.04482  -0.00625  -0.02152  -0.02345  -0.01008  -0.01594  -0.02453   0.01383   0.00000
          10  -0.03600   0.02257  -0.02131  -0.01223   0.00091  -0.00692  -0.01670  -0.00445  -0.02012   0.00000
          11  -0.04606  -0.00274  -0.01168  -0.01419  -0.01971  -0.02224  -0.01745   0.05332  -0.01790  -0.00081   0.00000
          12  -0.04825  -0.01266  -0.03557  -0.03266   0.00076  -0.01329  -0.02472   0.00258  -0.03493  -0.02257  -0.01603   0.00000
          13  -0.04374  -0.01654  -0.02106  -0.02564   0.00235  -0.01065  -0.01572   0.03637  -0.02564  -0.00242  -0.01819  -0.02477   0.00000
          14  -0.03743  -0.01542  -0.01746  -0.02478   0.00188  -0.00471  -0.02031   0.01765  -0.02356  -0.01032  -0.01383  -0.03663  -0.03121   0.00000
          15  -0.02383  -0.00293  -0.01784  -0.01821   0.01619   0.01071  -0.01682  -0.00983  -0.01673  -0.01771   0.00546  -0.03768  -0.01911  -0.02081   0.00000
          16  -0.03044   0.03515  -0.01869  -0.00179  -0.01295  -0.00773  -0.01894   0.00072  -0.01318  -0.02477  -0.00087  -0.01200   0.00956   0.00233  -0.00503   0.00000
          17  -0.00648   0.08053   0.01755   0.04158  -0.01025   0.00193   0.01600   0.07738   0.02346   0.02116   0.01473   0.04892   0.05752   0.05064   0.05468  -0.00972   0.00000
          18   0.00746   0.09851   0.01313   0.04745   0.00192   0.01652   0.01867   0.05176   0.02511   0.01453   0.03035   0.05592   0.07287   0.05402   0.04423  -0.02208  -0.02121   0.00000
          19  -0.03917   0.00890  -0.01274  -0.00745  -0.02477  -0.01845  -0.01861   0.04873  -0.01461  -0.00081  -0.02212  -0.00788  -0.00722  -0.00761   0.00753  -0.01039   0.00109   0.01208   0.00000
          20   0.01818   0.10757   0.02065   0.05720   0.00571   0.02411   0.02627   0.06402   0.03447   0.02560   0.03718   0.06882   0.08406   0.06419   0.05476  -0.01540  -0.02078  -0.02554   0.01664   0.00000


------------
FST P values
------------

Number of permutations : 1023

                       1                 2                 3                 4                 5                 6                 7                 8                 9                10                11                12                13                14                15                16                17                18                19                20
           1           *
           2   0.58789+-0.0159           *
           3   0.84375+-0.0106   0.52051+-0.0140           *
           4   0.88574+-0.0105   0.68359+-0.0172   0.90527+-0.0097           *
           5   0.74023+-0.0129   0.15625+-0.0106   0.48145+-0.0125   0.35547+-0.0119           *
           6   0.99902+-0.0002   0.24805+-0.0108   0.54199+-0.0194   0.53711+-0.0135   0.81348+-0.0110           *
           7   0.86719+-0.0113   0.40039+-0.0136   0.99902+-0.0002   0.86621+-0.0128   0.61816+-0.0131   0.63086+-0.0160           *
           8   0.32910+-0.0136   0.04395+-0.0059   0.25293+-0.0151   0.21191+-0.0106   0.04297+-0.0070   0.08105+-0.0075   0.21777+-0.0136           *
           9   0.99902+-0.0002   0.47852+-0.0128   0.98242+-0.0042   0.96875+-0.0056   0.47461+-0.0134   0.63574+-0.0161   0.98926+-0.0041   0.20801+-0.0112           *
          10   0.72461+-0.0128   0.14355+-0.0099   0.85059+-0.0126   0.60352+-0.0156   0.32129+-0.0180   0.46680+-0.0200   0.63867+-0.0116   0.48047+-0.0139   0.74707+-0.0114           *
          11   0.99902+-0.0002   0.37109+-0.0164   0.60254+-0.0134   0.63281+-0.0128   0.72461+-0.0133   0.87402+-0.0116   0.78516+-0.0111   0.03320+-0.0053   0.78223+-0.0123   0.36133+-0.0148           *
          12   0.81836+-0.0125   0.58105+-0.0110   0.97363+-0.0049   0.91992+-0.0084   0.29590+-0.0155   0.53906+-0.0135   0.78125+-0.0115   0.38867+-0.0134   0.96289+-0.0052   0.68750+-0.0139   0.59863+-0.0132           *
          13   0.83105+-0.0111   0.71582+-0.0098   0.84668+-0.0103   0.92480+-0.0086   0.28320+-0.0156   0.51367+-0.0162   0.66602+-0.0146   0.11426+-0.0084   0.93066+-0.0089   0.37598+-0.0149   0.68359+-0.0171   0.77246+-0.0125           *
          14   0.85352+-0.0102   0.68848+-0.0143   0.71680+-0.0107   0.94922+-0.0065   0.34473+-0.0140   0.46680+-0.0152   0.79980+-0.0122   0.22754+-0.0137   0.90918+-0.0079   0.50293+-0.0188   0.62207+-0.0145   0.96289+-0.0059   0.96191+-0.0050           *
          15   0.67090+-0.0128   0.44824+-0.0195   0.81934+-0.0111   0.81641+-0.0117   0.19824+-0.0129   0.24121+-0.0121   0.73242+-0.0119   0.61230+-0.0151   0.75586+-0.0139   0.73535+-0.0126   0.27832+-0.0148   0.98535+-0.0034   0.77637+-0.0118   0.84863+-0.0113           *
          16   0.73047+-0.0138   0.13965+-0.0090   0.71973+-0.0116   0.44238+-0.0129   0.58789+-0.0144   0.44434+-0.0109   0.69922+-0.0166   0.46484+-0.0128   0.56152+-0.0127   0.80566+-0.0125   0.38770+-0.0170   0.50293+-0.0164   0.27930+-0.0143   0.38867+-0.0188   0.49707+-0.0149           *
          17   0.45801+-0.0150   0.02246+-0.0053   0.15430+-0.0111   0.08398+-0.0085   0.48047+-0.0171   0.34668+-0.0158   0.18848+-0.0129   0.02930+-0.0043   0.10059+-0.0084   0.15332+-0.0116   0.18652+-0.0141   0.07324+-0.0078   0.08301+-0.0076   0.07422+-0.0074   0.05957+-0.0086   0.52051+-0.0141           *
          18   0.26953+-0.0151   0.01172+-0.0033   0.15234+-0.0114   0.07422+-0.0094   0.28125+-0.0137   0.22852+-0.0123   0.14648+-0.0110   0.04883+-0.0071   0.08398+-0.0091   0.25293+-0.0136   0.08496+-0.0082   0.04004+-0.0068   0.03516+-0.0063   0.06348+-0.0081   0.05957+-0.0071   0.69922+-0.0112   0.72461+-0.0133           *
          19   0.82227+-0.0101   0.25195+-0.0127   0.60547+-0.0181   0.50977+-0.0157   0.95996+-0.0057   0.77734+-0.0131   0.83496+-0.0083   0.05762+-0.0069   0.74023+-0.0137   0.34863+-0.0129   0.93848+-0.0070   0.44922+-0.0115   0.45117+-0.0134   0.47266+-0.0128   0.25293+-0.0156   0.55371+-0.0158   0.30859+-0.0152   0.18945+-0.0118           *
          20   0.22949+-0.0115   0.00977+-0.0038   0.09277+-0.0087   0.07129+-0.0086   0.24121+-0.0093   0.12988+-0.0114   0.10352+-0.0078   0.03223+-0.0042   0.05078+-0.0062   0.12891+-0.0134   0.07031+-0.0094   0.02246+-0.0042   0.02051+-0.0055   0.05273+-0.0077   0.04297+-0.0058   0.56836+-0.0197   0.81055+-0.0096   0.81836+-0.0092   0.15918+-0.0111           *


------------
Matrix of significant Fst P values
Significance Level=0.0500
------------

Number of permutations : 1023

                     1         2         3         4         5         6         7         8         9        10        11        12        13        14        15        16        17        18        19        20
           1                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           2         -                   -         -         -         -         -         +         -         -         -         -         -         -         -         -         +         +         -         +
           3         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           4         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           5         -         -         -         -                   -         -         +         -         -         -         -         -         -         -         -         -         -         -         -
           6         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -
           7         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -
           8         -         +         -         -         +         -         -                   -         -         +         -         -         -         -         -         +         +         -         +
           9         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -
          10         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -
          11         -         -         -         -         -         -         -         +         -         -                   -         -         -         -         -         -         -         -         -
          12         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         +         -         +
          13         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         +         -         +
          14         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         -
          15         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         +
          16         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -
          17         -         +         -         -         -         -         -         +         -         -         -         -         -         -         -         -                   -         -         -
          18         -         +         -         -         -         -         -         +         -         -         -         +         +         -         -         -         -                   -         -
          19         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -
          20         -         +         -         -         -         -         -         +         -         -         -         +         +         -         +         -         -         -         -          


---------------------------------------
Population average pairwise differences
----------------------------------------

Above diagonal    : Average number of pairwise differences between populations (PiXY)
Diagonal elements : Average number of pairwise differences within population (PiX)
Below diagonal    : Corrected average pairwise difference (PiXY-(PiX+PiY)/2)

Distance method: Pairwise differences
                       1          2          3          4          5          6          7          8          9         10         11         12         13         14         15         16         17         18         19         20
            1    0.58242    0.53214    0.59464    0.55714    0.53383    0.52820    0.54643    0.60714    0.57679    0.54135    0.54464    0.50476    0.51429    0.59874    0.62970    0.56633    0.50397    0.49248    0.54821    0.49206
            2   -0.01355    0.50897    0.57437    0.53625    0.52895    0.52303    0.53187    0.59702    0.56188    0.53684    0.53187    0.48750    0.49312    0.57426    0.60395    0.56607    0.51528    0.50658    0.53812    0.50625
            3   -0.02157   -0.00511    0.65000    0.60125    0.58355    0.58289    0.58750    0.63393    0.62250    0.58289    0.59688    0.54500    0.56000    0.64265    0.66447    0.60625    0.55417    0.53421    0.59625    0.53333
            4   -0.02317   -0.00734   -0.01285    0.57821    0.55263    0.54737    0.55625    0.60714    0.58625    0.55263    0.56000    0.51167    0.52250    0.60294    0.62895    0.58036    0.53056    0.51579    0.56375    0.51597
            5   -0.02124    0.01059   -0.00532   -0.00034    0.52774    0.51731    0.53421    0.60213    0.56908    0.53463    0.53224    0.50351    0.51184    0.59365    0.62535    0.54887    0.47807    0.46676    0.52961    0.46345
            6   -0.02688    0.00467   -0.00597   -0.00560   -0.01043    0.52774    0.53421    0.59962    0.56579    0.53047    0.53092    0.49649    0.50526    0.58978    0.62188    0.55169    0.48392    0.47368    0.53289    0.47222
            7   -0.02299   -0.00082   -0.01571   -0.01106   -0.00786   -0.00786    0.55641    0.59286    0.57500    0.53947    0.54750    0.50500    0.51687    0.59485    0.61908    0.55982    0.50556    0.48947    0.54688    0.48819
            8    0.01106    0.03766    0.00405    0.01316    0.03338    0.03088    0.00977    0.60976    0.62440    0.57268    0.61667    0.54603    0.57262    0.64496    0.64975    0.59779    0.56878    0.53509    0.61369    0.53704
            9   -0.02532   -0.00351   -0.01340   -0.01375   -0.00569   -0.00898   -0.01410    0.00863    0.62179    0.56974    0.57937    0.53167    0.54375    0.62500    0.65132    0.59554    0.54306    0.52632    0.58125    0.52639
           10   -0.02013    0.01208   -0.01238   -0.00674    0.00049   -0.00367   -0.00900   -0.00247   -0.01143    0.54054    0.54868    0.49825    0.51579    0.59288    0.61080    0.54887    0.50000    0.47922    0.54868    0.47953
           11   -0.02541   -0.00146   -0.00697   -0.00795   -0.01048   -0.01179   -0.00955    0.03294   -0.01037   -0.00043    0.55769    0.51000    0.51625    0.59926    0.63355    0.57054    0.50556    0.49605    0.54563    0.49444
           12   -0.02553   -0.00607   -0.01908   -0.01652    0.00056   -0.00646   -0.01229    0.00207   -0.01831   -0.01111   -0.00793    0.47816    0.47417    0.54804    0.56930    0.52500    0.48148    0.46667    0.51417    0.46759
           13   -0.02372   -0.00816   -0.01179   -0.01340    0.00118   -0.00540   -0.00813    0.02095   -0.01394   -0.00128   -0.00939   -0.01171    0.49359    0.55809    0.58684    0.54375    0.49444    0.48421    0.52187    0.48472
           14   -0.02135   -0.00910   -0.01123   -0.01504    0.00091   -0.00296   -0.01223    0.01120   -0.01477   -0.00627   -0.00846   -0.01992   -0.01758    0.65775    0.66641    0.62290    0.57680    0.56037    0.60294    0.56127
           15   -0.01286   -0.00189   -0.01188   -0.01151    0.01013    0.00666   -0.01048   -0.00648   -0.01093   -0.01082    0.00336   -0.02113   -0.01130   -0.01382    0.70270    0.64098    0.60380    0.57895    0.63487    0.58041
           16   -0.01721    0.01926   -0.01108   -0.00107   -0.00732   -0.00451   -0.01071    0.00058   -0.00769   -0.01373   -0.00064   -0.00641    0.00463    0.00169   -0.00270    0.58466    0.50595    0.48308    0.56518    0.48115
           17   -0.00629    0.04174    0.01012    0.02241   -0.00485    0.00100    0.00830    0.04486    0.01311    0.01068    0.00766    0.02335    0.02860    0.02887    0.03340   -0.00542    0.43810    0.41228    0.49861    0.40741
           18   -0.00072    0.05010    0.00722    0.02470    0.00090    0.00782    0.00928    0.02822    0.01343    0.00696    0.01522    0.02559    0.03542    0.02950    0.02560   -0.01124   -0.00876    0.40398    0.48684    0.38889
           19   -0.02184    0.00479   -0.00760   -0.00420   -0.01311   -0.00982   -0.01018    0.02997   -0.00849   -0.00043   -0.01207   -0.00376   -0.00377   -0.00478    0.00467   -0.00600    0.00072    0.00600    0.55769    0.48403
           20    0.00403    0.05494    0.01151    0.03004    0.00276    0.01153    0.01316    0.03533    0.01867    0.01244    0.01877    0.03169    0.04110    0.03557    0.03223   -0.00800   -0.00847   -0.00993    0.00836    0.39365


-----------
PXY P value
-----------

                         1           2           3           4           5           6           7           8           9          10          11          12          13          14          15          16          17          18          19

             2     0.47898
             3     0.86804     0.52102
             4     0.68231     0.68426     0.90616
             5     0.48876     0.17204     0.67644     0.39101
             6     0.53763     0.26686     0.72043     0.62757     0.81427
             7     0.57185     0.40078     1.00000     0.86706     0.69306     0.69110
             8     0.60606     0.05181     0.24047     0.23949     0.05963     0.10068     0.25024
             9     0.88270     0.47898     0.98338     0.96970     0.60606     0.91105     0.99022     0.21603
            10     0.60117     0.13783     0.95992     0.66667     0.32160     0.46725     0.70186     0.62463     0.89736
            11     0.62757     0.37146     0.60313     0.63343     0.86510     0.96188     0.78592     0.04594     0.78299     0.39980
            12     0.52102     0.70088     0.95406     0.90029     0.57771     0.74487     0.82893     0.81036     0.91496     0.80547     0.81036
            13     0.51711     0.71652     0.84751     0.92571     0.28641     0.49267     0.66667     0.13392     0.93157     0.37146     0.68426     0.68524
            14     0.95797     0.37146     0.71359     0.73021     0.25611     0.36364     0.56109     0.18671     0.78201     0.39394     0.35484     0.97556     0.52395
            15     0.97947     0.34409     0.78397     0.72336     0.19844     0.24145     0.60508     0.48778     0.68133     0.73607     0.23558     0.99707     0.60215     0.96676
            16     0.65982     0.12610     0.91105     0.52102     0.43597     0.35679     0.58358     0.58065     0.68817     0.61681     0.33138     0.46921     0.20626     0.53177     0.93548
            17     0.25904     0.03226     0.35679     0.16227     0.65494     0.42424     0.32356     0.06843     0.26491     0.25513     0.32258     0.07918     0.09384     0.06061     0.08211     0.20919
            18     0.17302     0.01662     0.28152     0.11241     0.28152     0.22874     0.22190     0.10459     0.15152     0.25318     0.12414     0.06256     0.04106     0.02542     0.05963     0.21310     0.75367
            19     0.54252     0.25220     0.60606     0.51026     1.00000     0.87879     0.83578     0.06745     0.74096     0.34995     0.93939     0.73412     0.45161     0.33822     0.22190     0.43695     0.55132     0.27077
            20     0.14761     0.01271     0.37537     0.11828     0.36364     0.21212     0.26393     0.11730     0.15934     0.21701     0.16618     0.01857     0.04106     0.03030     0.08407     0.20430     0.81134     0.89150     0.36657


---------------------
Corrected PXY P value
---------------------

                           1            2            3            4            5            6            7            8            9           10           11           12           13           14           15           16           17           18           19

              2      0.64027
              3      0.79961      0.51320
              4      0.91300      0.67546      0.90616
              5      0.81427      0.15347      0.47996      0.35582
              6      1.00000      0.24829      0.54252      0.53763      0.81427
              7      0.89834      0.38807      1.00000      0.86706      0.61877      0.63148
              8      0.31769      0.04301      0.25220      0.21114      0.04008      0.08016      0.21701
              9      0.94917      0.47019      0.98338      0.96970      0.47507      0.63636      0.99022      0.20723
             10      0.72532      0.14174      0.84066      0.59433      0.32160      0.46725      0.63734      0.45943      0.73314
             11      1.00000      0.35875      0.60313      0.63343      0.72532      0.87488      0.78592      0.03324      0.78299      0.36168
             12      0.89150      0.56989      0.95894      0.90909      0.28446      0.53959      0.75464      0.37439      0.94526      0.68426      0.57771
             13      0.90323      0.65787      0.84653      0.92571      0.28348      0.51417      0.66373      0.11339      0.93157      0.37537      0.68426      0.76540
             14      0.66862      0.70479      0.70577      0.94917      0.34506      0.46334      0.81232      0.22678      0.89150      0.50929      0.63050      0.95503      0.96285
             15      0.64027      0.44966      0.82014      0.81818      0.19844      0.24145      0.73803      0.61388      0.75660      0.73607      0.27859      0.97752      0.77713      0.83480
             16      0.69697      0.14370      0.68426      0.44184      0.58847      0.44477      0.70381      0.45064      0.54350      0.78886      0.38514      0.50342      0.27957      0.37634      0.48485
             17      0.52981      0.02151      0.15347      0.08407      0.47703      0.34702      0.18768      0.02444      0.10068      0.15347      0.18671      0.06940      0.08309      0.07429      0.05963      0.55132
             18      0.38612      0.01075      0.14761      0.07234      0.28055      0.22874      0.14370      0.04888      0.08407      0.25318      0.08407      0.04008      0.03519      0.06354      0.05963      0.72141      0.72532
             19      0.82307      0.25122      0.60606      0.51026      0.96090      0.77810      0.83578      0.05767      0.74096      0.34897      0.93939      0.43500      0.45161      0.45846      0.25318      0.51808      0.30792      0.18768
             20      0.26686      0.00978      0.08993      0.06549      0.24047      0.12121      0.09775      0.03128      0.04888      0.12414      0.07038      0.02248      0.02053      0.05279      0.04301      0.60704      0.62170      0.78788      0.15347

<A NAME="08_01_09at11_38_25_exct_tests">&nbsp;</A>

================================================================================
== Exact Test of Sample Differentiation Based on Haplotype Frequencies
================================================================================
Reference: <i>Raymond M. and F. Rousset. 1995 .</i>
           <i>Goudet, J., M. Raymond, T. de Meeüs and F. Rousset, 1996.</i>

List of labels for population samples used below:
-------------------------------------------------

Label  	Population name
-----  	---------------
  1	2_Handumon
  2	7_Santander
  3	8_Boljoon
  4	9_Argao
  5	10_Carcar
  6	11_Minglanilla
  7	13_Danao
  8	14_Sogod
  9	15_Tabogon
  10	16_Daanbantayan
  11	17_Malapascua
  12	18_Pintuyan
  13	19_PadreBurgos
  14	20_Maasin
  15	22_InopacanIsland
  16	22_InopacanMainland
  17	23_Baybay
  18	24_Albuera
  19	25_Ormoc
  20	27_Palompon


--------------------------------------------------------------------------
Global test of differentiation among sample:
--------------------------------------------

 Non-differentiation: Exact P value = 0.40647 +- 0.16184  (100000 Markov steps done)

--------------------------------------------------------------------------
Differentiation test between all pairs of samples:
--------------------------------------------------

Markov chain length : 100000 steps)

Non-differentiation exact P values :

                1                 2                 3                 4                 5                 6                 7                 8                 9                10                11                12                13                14                15                16                17                18                19

           2   1.00000+-0.0000
           3   1.00000+-0.0000   1.00000+-0.0000
           4   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000
           5   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000
           6   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000
           7   0.60768+-0.0093   0.48325+-0.0202   0.47334+-0.0064   0.49354+-0.0081   0.74406+-0.0091   0.46083+-0.0134
           8   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.49279+-0.0198
           9   0.61515+-0.0082   0.74162+-0.0083   0.51381+-0.0118   0.48210+-0.0188   0.49075+-0.0196   0.74135+-0.0096   0.24812+-0.0123   0.72499+-0.0112
          10   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.46218+-0.0152   1.00000+-0.0000   0.49023+-0.0141
          11   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.49478+-0.0116   1.00000+-0.0000   0.49971+-0.0185   1.00000+-0.0000
          12   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.50933+-0.0156   1.00000+-0.0000   0.75082+-0.0124   1.00000+-0.0000   1.00000+-0.0000
          13   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.48693+-0.0140   1.00000+-0.0000   0.74021+-0.0113   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000
          14   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.74207+-0.0142   1.00000+-0.0000   0.49199+-0.0160   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000
          15   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.50492+-0.0180   1.00000+-0.0000   0.48798+-0.0136   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000
          16   0.53623+-0.0106   0.47762+-0.0087   0.47872+-0.0155   1.00000+-0.0000   0.50488+-0.0090   0.49196+-0.0166   0.81116+-0.0131   1.00000+-0.0000   0.25692+-0.0078   0.49873+-0.0168   0.49275+-0.0085   0.46036+-0.0084   0.51395+-0.0112   0.74653+-0.0064   0.51050+-0.0153
          17   0.19667+-0.0053   0.31927+-0.0075   0.11829+-0.0060   0.35727+-0.0114   0.35535+-0.0138   0.10626+-0.0074   0.17224+-0.0070   0.32826+-0.0078   0.17767+-0.0081   0.11511+-0.0072   0.11780+-0.0055   0.12110+-0.0087   0.10857+-0.0035   0.11452+-0.0076   0.46643+-0.0099   0.33834+-0.0131
          18   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.49921+-0.0206   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.75909+-0.0100   0.11127+-0.0044
          19   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.86748+-0.0077   1.00000+-0.0000   0.51286+-0.0162   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.75705+-0.0110   0.10927+-0.0043   1.00000+-0.0000
          20   0.59761+-0.0087   0.48457+-0.0125   0.74274+-0.0131   0.48038+-0.0109   0.47907+-0.0124   0.50233+-0.0118   0.61411+-0.0182   0.47588+-0.0122   0.25335+-0.0135   0.49663+-0.0141   0.49280+-0.0114   0.47644+-0.0132   0.71310+-0.0078   0.46598+-0.0139   0.48495+-0.0094   0.22803+-0.0060   0.05948+-0.0046   0.51664+-0.0068   0.50949+-0.0138


--------------------------------
Table of significant differences (significance level=0.0500):
--------------------------------

                     1         2         3         4         5         6         7         8         9        10        11        12        13        14        15        16        17        18        19        20
           1                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           2         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           3         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           4         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           5         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           6         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -
           7         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -
           8         -         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -
           9         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -
          10         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -
          11         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -
          12         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -         -
          13         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -
          14         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         -
          15         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         -
          16         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -
          17         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -
          18         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -
          19         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -
          20         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -          


------------------------------------------------------------
Histogram of the number of significant different populations (significance level=0.0500):
------------------------------------------------------------

------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
|         1|         2|         3|         4|         5|         6|         7|         8|         9|        10|        11|        12|        13|        14|        15|        16|        17|        18|        19|        20|
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="08_01_09at11_38_25_samp0">&nbsp;</A>

===============================================================================
== Sample : 	2_Handumon
===============================================================================


===============================
== Standard diversity indices : (2_Handumon)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 14
No. of loci              : 4
No. of usable loci       : 1 loci with less than  5.00 % missing data
No. of polymorphic loci  : 1


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        14         3   0.57143   0.58242
     2         8         6   0.75000   0.92857
     4         4         3   1.00000   0.83333
-------------------------------------------------
  Mean     8.667     4.000   0.77381   0.78144
  s.d.     5.033     1.732   0.21528   0.17882
-------------------------------------------------



================================
== Molecular diversity indices : (2_Handumon)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 14.00000
No. of haplotypes                            : 13

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 1
Number of usable loci                        : 1


Average gene diversity over loci              :     0.582418 +/-     0.557654

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (2_Handumon)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -25.50310          LnLHood LE :  -30.83571
          Exact P=  0.58594  +-   0.00463 (10100 permutations done)     Chi-square test value=10.66522 (P =  0.55781,  12 d.f.)
Pair(0, 2)
          LnLHood LD :    0.00000          LnLHood LE :    0.00000
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=10.66522 (P =  0.55781,  12 d.f.)
Pair(1, 2)
          LnLHood LD :    0.00000          LnLHood LE :    0.00000
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=10.66522 (P =  0.55781,  12 d.f.)
Pair(0, 3)
          LnLHood LD :  -17.08711          LnLHood LE :  -20.63985
          Exact P=  0.28832  +-   0.00485 (10100 permutations done)     Chi-square test value= 7.10550 (P =  0.31120,  6 d.f.)
Pair(1, 3)
          LnLHood LD :  -27.93589          LnLHood LE :  -32.49258
          Exact P=  0.87277  +-   0.00310 (10100 permutations done)     Chi-square test value= 9.11338 (P =  0.95706,  18 d.f.)
Pair(2, 3)
          LnLHood LD :    0.00000          LnLHood LE :    0.00000
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value= 9.11338 (P =  0.95706,  18 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (2_Handumon)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1       7       0.57143      0.58242    1.00000  0.00000     1001000
    2       4       0.75000      0.92857    0.31714  0.00045     1001000
    3   This locus is monomorphic: no test done.
    4       2       1.00000      0.83333    1.00000  0.00000     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp1">&nbsp;</A>

===============================================================================
== Sample : 	7_Santander
===============================================================================


===============================
== Standard diversity indices : (7_Santander)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 40
No. of loci              : 4
No. of usable loci       : 0 loci with less than  5.00 % missing data
No. of polymorphic loci  : 0


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        36         4   0.55556   0.63016
     2        34         8   0.64706   0.69340
     3        14         4   0.71429   0.70330
     4         6         2   0.00000   0.53333
-------------------------------------------------
  Mean    22.500     4.500   0.47923   0.64005
  s.d.    14.821     2.517   0.32604   0.07817
-------------------------------------------------



================================
== Molecular diversity indices : (7_Santander)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 40.00000
No. of haplotypes                            : 29

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 0
Number of usable loci                        : 0


Average gene diversity over loci              :     0.000000 +/-     0.000000

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (7_Santander)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -89.56542          LnLHood LE : -105.27950
          Exact P=  0.15802  +-   0.00416 (10100 permutations done)     Chi-square test value=31.42816 (P =  0.49534,  32 d.f.)
Pair(0, 2)
          LnLHood LD :  -72.69964          LnLHood LE :  -83.58781
          Exact P=  0.04356  +-   0.00226 (10100 permutations done)     Chi-square test value=21.77634 (P =  0.15056,  16 d.f.)
Pair(1, 2)
          LnLHood LD :  -89.71997          LnLHood LE : -100.02346
          Exact P=  0.74644  +-   0.00419 (10100 permutations done)     Chi-square test value=20.60698 (P =  0.93991,  32 d.f.)
Pair(0, 3)
          LnLHood LD :  -61.72786          LnLHood LE :  -65.14937
          Exact P=  0.50347  +-   0.00494 (10100 permutations done)     Chi-square test value= 6.84301 (P =  0.55366,  8 d.f.)
Pair(1, 3)
          LnLHood LD :  -72.69761          LnLHood LE :  -81.58502
          Exact P=  0.11525  +-   0.00298 (10100 permutations done)     Chi-square test value=17.77483 (P =  0.33725,  16 d.f.)
Pair(2, 3)
          LnLHood LD :  -57.03028          LnLHood LE :  -59.89334
          Exact P=  0.49168  +-   0.00461 (10100 permutations done)     Chi-square test value= 5.72611 (P =  0.67788,  8 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           1    0    1    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   +   - 
       1 |  -   *   -   - 
       2 |  +   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (7_Santander)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      18       0.55556      0.63016    0.47280  0.00045     1001000
    2      17       0.64706      0.69340    0.58872  0.00052     1001000
    3       7       0.71429      0.70330    0.83251  0.00038     1001000
    4       3       0.00000      0.53333    0.20044  0.00039     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp2">&nbsp;</A>

===============================================================================
== Sample : 	8_Boljoon
===============================================================================


===============================
== Standard diversity indices : (8_Boljoon)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 40
No. of loci              : 4
No. of usable loci       : 1 loci with less than  5.00 % missing data
No. of polymorphic loci  : 1


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        40         5   0.75000   0.65000
     2        34         7   0.64706   0.71480
     3        18         3   0.66667   0.67974
     4        30         3   0.33333   0.54483
-------------------------------------------------
  Mean    30.500     4.500   0.59926   0.64734
  s.d.     9.292     1.915   0.18282   0.07329
-------------------------------------------------



================================
== Molecular diversity indices : (8_Boljoon)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 40.00000
No. of haplotypes                            : 34

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 1
Number of usable loci                        : 1


Average gene diversity over loci              :     0.650000 +/-     0.571137

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (8_Boljoon)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -88.68121          LnLHood LE :  -98.76725
          Exact P=  0.82574  +-   0.00396 (10100 permutations done)     Chi-square test value=20.17207 (P =  0.85811,  28 d.f.)
Pair(0, 2)
          LnLHood LD :  -74.00422          LnLHood LE :  -80.99048
          Exact P=  0.35554  +-   0.00445 (10100 permutations done)     Chi-square test value=13.97252 (P =  0.30247,  12 d.f.)
Pair(1, 2)
          LnLHood LD :  -93.40693          LnLHood LE : -102.70096
          Exact P=  0.71931  +-   0.00412 (10100 permutations done)     Chi-square test value=18.58807 (P =  0.61155,  21 d.f.)
Pair(0, 3)
          LnLHood LD :  -76.55912          LnLHood LE :  -81.88622
          Exact P=  0.64446  +-   0.00431 (10100 permutations done)     Chi-square test value=10.65419 (P =  0.55876,  12 d.f.)
Pair(1, 3)
          LnLHood LD :  -93.54782          LnLHood LE : -103.59670
          Exact P=  0.49426  +-   0.00473 (10100 permutations done)     Chi-square test value=20.09775 (P =  0.51508,  21 d.f.)
Pair(2, 3)
          LnLHood LD :  -76.98586          LnLHood LE :  -85.81992
          Exact P=  0.03436  +-   0.00167 (10100 permutations done)     Chi-square test value=17.66812 (P =  0.03923,  9 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    1    1
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   + 
       3 |  -   -   +   * 

===============================
== Hardy-Weinberg equilibrium : (8_Boljoon)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.75000      0.65000    0.49264  0.00050     1001000
    2      17       0.64706      0.71480    0.11144  0.00031     1001000
    3       9       0.66667      0.67974    1.00000  0.00000     1001000
    4      15       0.33333      0.54483    0.11720  0.00030     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp3">&nbsp;</A>

===============================================================================
== Sample : 	9_Argao
===============================================================================


===============================
== Standard diversity indices : (9_Argao)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 40
No. of loci              : 4
No. of usable loci       : 1 loci with less than  5.00 % missing data
No. of polymorphic loci  : 1


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        38         6   0.52632   0.64154
     2        34        10   0.88235   0.85205
     3        16         4   0.87500   0.71667
     4        28         2   0.71429   0.51852
-------------------------------------------------
  Mean    29.000     5.500   0.74949   0.68219
  s.d.     9.592     3.416   0.16778   0.13962
-------------------------------------------------



================================
== Molecular diversity indices : (9_Argao)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 40.00000
No. of haplotypes                            : 37

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 1
Number of usable loci                        : 1


Average gene diversity over loci              :     0.578205 +/-     0.529620

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (9_Argao)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD : -101.43789          LnLHood LE : -120.38793
          Exact P=  0.59050  +-   0.00526 (10100 permutations done)     Chi-square test value=37.90007 (P =  0.98853,  60 d.f.)
Pair(0, 2)
          LnLHood LD :  -77.89147          LnLHood LE :  -88.11835
          Exact P=  0.29713  +-   0.00434 (10100 permutations done)     Chi-square test value=20.45377 (P =  0.67070,  24 d.f.)
Pair(1, 2)
          LnLHood LD : -102.55873          LnLHood LE : -115.20049
          Exact P=  0.90634  +-   0.00273 (10100 permutations done)     Chi-square test value=25.28352 (P =  0.96628,  40 d.f.)
Pair(0, 3)
          LnLHood LD :  -76.52941          LnLHood LE :  -83.56411
          Exact P=  0.30644  +-   0.00484 (10100 permutations done)     Chi-square test value=14.06941 (P =  0.29630,  12 d.f.)
Pair(1, 3)
          LnLHood LD :  -97.21246          LnLHood LE : -110.64626
          Exact P=  0.25851  +-   0.00432 (10100 permutations done)     Chi-square test value=26.86759 (P =  0.13903,  20 d.f.)
Pair(2, 3)
          LnLHood LD :  -71.24463          LnLHood LE :  -78.37668
          Exact P=  0.09218  +-   0.00273 (10100 permutations done)     Chi-square test value=14.26410 (P =  0.07514,  8 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (9_Argao)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      19       0.52632      0.64154    0.35959  0.00036     1001000
    2      17       0.88235      0.85205    0.72211  0.00039     1001000
    3       8       0.87500      0.71667    0.88793  0.00030     1001000
    4      14       0.71429      0.51852    0.28252  0.00044     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp4">&nbsp;</A>

===============================================================================
== Sample : 	10_Carcar
===============================================================================


===============================
== Standard diversity indices : (10_Carcar)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 38
No. of loci              : 4
No. of usable loci       : 1 loci with less than  5.00 % missing data
No. of polymorphic loci  : 1


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        38         4   0.57895   0.52774
     2        36        10   0.77778   0.76032
     3        16         3   0.75000   0.70000
     4        26         3   0.46154   0.55077
-------------------------------------------------
  Mean    29.000     5.000   0.64207   0.63471
  s.d.    10.132     3.367   0.14904   0.11333
-------------------------------------------------



================================
== Molecular diversity indices : (10_Carcar)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 38.00000
No. of haplotypes                            : 30

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 1
Number of usable loci                        : 1


Average gene diversity over loci              :     0.527738 +/-     0.500466

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (10_Carcar)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -80.79627          LnLHood LE :  -89.24478
          Exact P=  0.91079  +-   0.00285 (10100 permutations done)     Chi-square test value=16.89701 (P =  0.97379,  30 d.f.)
Pair(0, 2)
          LnLHood LD :  -60.50896          LnLHood LE :  -65.58739
          Exact P=  0.29020  +-   0.00431 (10100 permutations done)     Chi-square test value=10.15685 (P =  0.33794,  9 d.f.)
Pair(1, 2)
          LnLHood LD :  -87.98623          LnLHood LE : -101.69772
          Exact P=  0.44406  +-   0.00490 (10100 permutations done)     Chi-square test value=27.42298 (P =  0.60099,  30 d.f.)
Pair(0, 3)
          LnLHood LD :  -63.81363          LnLHood LE :  -67.49408
          Exact P=  0.58733  +-   0.00465 (10100 permutations done)     Chi-square test value= 7.36090 (P =  0.59960,  9 d.f.)
Pair(1, 3)
          LnLHood LD :  -85.09942          LnLHood LE : -103.60442
          Exact P=  0.02614  +-   0.00156 (10100 permutations done)     Chi-square test value=37.01000 (P =  0.17685,  30 d.f.)
Pair(2, 3)
          LnLHood LD :  -71.11511          LnLHood LE :  -79.94703
          Exact P=  0.03465  +-   0.00185 (10100 permutations done)     Chi-square test value=17.66384 (P =  0.03928,  9 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    1    1    2
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   + 
       2 |  -   -   *   + 
       3 |  -   +   +   * 

===============================
== Hardy-Weinberg equilibrium : (10_Carcar)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      19       0.57895      0.52774    0.57609  0.00050     1001000
    2      18       0.77778      0.76032    0.78586  0.00021     1001000
    3       8       0.75000      0.70000    0.17416  0.00036     1001000
    4      13       0.46154      0.55077    0.77267  0.00040     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp5">&nbsp;</A>

===============================================================================
== Sample : 	11_Minglanilla
===============================================================================


===============================
== Standard diversity indices : (11_Minglanilla)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 38
No. of loci              : 4
No. of usable loci       : 1 loci with less than  5.00 % missing data
No. of polymorphic loci  : 1


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        38         3   0.47368   0.52774
     2        34         8   0.76471   0.72549
     3        12         4   0.66667   0.69697
     4        28         4   0.50000   0.61905
-------------------------------------------------
  Mean    28.000     4.750   0.60126   0.64231
  s.d.    11.431     2.217   0.13847   0.08865
-------------------------------------------------



================================
== Molecular diversity indices : (11_Minglanilla)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 38.00000
No. of haplotypes                            : 29

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 1
Number of usable loci                        : 1


Average gene diversity over loci              :     0.527738 +/-     0.500466

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (11_Minglanilla)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -75.52434          LnLHood LE :  -83.23905
          Exact P=  0.48624  +-   0.00465 (10100 permutations done)     Chi-square test value=15.42941 (P =  0.49344,  16 d.f.)
Pair(0, 2)
          LnLHood LD :  -55.61403          LnLHood LE :  -59.79374
          Exact P=  0.28584  +-   0.00445 (10100 permutations done)     Chi-square test value= 8.35943 (P =  0.39917,  8 d.f.)
Pair(1, 2)
          LnLHood LD :  -83.13420          LnLHood LE :  -92.74690
          Exact P=  0.68723  +-   0.00467 (10100 permutations done)     Chi-square test value=19.22539 (P =  0.96344,  32 d.f.)
Pair(0, 3)
          LnLHood LD :  -67.42742          LnLHood LE :  -71.80173
          Exact P=  0.37010  +-   0.00487 (10100 permutations done)     Chi-square test value= 8.74861 (P =  0.36395,  8 d.f.)
Pair(1, 3)
          LnLHood LD :  -87.83884          LnLHood LE : -104.75489
          Exact P=  0.16376  +-   0.00370 (10100 permutations done)     Chi-square test value=33.83210 (P =  0.37910,  32 d.f.)
Pair(2, 3)
          LnLHood LD :  -76.07305          LnLHood LE :  -81.30958
          Exact P=  0.84941  +-   0.00330 (10100 permutations done)     Chi-square test value=10.47306 (P =  0.84079,  16 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (11_Minglanilla)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      19       0.47368      0.52774    0.66504  0.00047     1001000
    2      17       0.76471      0.72549    0.74328  0.00038     1001000
    3       6       0.66667      0.69697    0.24447  0.00046     1001000
    4      14       0.50000      0.61905    0.61939  0.00050     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp6">&nbsp;</A>

===============================================================================
== Sample : 	13_Danao
===============================================================================


===============================
== Standard diversity indices : (13_Danao)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 40
No. of loci              : 4
No. of usable loci       : 2 loci with less than  5.00 % missing data
No. of polymorphic loci  : 2


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        38         5   0.63158   0.61735
     2        38         9   0.73684   0.77952
     3        12         4   0.50000   0.45455
     4        26         4   0.38462   0.58154
-------------------------------------------------
  Mean    28.500     5.500   0.56326   0.60824
  s.d.    12.369     2.380   0.15353   0.13386
-------------------------------------------------



================================
== Molecular diversity indices : (13_Danao)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 40.00000
No. of haplotypes                            : 33

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 2
Number of usable loci                        : 2


Average gene diversity over loci              :     0.629487 +/-     0.450112

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (13_Danao)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -88.61498          LnLHood LE : -104.59314
          Exact P=  0.37644  +-   0.00443 (10100 permutations done)     Chi-square test value=31.95631 (P =  0.92838,  45 d.f.)
Pair(0, 2)
          LnLHood LD :  -70.46820          LnLHood LE :  -75.25616
          Exact P=  0.88525  +-   0.00290 (10100 permutations done)     Chi-square test value= 9.57591 (P =  0.97523,  20 d.f.)
Pair(1, 2)
          LnLHood LD :  -81.39742          LnLHood LE :  -94.13496
          Exact P=  0.18505  +-   0.00355 (10100 permutations done)     Chi-square test value=25.47508 (P =  0.90431,  36 d.f.)
Pair(0, 3)
          LnLHood LD :  -82.72314          LnLHood LE :  -90.02715
          Exact P=  0.72149  +-   0.00449 (10100 permutations done)     Chi-square test value=14.60802 (P =  0.79838,  20 d.f.)
Pair(1, 3)
          LnLHood LD :  -95.62401          LnLHood LE : -108.90595
          Exact P=  0.54307  +-   0.00543 (10100 permutations done)     Chi-square test value=26.56389 (P =  0.87430,  36 d.f.)
Pair(2, 3)
          LnLHood LD :  -74.15515          LnLHood LE :  -79.56898
          Exact P=  0.60248  +-   0.00490 (10100 permutations done)     Chi-square test value=10.82764 (P =  0.82000,  16 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (13_Danao)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      19       0.63158      0.61735    0.07474  0.00025     1001000
    2      19       0.73684      0.77952    0.49438  0.00035     1001000
    3       6       0.50000      0.45455    1.00000  0.00000     1001000
    4      13       0.38462      0.58154    0.19467  0.00043     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp7">&nbsp;</A>

===============================================================================
== Sample : 	14_Sogod
===============================================================================


===============================
== Standard diversity indices : (14_Sogod)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 42
No. of loci              : 4
No. of usable loci       : 1 loci with less than  5.00 % missing data
No. of polymorphic loci  : 1


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        40         4   0.55000   0.67308
     2        36         8   0.83333   0.78730
     3        22         5   0.81818   0.73593
     4        34         4   0.70588   0.59358
-------------------------------------------------
  Mean    33.000     5.250   0.72685   0.69747
  s.d.     7.746     1.893   0.13089   0.08354
-------------------------------------------------



================================
== Molecular diversity indices : (14_Sogod)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 42.00000
No. of haplotypes                            : 39

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 1
Number of usable loci                        : 1


Average gene diversity over loci              :     0.609756 +/-     0.547335

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (14_Sogod)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -97.05725          LnLHood LE : -116.77279
          Exact P=  0.03277  +-   0.00173 (10100 permutations done)     Chi-square test value=39.43107 (P =  0.17173,  32 d.f.)
Pair(0, 2)
          LnLHood LD :  -89.21005          LnLHood LE : -100.37883
          Exact P=  0.22129  +-   0.00428 (10100 permutations done)     Chi-square test value=22.33755 (P =  0.32248,  20 d.f.)
Pair(1, 2)
          LnLHood LD : -106.44050          LnLHood LE : -121.89522
          Exact P=  0.65515  +-   0.00489 (10100 permutations done)     Chi-square test value=30.90943 (P =  0.84835,  40 d.f.)
Pair(0, 3)
          LnLHood LD :  -84.47429          LnLHood LE :  -93.15930
          Exact P=  0.28356  +-   0.00454 (10100 permutations done)     Chi-square test value=17.37002 (P =  0.36205,  16 d.f.)
Pair(1, 3)
          LnLHood LD : -103.83781          LnLHood LE : -114.67569
          Exact P=  0.91188  +-   0.00239 (10100 permutations done)     Chi-square test value=21.67576 (P =  0.91579,  32 d.f.)
Pair(2, 3)
          LnLHood LD :  -90.46720          LnLHood LE :  -98.28173
          Exact P=  0.68158  +-   0.00492 (10100 permutations done)     Chi-square test value=15.62907 (P =  0.73935,  20 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           1    1    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   +   -   - 
       1 |  +   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (14_Sogod)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.55000      0.67308    0.49784  0.00052     1001000
    2      18       0.83333      0.78730    0.84109  0.00038     1001000
    3      11       0.81818      0.73593    0.69299  0.00043     1001000
    4      17       0.70588      0.59358    0.07013  0.00026     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp8">&nbsp;</A>

===============================================================================
== Sample : 	15_Tabogon
===============================================================================


===============================
== Standard diversity indices : (15_Tabogon)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 40
No. of loci              : 4
No. of usable loci       : 2 loci with less than  5.00 % missing data
No. of polymorphic loci  : 2


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        40         4   0.75000   0.62179
     2        36         8   0.83333   0.81111
     3        16         3   0.50000   0.49167
     4        38         4   0.31579   0.55192
-------------------------------------------------
  Mean    32.500     4.750   0.59978   0.61912
  s.d.    11.121     2.217   0.23645   0.13860
-------------------------------------------------



================================
== Molecular diversity indices : (15_Tabogon)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 40.00000
No. of haplotypes                            : 33

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 2
Number of usable loci                        : 2


Average gene diversity over loci              :     0.559615 +/-     0.413526

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (15_Tabogon)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -89.20169          LnLHood LE : -100.83746
          Exact P=  0.52554  +-   0.00442 (10100 permutations done)     Chi-square test value=23.27155 (P =  0.50382,  24 d.f.)
Pair(0, 2)
          LnLHood LD :  -66.47900          LnLHood LE :  -69.53446
          Exact P=  0.79644  +-   0.00394 (10100 permutations done)     Chi-square test value= 6.11091 (P =  0.72877,  9 d.f.)
Pair(1, 2)
          LnLHood LD :  -93.40693          LnLHood LE : -104.47755
          Exact P=  0.39069  +-   0.00505 (10100 permutations done)     Chi-square test value=22.14124 (P =  0.57084,  24 d.f.)
Pair(0, 3)
          LnLHood LD :  -66.54625          LnLHood LE :  -70.40146
          Exact P=  0.78614  +-   0.00409 (10100 permutations done)     Chi-square test value= 7.71042 (P =  0.80733,  12 d.f.)
Pair(1, 3)
          LnLHood LD :  -87.91742          LnLHood LE : -105.34455
          Exact P=  0.02297  +-   0.00175 (10100 permutations done)     Chi-square test value=34.85427 (P =  0.33376,  32 d.f.)
Pair(2, 3)
          LnLHood LD :  -69.47547          LnLHood LE :  -74.04155
          Exact P=  0.54891  +-   0.00508 (10100 permutations done)     Chi-square test value= 9.13215 (P =  0.69160,  12 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    1    0    1
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   + 
       2 |  -   -   *   - 
       3 |  -   +   -   * 

===============================
== Hardy-Weinberg equilibrium : (15_Tabogon)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.75000      0.62179    0.73422  0.00043     1001000
    2      18       0.83333      0.81111    0.62517  0.00050     1001000
    3       8       0.50000      0.49167    0.28090  0.00045     1001000
    4      19       0.31579      0.55192    0.04777  0.00021     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp9">&nbsp;</A>

===============================================================================
== Sample : 	16_Daanbantayan
===============================================================================


===============================
== Standard diversity indices : (16_Daanbantayan)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 38
No. of loci              : 4
No. of usable loci       : 0 loci with less than  5.00 % missing data
No. of polymorphic loci  : 0


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        36         3   0.61111   0.60317
     2        34         9   0.76471   0.81640
     3        12         3   0.33333   0.43939
     4        24         3   0.50000   0.58333
-------------------------------------------------
  Mean    26.500     4.500   0.55229   0.61058
  s.d.    11.000     3.000   0.18189   0.15542
-------------------------------------------------



================================
== Molecular diversity indices : (16_Daanbantayan)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 38.00000
No. of haplotypes                            : 30

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 0
Number of usable loci                        : 0


Average gene diversity over loci              :     0.000000 +/-     0.000000

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (16_Daanbantayan)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -86.27287          LnLHood LE : -102.55772
          Exact P=  0.09535  +-   0.00289 (10100 permutations done)     Chi-square test value=32.56971 (P =  0.21164,  27 d.f.)
Pair(0, 2)
          LnLHood LD :  -59.15829          LnLHood LE :  -66.49646
          Exact P=  0.04050  +-   0.00190 (10100 permutations done)     Chi-square test value=14.67634 (P =  0.10022,  9 d.f.)
Pair(1, 2)
          LnLHood LD :  -83.79589          LnLHood LE :  -98.00157
          Exact P=  0.04733  +-   0.00197 (10100 permutations done)     Chi-square test value=28.41135 (P =  0.39001,  27 d.f.)
Pair(0, 3)
          LnLHood LD :  -72.37009          LnLHood LE :  -78.14324
          Exact P=  0.31703  +-   0.00513 (10100 permutations done)     Chi-square test value=11.54629 (P =  0.24011,  9 d.f.)
Pair(1, 3)
          LnLHood LD :  -93.36647          LnLHood LE : -109.64835
          Exact P=  0.16545  +-   0.00350 (10100 permutations done)     Chi-square test value=32.56375 (P =  0.21185,  27 d.f.)
Pair(2, 3)
          LnLHood LD :  -67.78429          LnLHood LE :  -73.58709
          Exact P=  0.19277  +-   0.00370 (10100 permutations done)     Chi-square test value=11.60560 (P =  0.23647,  9 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           1    1    2    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   +   - 
       1 |  -   *   +   - 
       2 |  +   +   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (16_Daanbantayan)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      18       0.61111      0.60317    0.75048  0.00044     1001000
    2      17       0.76471      0.81640    0.77891  0.00031     1001000
    3       6       0.33333      0.43939    0.27615  0.00046     1001000
    4      12       0.50000      0.58333    0.79210  0.00043     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp10">&nbsp;</A>

===============================================================================
== Sample : 	17_Malapascua
===============================================================================


===============================
== Standard diversity indices : (17_Malapascua)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 40
No. of loci              : 4
No. of usable loci       : 2 loci with less than  5.00 % missing data
No. of polymorphic loci  : 2


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        40         4   0.60000   0.55769
     2        36        11   0.83333   0.84762
     3        22         6   0.72727   0.67532
     4        38         4   0.68421   0.59602
-------------------------------------------------
  Mean    34.000     6.250   0.71120   0.66916
  s.d.     8.165     3.304   0.09707   0.12866
-------------------------------------------------



================================
== Molecular diversity indices : (17_Malapascua)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 40.00000
No. of haplotypes                            : 33

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 2
Number of usable loci                        : 2


Average gene diversity over loci              :     0.547436 +/-     0.407098

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (17_Malapascua)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -95.84792          LnLHood LE : -103.86522
          Exact P=  0.98614  +-   0.00129 (10100 permutations done)     Chi-square test value=16.03458 (P =  0.99432,  33 d.f.)
Pair(0, 2)
          LnLHood LD :  -69.68844          LnLHood LE :  -80.11136
          Exact P=  0.05347  +-   0.00247 (10100 permutations done)     Chi-square test value=20.84585 (P =  0.28726,  18 d.f.)
Pair(1, 2)
          LnLHood LD : -108.21569          LnLHood LE : -126.40068
          Exact P=  0.90673  +-   0.00371 (10100 permutations done)     Chi-square test value=36.36999 (P =  0.99887,  66 d.f.)
Pair(0, 3)
          LnLHood LD :  -61.82430          LnLHood LE :  -65.78863
          Exact P=  0.69218  +-   0.00461 (10100 permutations done)     Chi-square test value= 7.92866 (P =  0.79068,  12 d.f.)
Pair(1, 3)
          LnLHood LD : -100.65606          LnLHood LE : -112.07795
          Exact P=  0.99099  +-   0.00095 (10100 permutations done)     Chi-square test value=22.84378 (P =  0.99651,  44 d.f.)
Pair(2, 3)
          LnLHood LD :  -76.92774          LnLHood LE :  -88.32410
          Exact P=  0.16950  +-   0.00361 (10100 permutations done)     Chi-square test value=22.79272 (P =  0.53206,  24 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (17_Malapascua)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.60000      0.55769    1.00000  0.00000     1001000
    2      18       0.83333      0.84762    0.55526  0.00029     1001000
    3      11       0.72727      0.67532    0.30125  0.00039     1001000
    4      19       0.68421      0.59602    0.37535  0.00050     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp11">&nbsp;</A>

===============================================================================
== Sample : 	18_Pintuyan
===============================================================================


===============================
== Standard diversity indices : (18_Pintuyan)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 30
No. of loci              : 4
No. of usable loci       : 0 loci with less than  5.00 % missing data
No. of polymorphic loci  : 0


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        26         4   0.61538   0.64000
     2        24         6   0.83333   0.81884
     3        14         3   0.42857   0.64835
     4        16         2   0.87500   0.52500
-------------------------------------------------
  Mean    20.000     3.750   0.68807   0.65805
  s.d.     5.888     1.708   0.20710   0.12107
-------------------------------------------------



================================
== Molecular diversity indices : (18_Pintuyan)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 30.00000
No. of haplotypes                            : 25

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 0
Number of usable loci                        : 0


Average gene diversity over loci              :     0.000000 +/-     0.000000

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (18_Pintuyan)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -67.88826          LnLHood LE :  -81.78301
          Exact P=  0.15970  +-   0.00409 (10100 permutations done)     Chi-square test value=27.78950 (P =  0.26902,  24 d.f.)
Pair(0, 2)
          LnLHood LD :  -58.41171          LnLHood LE :  -67.07057
          Exact P=  0.12228  +-   0.00296 (10100 permutations done)     Chi-square test value=17.31773 (P =  0.13803,  12 d.f.)
Pair(1, 2)
          LnLHood LD :  -69.45800          LnLHood LE :  -79.79285
          Exact P=  0.34574  +-   0.00451 (10100 permutations done)     Chi-square test value=20.66969 (P =  0.29640,  18 d.f.)
Pair(0, 3)
          LnLHood LD :  -54.98904          LnLHood LE :  -61.37106
          Exact P=  0.16584  +-   0.00340 (10100 permutations done)     Chi-square test value=12.76405 (P =  0.12023,  8 d.f.)
Pair(1, 3)
          LnLHood LD :  -65.84027          LnLHood LE :  -74.09334
          Exact P=  0.30010  +-   0.00446 (10100 permutations done)     Chi-square test value=16.50613 (P =  0.16914,  12 d.f.)
Pair(2, 3)
          LnLHood LD :  -54.39657          LnLHood LE :  -59.38090
          Exact P=  0.17307  +-   0.00389 (10100 permutations done)     Chi-square test value= 9.96866 (P =  0.12598,  6 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (18_Pintuyan)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      13       0.61538      0.64000    0.21234  0.00047     1001000
    2      12       0.83333      0.81884    0.72451  0.00046     1001000
    3       7       0.42857      0.64835    0.25532  0.00046     1001000
    4       8       0.87500      0.52500    0.13885  0.00035     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp12">&nbsp;</A>

===============================================================================
== Sample : 	19_PadreBurgos
===============================================================================


===============================
== Standard diversity indices : (19_PadreBurgos)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 40
No. of loci              : 4
No. of usable loci       : 0 loci with less than  5.00 % missing data
No. of polymorphic loci  : 0


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        36         5   0.55556   0.61111
     2        32        10   0.87500   0.84879
     3        18         5   0.77778   0.73203
     4        28         2   0.35714   0.49471
-------------------------------------------------
  Mean    28.500     5.500   0.64137   0.67166
  s.d.     7.724     3.317   0.23190   0.15275
-------------------------------------------------



================================
== Molecular diversity indices : (19_PadreBurgos)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 40.00000
No. of haplotypes                            : 36

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 0
Number of usable loci                        : 0


Average gene diversity over loci              :     0.000000 +/-     0.000000

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (19_PadreBurgos)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD : -100.47929          LnLHood LE : -118.04230
          Exact P=  0.53030  +-   0.00532 (10100 permutations done)     Chi-square test value=35.12602 (P =  0.94507,  50 d.f.)
Pair(0, 2)
          LnLHood LD :  -84.60803          LnLHood LE :  -91.44040
          Exact P=  0.90752  +-   0.00285 (10100 permutations done)     Chi-square test value=13.66473 (P =  0.96727,  25 d.f.)
Pair(1, 2)
          LnLHood LD : -104.46828          LnLHood LE : -120.04522
          Exact P=  0.81228  +-   0.00381 (10100 permutations done)     Chi-square test value=31.15388 (P =  0.98315,  50 d.f.)
Pair(0, 3)
          LnLHood LD :  -76.63887          LnLHood LE :  -84.44783
          Exact P=  0.10743  +-   0.00295 (10100 permutations done)     Chi-square test value=15.61792 (P =  0.11110,  10 d.f.)
Pair(1, 3)
          LnLHood LD : -102.60285          LnLHood LE : -113.05265
          Exact P=  0.68990  +-   0.00441 (10100 permutations done)     Chi-square test value=20.89959 (P =  0.40306,  20 d.f.)
Pair(2, 3)
          LnLHood LD :  -78.69884          LnLHood LE :  -86.45074
          Exact P=  0.14594  +-   0.00332 (10100 permutations done)     Chi-square test value=15.50380 (P =  0.11475,  10 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (19_PadreBurgos)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      18       0.55556      0.61111    0.30367  0.00039     1001000
    2      16       0.87500      0.84879    0.47949  0.00037     1001000
    3       9       0.77778      0.73203    0.84525  0.00039     1001000
    4      14       0.35714      0.49471    0.57092  0.00050     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp13">&nbsp;</A>

===============================================================================
== Sample : 	20_Maasin
===============================================================================


===============================
== Standard diversity indices : (20_Maasin)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 34
No. of loci              : 4
No. of usable loci       : 1 loci with less than  5.00 % missing data
No. of polymorphic loci  : 1


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        34         4   0.58824   0.65775
     2        30         9   0.86667   0.79770
     3        14         5   0.71429   0.72527
     4        26         3   0.30769   0.52000
-------------------------------------------------
  Mean    26.000     5.250   0.61922   0.67518
  s.d.     8.641     2.630   0.23684   0.11819
-------------------------------------------------



================================
== Molecular diversity indices : (20_Maasin)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 34.00000
No. of haplotypes                            : 29

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 1
Number of usable loci                        : 1


Average gene diversity over loci              :     0.657754 +/-     0.578098

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (20_Maasin)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -70.91594          LnLHood LE :  -88.45217
          Exact P=  0.01673  +-   0.00131 (10100 permutations done)     Chi-square test value=35.07247 (P =  0.13704,  27 d.f.)
Pair(0, 2)
          LnLHood LD :  -64.74703          LnLHood LE :  -70.40519
          Exact P=  0.76901  +-   0.00458 (10100 permutations done)     Chi-square test value=11.31632 (P =  0.72987,  15 d.f.)
Pair(1, 2)
          LnLHood LD :  -76.66846          LnLHood LE :  -94.47174
          Exact P=  0.13129  +-   0.00323 (10100 permutations done)     Chi-square test value=35.60655 (P =  0.84068,  45 d.f.)
Pair(0, 3)
          LnLHood LD :  -64.43093          LnLHood LE :  -68.54451
          Exact P=  0.61525  +-   0.00470 (10100 permutations done)     Chi-square test value= 8.22715 (P =  0.51143,  9 d.f.)
Pair(1, 3)
          LnLHood LD :  -80.05574          LnLHood LE :  -92.61106
          Exact P=  0.43941  +-   0.00459 (10100 permutations done)     Chi-square test value=25.11063 (P =  0.56826,  27 d.f.)
Pair(2, 3)
          LnLHood LD :  -65.96057          LnLHood LE :  -74.56407
          Exact P=  0.19743  +-   0.00388 (10100 permutations done)     Chi-square test value=17.20699 (P =  0.30664,  15 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           1    1    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   +   -   - 
       1 |  +   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (20_Maasin)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      17       0.58824      0.65775    0.36771  0.00049     1001000
    2      15       0.86667      0.79770    0.54661  0.00041     1001000
    3       7       0.71429      0.72527    0.88768  0.00031     1001000
    4      13       0.30769      0.52000    0.20968  0.00039     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp14">&nbsp;</A>

===============================================================================
== Sample : 	22_InopacanIsland
===============================================================================


===============================
== Standard diversity indices : (22_InopacanIsland)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 38
No. of loci              : 4
No. of usable loci       : 1 loci with less than  5.00 % missing data
No. of polymorphic loci  : 1


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        38         5   0.63158   0.70270
     2        34         6   0.76471   0.74866
     3        14         2   0.42857   0.36264
     4        24         3   0.50000   0.51812
-------------------------------------------------
  Mean    27.500     4.000   0.58121   0.58303
  s.d.    10.755     1.826   0.14844   0.17752
-------------------------------------------------



================================
== Molecular diversity indices : (22_InopacanIsland)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 38.00000
No. of haplotypes                            : 30

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 1
Number of usable loci                        : 1


Average gene diversity over loci              :     0.702703 +/-     0.601846

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (22_InopacanIsland)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -84.60444          LnLHood LE :  -96.63708
          Exact P=  0.42851  +-   0.00513 (10100 permutations done)     Chi-square test value=24.06529 (P =  0.45787,  24 d.f.)
Pair(0, 2)
          LnLHood LD :  -68.19821          LnLHood LE :  -71.52550
          Exact P=  0.67426  +-   0.00471 (10100 permutations done)     Chi-square test value= 6.65457 (P =  0.57432,  8 d.f.)
Pair(1, 2)
          LnLHood LD :  -83.06498          LnLHood LE :  -85.51730
          Exact P=  0.99683  +-   0.00061 (10100 permutations done)     Chi-square test value= 4.90464 (P =  0.96109,  12 d.f.)
Pair(0, 3)
          LnLHood LD :  -71.92284          LnLHood LE :  -81.18894
          Exact P=  0.05743  +-   0.00216 (10100 permutations done)     Chi-square test value=18.53221 (P =  0.10046,  12 d.f.)
Pair(1, 3)
          LnLHood LD :  -82.30940          LnLHood LE :  -95.18075
          Exact P=  0.06535  +-   0.00256 (10100 permutations done)     Chi-square test value=25.74269 (P =  0.10579,  18 d.f.)
Pair(2, 3)
          LnLHood LD :  -68.31189          LnLHood LE :  -70.06916
          Exact P=  0.82515  +-   0.00387 (10100 permutations done)     Chi-square test value= 3.51454 (P =  0.74203,  6 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (22_InopacanIsland)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      19       0.63158      0.70270    0.54634  0.00048     1001000
    2      17       0.76471      0.74866    0.91805  0.00026     1001000
    3       7       0.42857      0.36264    1.00000  0.00000     1001000
    4      12       0.50000      0.51812    0.47151  0.00049     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp15">&nbsp;</A>

===============================================================================
== Sample : 	22_InopacanMainland
===============================================================================


===============================
== Standard diversity indices : (22_InopacanMainland)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 28
No. of loci              : 4
No. of usable loci       : 1 loci with less than  5.00 % missing data
No. of polymorphic loci  : 1


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        28         4   0.50000   0.58466
     2        22         9   0.72727   0.84848
     3        12         4   0.66667   0.74242
     4        18         2   0.77778   0.52941
-------------------------------------------------
  Mean    20.000     4.750   0.66793   0.67624
  s.d.     6.733     2.986   0.12082   0.14605
-------------------------------------------------



================================
== Molecular diversity indices : (22_InopacanMainland)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 28.00000
No. of haplotypes                            : 24

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 1
Number of usable loci                        : 1


Average gene diversity over loci              :     0.584656 +/-     0.539122

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (22_InopacanMainland)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -63.32097          LnLHood LE :  -74.64464
          Exact P=  0.51317  +-   0.00482 (10100 permutations done)     Chi-square test value=22.64733 (P =  0.70381,  27 d.f.)
Pair(0, 2)
          LnLHood LD :  -49.93600          LnLHood LE :  -55.17839
          Exact P=  0.53545  +-   0.00484 (10100 permutations done)     Chi-square test value=10.48479 (P =  0.57351,  12 d.f.)
Pair(1, 2)
          LnLHood LD :  -65.15327          LnLHood LE :  -81.84436
          Exact P=  0.10446  +-   0.00266 (10100 permutations done)     Chi-square test value=33.38218 (P =  0.59374,  36 d.f.)
Pair(0, 3)
          LnLHood LD :  -47.78203          LnLHood LE :  -49.86314
          Exact P=  0.85119  +-   0.00363 (10100 permutations done)     Chi-square test value= 4.16221 (P =  0.65474,  6 d.f.)
Pair(1, 3)
          LnLHood LD :  -67.82518          LnLHood LE :  -76.52910
          Exact P=  0.81099  +-   0.00368 (10100 permutations done)     Chi-square test value=17.40785 (P =  0.49525,  18 d.f.)
Pair(2, 3)
          LnLHood LD :  -50.14560          LnLHood LE :  -57.06286
          Exact P=  0.08782  +-   0.00276 (10100 permutations done)     Chi-square test value=13.83451 (P =  0.08618,  8 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (22_InopacanMainland)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      14       0.50000      0.58466    0.42289  0.00047     1001000
    2      11       0.72727      0.84848    0.40679  0.00040     1001000
    3       6       0.66667      0.74242    0.12023  0.00037     1001000
    4       9       0.77778      0.52941    0.22527  0.00044     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp16">&nbsp;</A>

===============================================================================
== Sample : 	23_Baybay
===============================================================================


===============================
== Standard diversity indices : (23_Baybay)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 36
No. of loci              : 4
No. of usable loci       : 1 loci with less than  5.00 % missing data
No. of polymorphic loci  : 1


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        36         3   0.44444   0.43810
     2        28         8   0.85714   0.80159
     3        12         5   0.83333   0.83333
     4        22         2   0.54545   0.51948
-------------------------------------------------
  Mean    24.500     4.500   0.67009   0.64812
  s.d.    10.116     2.646   0.20663   0.19876
-------------------------------------------------



================================
== Molecular diversity indices : (23_Baybay)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 36.00000
No. of haplotypes                            : 25

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 1
Number of usable loci                        : 1


Average gene diversity over loci              :     0.438095 +/-     0.446463

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (23_Baybay)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -73.33135          LnLHood LE :  -79.29431
          Exact P=  0.78990  +-   0.00392 (10100 permutations done)     Chi-square test value=11.92592 (P =  0.74906,  16 d.f.)
Pair(0, 2)
          LnLHood LD :  -55.87521          LnLHood LE :  -58.38322
          Exact P=  0.92119  +-   0.00272 (10100 permutations done)     Chi-square test value= 5.01603 (P =  0.89010,  10 d.f.)
Pair(1, 2)
          LnLHood LD :  -77.85659          LnLHood LE :  -96.21920
          Exact P=  0.03050  +-   0.00157 (10100 permutations done)     Chi-square test value=36.72522 (P =  0.61848,  40 d.f.)
Pair(0, 3)
          LnLHood LD :  -53.19439          LnLHood LE :  -55.78543
          Exact P=  0.38505  +-   0.00493 (10100 permutations done)     Chi-square test value= 5.18208 (P =  0.26912,  4 d.f.)
Pair(1, 3)
          LnLHood LD :  -79.45779          LnLHood LE :  -93.62141
          Exact P=  0.02653  +-   0.00178 (10100 permutations done)     Chi-square test value=28.32725 (P =  0.02889,  16 d.f.)
Pair(2, 3)
          LnLHood LD :  -65.04259          LnLHood LE :  -72.71032
          Exact P=  0.13317  +-   0.00363 (10100 permutations done)     Chi-square test value=15.33545 (P =  0.12030,  10 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    2    1    1
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   +   + 
       2 |  -   +   *   - 
       3 |  -   +   -   * 

===============================
== Hardy-Weinberg equilibrium : (23_Baybay)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      18       0.44444      0.43810    1.00000  0.00000     1001000
    2      14       0.85714      0.80159    0.58639  0.00042     1001000
    3       6       0.83333      0.83333    0.55580  0.00057     1001000
    4      11       0.54545      0.51948    1.00000  0.00000     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp17">&nbsp;</A>

===============================================================================
== Sample : 	24_Albuera
===============================================================================


===============================
== Standard diversity indices : (24_Albuera)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 38
No. of loci              : 4
No. of usable loci       : 0 loci with less than  5.00 % missing data
No. of polymorphic loci  : 0


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        36         3   0.55556   0.45079
     2        30         8   0.86667   0.75862
     3        20         5   0.90000   0.72105
     4        26         2   0.46154   0.51692
-------------------------------------------------
  Mean    28.000     4.500   0.69594   0.61185
  s.d.     6.733     2.646   0.22018   0.15102
-------------------------------------------------



================================
== Molecular diversity indices : (24_Albuera)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 38.00000
No. of haplotypes                            : 33

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 0
Number of usable loci                        : 0


Average gene diversity over loci              :     0.000000 +/-     0.000000

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (24_Albuera)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -79.44599          LnLHood LE :  -89.62985
          Exact P=  0.61208  +-   0.00504 (10100 permutations done)     Chi-square test value=20.36771 (P =  0.67568,  24 d.f.)
Pair(0, 2)
          LnLHood LD :  -69.31148          LnLHood LE :  -75.56935
          Exact P=  0.61762  +-   0.00488 (10100 permutations done)     Chi-square test value=12.51575 (P =  0.63965,  15 d.f.)
Pair(1, 2)
          LnLHood LD :  -92.99026          LnLHood LE : -107.39068
          Exact P=  0.73713  +-   0.00446 (10100 permutations done)     Chi-square test value=28.80084 (P =  0.90596,  40 d.f.)
Pair(0, 3)
          LnLHood LD :  -60.37469          LnLHood LE :  -66.38909
          Exact P=  0.08653  +-   0.00267 (10100 permutations done)     Chi-square test value=12.02881 (P =  0.06133,  6 d.f.)
Pair(1, 3)
          LnLHood LD :  -89.61165          LnLHood LE :  -98.21041
          Exact P=  0.67069  +-   0.00453 (10100 permutations done)     Chi-square test value=17.19753 (P =  0.37293,  16 d.f.)
Pair(2, 3)
          LnLHood LD :  -77.48835          LnLHood LE :  -84.14992
          Exact P=  0.31901  +-   0.00435 (10100 permutations done)     Chi-square test value=13.32314 (P =  0.20616,  10 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (24_Albuera)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      18       0.55556      0.45079    0.76432  0.00042     1001000
    2      15       0.86667      0.75862    0.79397  0.00031     1001000
    3      10       0.90000      0.72105    0.94933  0.00021     1001000
    4      13       0.46154      0.51692    1.00000  0.00000     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp18">&nbsp;</A>

===============================================================================
== Sample : 	25_Ormoc
===============================================================================


===============================
== Standard diversity indices : (25_Ormoc)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 40
No. of loci              : 4
No. of usable loci       : 1 loci with less than  5.00 % missing data
No. of polymorphic loci  : 1


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        40         4   0.60000   0.55769
     2        36         7   0.77778   0.79841
     3        12         5   0.83333   0.83333
     4        26         3   0.38462   0.55692
-------------------------------------------------
  Mean    28.500     4.750   0.64893   0.68659
  s.d.    12.477     1.708   0.20237   0.14996
-------------------------------------------------



================================
== Molecular diversity indices : (25_Ormoc)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 40.00000
No. of haplotypes                            : 30

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 1
Number of usable loci                        : 1


Average gene diversity over loci              :     0.557692 +/-     0.517573

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (25_Ormoc)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -85.68929          LnLHood LE :  -94.52661
          Exact P=  0.59941  +-   0.00442 (10100 permutations done)     Chi-square test value=17.67465 (P =  0.66949,  21 d.f.)
Pair(0, 2)
          LnLHood LD :  -59.27449          LnLHood LE :  -69.02352
          Exact P=  0.03277  +-   0.00170 (10100 permutations done)     Chi-square test value=19.49805 (P =  0.19205,  15 d.f.)
Pair(1, 2)
          LnLHood LD :  -85.66808          LnLHood LE : -103.85132
          Exact P=  0.01673  +-   0.00122 (10100 permutations done)     Chi-square test value=36.36649 (P =  0.40486,  35 d.f.)
Pair(0, 3)
          LnLHood LD :  -69.64855          LnLHood LE :  -73.82882
          Exact P=  0.48257  +-   0.00466 (10100 permutations done)     Chi-square test value= 8.36055 (P =  0.49826,  9 d.f.)
Pair(1, 3)
          LnLHood LD :  -99.45047          LnLHood LE : -108.65663
          Exact P=  0.70980  +-   0.00482 (10100 permutations done)     Chi-square test value=18.41232 (P =  0.62278,  21 d.f.)
Pair(2, 3)
          LnLHood LD :  -75.59712          LnLHood LE :  -83.15353
          Exact P=  0.44832  +-   0.00455 (10100 permutations done)     Chi-square test value=15.11283 (P =  0.44332,  15 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           1    1    2    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   +   - 
       1 |  -   *   +   - 
       2 |  +   +   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (25_Ormoc)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.60000      0.55769    0.74516  0.00044     1001000
    2      18       0.77778      0.79841    0.79797  0.00035     1001000
    3       6       0.83333      0.83333    0.55818  0.00049     1001000
    4      13       0.38462      0.55692    0.26253  0.00044     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at11_38_25_samp19">&nbsp;</A>

===============================================================================
== Sample : 	27_Palompon
===============================================================================


===============================
== Standard diversity indices : (27_Palompon)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 36
No. of loci              : 4
No. of usable loci       : 0 loci with less than  5.00 % missing data
No. of polymorphic loci  : 0


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        34         4   0.52941   0.44207
     2        34         9   0.82353   0.83066
     3        16         5   0.50000   0.75000
     4        28         4   0.42857   0.58466
-------------------------------------------------
  Mean    28.000     5.500   0.57038   0.65185
  s.d.     8.485     2.380   0.17400   0.17333
-------------------------------------------------



================================
== Molecular diversity indices : (27_Palompon)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 36.00000
No. of haplotypes                            : 29

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 0
Number of usable loci                        : 0


Average gene diversity over loci              :     0.000000 +/-     0.000000

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (27_Palompon)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -71.44836          LnLHood LE :  -92.00459
          Exact P=  0.00297  +-   0.00055 (10100 permutations done)     Chi-square test value=41.11245 (P =  0.25658,  36 d.f.)
Pair(0, 2)
          LnLHood LD :  -62.44940          LnLHood LE :  -73.42420
          Exact P=  0.06980  +-   0.00283 (10100 permutations done)     Chi-square test value=21.94961 (P =  0.34325,  20 d.f.)
Pair(1, 2)
          LnLHood LD :  -88.55147          LnLHood LE : -106.29451
          Exact P=  0.33950  +-   0.00466 (10100 permutations done)     Chi-square test value=35.48608 (P =  0.84426,  45 d.f.)
Pair(0, 3)
          LnLHood LD :  -64.37742          LnLHood LE :  -71.01372
          Exact P=  0.59436  +-   0.00437 (10100 permutations done)     Chi-square test value=13.27262 (P =  0.65273,  16 d.f.)
Pair(1, 3)
          LnLHood LD :  -85.74608          LnLHood LE : -103.88403
          Exact P=  0.10287  +-   0.00275 (10100 permutations done)     Chi-square test value=36.27591 (P =  0.45579,  36 d.f.)
Pair(2, 3)
          LnLHood LD :  -75.60725          LnLHood LE :  -85.30365
          Exact P=  0.27574  +-   0.00426 (10100 permutations done)     Chi-square test value=19.39280 (P =  0.49644,  20 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           1    1    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   +   -   - 
       1 |  +   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (27_Palompon)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      17       0.52941      0.44207    1.00000  0.00000     1001000
    2      17       0.82353      0.83066    0.42408  0.00047     1001000
    3       8       0.50000      0.75000    0.08923  0.00026     1001000
    4      14       0.42857      0.58466    0.23372  0.00042     1001000
------------------------------------------------------------------------




================================================================================
== Summary of computations done within populations
================================================================================

<A NAME="08_01_09at11_38_25_comp_sum_Basic">&nbsp;</A>

----------------
Basic properties
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
          Statistics           2_Handumon          7_Santander            8_Boljoon              9_Argao            10_Carcar       11_Minglanilla             13_Danao             14_Sogod           15_Tabogon      16_Daanbantayan        17_Malapascua          18_Pintuyan       19_PadreBurgos            20_Maasin    22_InopacanIsland  22_InopacanMainland            23_Baybay           24_Albuera             25_Ormoc          27_Palompon        Mean        s.d.
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  No. of gene copies                   14                   40                   40                   40                   38                   38                   40                   42                   40                   38                   40                   30                   40                   34                   38                   28                   36                   38                   40                   36      36.500       6.354
         No. of loci                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4       4.000       0.000
  No. of usable loci                    1                    0                    1                    1                    1                    1                    2                    1                    2                    0                    2                    0                    0                    1                    1                    1                    1                    0                    1                    0       0.850       0.671
  No. of polym. loci                    1                    0                    1                    1                    1                    1                    2                    1                    2                    0                    2                    0                    0                    1                    1                    1                    1                    0                    1                    0       0.850       0.671
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
<A NAME="08_01_09at11_38_25_comp_sum_het">&nbsp;</A>

-----------------------
Expected heterozygosity
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Locus#           2_Handumon          7_Santander            8_Boljoon              9_Argao            10_Carcar       11_Minglanilla             13_Danao             14_Sogod           15_Tabogon      16_Daanbantayan        17_Malapascua          18_Pintuyan       19_PadreBurgos            20_Maasin    22_InopacanIsland  22_InopacanMainland            23_Baybay           24_Albuera             25_Ormoc          27_Palompon        Mean        s.d.   Tot. Het.
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
     1              0.58242              0.63016              0.65000              0.64154              0.52774              0.52774              0.61735              0.67308              0.62179              0.60317              0.55769              0.64000              0.61111              0.65775              0.70270              0.58466              0.43810              0.45079              0.55769              0.44207     0.58588     0.07639     0.58817
     2              0.92857              0.69340              0.71480              0.85205              0.76032              0.72549              0.77952              0.78730              0.81111              0.81640              0.84762              0.81884              0.84879              0.79770              0.74866              0.84848              0.80159              0.75862              0.79841              0.83066     0.79842     0.05552     0.79336
     3              0.00000              0.70330              0.67974              0.71667              0.70000              0.69697              0.45455              0.73593              0.49167              0.43939              0.67532              0.64835              0.73203              0.72527              0.36264              0.74242              0.83333              0.72105              0.83333              0.75000     0.63210     0.19607     0.67507
     4              0.83333              0.53333              0.54483              0.51852              0.55077              0.61905              0.58154              0.59358              0.55192              0.58333              0.59602              0.52500              0.49471              0.52000              0.51812              0.52941              0.51948              0.51692              0.55692              0.58466     0.56357     0.07185     0.54362
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Mean              0.58608              0.64005              0.64734              0.68219              0.63471              0.64231              0.60824              0.69747              0.61912              0.61058              0.66916              0.65805              0.67166              0.67518              0.58303              0.67624              0.64812              0.61185              0.68659              0.65185     0.64499     0.03327     0.65006
  s.d.              0.41711              0.07817              0.07329              0.13962              0.11333              0.08865              0.13386              0.08354              0.13860              0.15542              0.12866              0.12107              0.15275              0.11819              0.17752              0.14605              0.19876              0.15102              0.14996              0.17333     0.14694     0.07192     0.09529
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
<A NAME="08_01_09at11_38_25_comp_sum_thetaH">&nbsp;</A>

----------------------------------------
Theta(H) under the infinite-allele model
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Locus#           2_Handumon          7_Santander            8_Boljoon              9_Argao            10_Carcar       11_Minglanilla             13_Danao             14_Sogod           15_Tabogon      16_Daanbantayan        17_Malapascua          18_Pintuyan       19_PadreBurgos            20_Maasin    22_InopacanIsland  22_InopacanMainland            23_Baybay           24_Albuera             25_Ormoc          27_Palompon        Mean        s.d.
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
     1              1.39474              1.70386              1.85714              1.78968              1.11747              1.11747              1.61338              2.05882              1.64407              1.52000              1.26087              1.77778              1.57143              1.92187              2.36364              1.40764              0.77966              0.82081              1.26087              0.79233     1.48868     0.41896
     2             13.00000              2.26163              2.50625              5.75904              3.17219              2.64286              3.53548              3.70149              4.29412              4.44660              5.56250              4.52000              5.61333              3.94318              2.97872              5.60000              4.04000              3.14286              3.96063              4.90526     4.47931     2.21753
     3              0.00000              2.37037              2.12245              2.52941              2.33333              2.30000              0.83333              2.78689              0.96721              0.78378              2.08000              1.84375              2.73171              2.64000              0.56897              2.88235              5.00000              2.58491              5.00000              3.00000     2.26792     1.24234
     4              5.00000              1.14286              1.19697              1.07692              1.22603              1.62500              1.38971              1.46053              1.23175              1.40000              1.47535              1.10526              0.97906              1.08333              1.07519              1.12500              1.08108              1.07006              1.25694              1.40764     1.42043     0.83841
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Mean              1.41593              1.77815              1.83559              2.14656              1.73752              1.79573              1.55257              2.30549              1.62552              1.56789              2.02264              1.92439              2.04562              2.07865              1.39825              2.08875              1.84191              1.57631              2.19071              1.87229     1.84002     0.26155
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
<A NAME="08_01_09at11_38_25_comp_sum_numAll">&nbsp;</A>

-----------------
Number of alleles
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Locus#           2_Handumon          7_Santander            8_Boljoon              9_Argao            10_Carcar       11_Minglanilla             13_Danao             14_Sogod           15_Tabogon      16_Daanbantayan        17_Malapascua          18_Pintuyan       19_PadreBurgos            20_Maasin    22_InopacanIsland  22_InopacanMainland            23_Baybay           24_Albuera             25_Ormoc          27_Palompon        Mean        s.d. Tot. number
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
     1                    3                    4                    5                    6                    4                    3                    5                    4                    4                    3                    4                    4                    5                    4                    5                    4                    3                    3                    4                    4       4.050       0.826           7
     2                    6                    8                    7                   10                   10                    8                    9                    8                    8                    9                   11                    6                   10                    9                    6                    9                    8                    8                    7                    9       8.300       1.418          14
     3                    0                    4                    3                    4                    3                    4                    4                    5                    3                    3                    6                    3                    5                    5                    2                    4                    5                    5                    5                    5       3.900       1.373           8
     4                    3                    2                    3                    2                    3                    4                    4                    4                    4                    3                    4                    2                    2                    3                    3                    2                    2                    2                    3                    4       2.950       0.826           4
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Mean                3.000                4.500                4.500                5.500                5.000                4.750                5.500                5.250                4.750                4.500                6.250                3.750                5.500                5.250                4.000                4.750                4.500                4.500                4.750                5.500       4.800       0.724       8.250
  s.d.                2.449                2.517                1.915                3.416                3.367                2.217                2.380                1.893                2.217                3.000                3.304                1.708                3.317                2.630                1.826                2.986                2.646                2.646                1.708                2.380       2.526       0.564       3.631
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 1 (08/01/09 at 11:50:14))
Total computing time for this run : 0h 11m 48s 750 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>


<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>
<A NAME="08_01_09at12_58_07">&nbsp;</A>
////////////////////////////////////////////////////////////////////
RUN NUMBER 1 (08/01/09 at 12:58:07)
////////////////////////////////////////////////////////////////////

<A NAME="08_01_09at12_58_07_run_information">&nbsp;</A>

Project information:
--------------------
	NbSamples      =  20
	DataType       =  STANDARD
	GenotypicData  =  1
	GameticPhase   =  0
	RecessiveData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Infer haplotype definition from distance matrix
        Alllowed level of missing data = 0.2

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise differences
        GammaA Value       = 0.632
        Theta estimators   : 

    Linkage disequilibrium test:
    ----------------------------
        No. of permutations          = 10000
        No. of Initial conditions    = 2
        Print Histogram and table : No

    Hardy-Weinberg equilibrium test:
    --------------------------------
        No. of steps in Markov chain      = 1000000
        No. of Dememorisation Steps       = 100000
        Required precision on Probability = 0
        Test association at the Locus level

    Analysis of Molecular Variance:
    -------------------------------
        No. of Permutations = 1000

    Population pairwise Fst values:
    -------------------------------
        Compute pairwise differences
        No. of permutations for significance = 1000
        No. of permutations for Mantel test  = 500

       Distance matrix:
            Compute distance matrix
            Molecular distance : Pairwise differences
            Gamma a value      = 0

    Exact test of population differentiation:
    -----------------------------------------
        No. of steps in Markov chain      = 100000
        No. of Dememorisation Steps       = 10000
        Required precision on Probability = 0
        Print Histogram and table : Yes
        Significance level        = 0.05


    Correlation between distance matrices.
    --------------------------------------------------------

        No. of Permutations for significance = 500
<A NAME="08_01_09at12_58_07_gen_struct">&nbsp;</A>
===============================================================================
== GENETIC STRUCTURE ANALYSIS
===============================================================================


Number of usable loci for distance computation : 2
Allowed level of missing data                  : 0.2

List of usable loci :
---------------------
   1     2  

List of loci with too much missing data :
-----------------------------------------

   3     4  

<A NAME="08_01_09at12_58_07_amova">&nbsp;</A>

=================================================================
  AMOVA ANALYSIS
=================================================================


---------------------------
Genetic structure to test :
---------------------------

No. of Groups = 2


	[[Structure]]

		StructureName = "New Edited Structure"
		NbGroups = 2
		Group={
		"2_Handumon"
		"7_Santander"
		"8_Boljoon"
		"9_Argao"
		"10_Carcar"
		"11_Minglanilla"
		"13_Danao"
		"14_Sogod"
		"15_Tabogon"
		"16_Daanbantayan"
		"17_Malapascua"
		}

		Group={
		"18_Pintuyan"
		"19_PadreBurgos"
		"20_Maasin"
		"22_InopacanIsland"
		"22_InopacanMainland"
		"23_Baybay"
		"24_Albuera"
		"25_Ormoc"
		"27_Palompon"
		}

---------------------------


Distance method: Pairwise differences

--------------------------
AMOVA design and results :
--------------------------

Reference: <i>Weir, B.S. and Cockerham, C.C. 1984.</i>
           <i>Excoffier, L., Smouse, P., and Quattro, J. 1992.</i>
           <i>Weir, B. S., 1996.</i>
----------------------------------------------------------------------
 Source of                  Sum of      Variance         Percentage
 variation      d.f.        squares     components       of variation
----------------------------------------------------------------------
 Among
 groups           1          0.559       -0.00016 Va            -0.03

 Among
 populations
 within
 groups          18         11.045        0.00122 Vb             0.21

 Within
 populations    710        404.243        0.56936 Vc            99.81
----------------------------------------------------------------------
 Total          729        415.847        0.57042
----------------------------------------------------------------------
 Fixation Indices
      FST :      0.00186
      FSC :      0.00213
      FCT :     -0.00027
----------------------------------------------------------------------

Significance tests (1023 permutations)
------------------

Vc and FST : P(rand. value < obs. value)  =  0.13881
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.13881+-0.01113

Vb and FSC : P(rand. value > obs. value)  =  0.11926
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.11926+-0.01003

Va and FCT : P(rand. value > obs. value)  =  0.51515
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.51515+-0.01159

<A NAME="08_01_09at12_58_07_pairw_diff">&nbsp;</A>

======================================================================================
== Comparisons of pairs of population samples
======================================================================================

List of labels for population samples used below:
-------------------------------------------------

Label  	Population name
-----  	---------------
  1:	2_Handumon
  2:	7_Santander
  3:	8_Boljoon
  4:	9_Argao
  5:	10_Carcar
  6:	11_Minglanilla
  7:	13_Danao
  8:	14_Sogod
  9:	15_Tabogon
  10:	16_Daanbantayan
  11:	17_Malapascua
  12:	18_Pintuyan
  13:	19_PadreBurgos
  14:	20_Maasin
  15:	22_InopacanIsland
  16:	22_InopacanMainland
  17:	23_Baybay
  18:	24_Albuera
  19:	25_Ormoc
  20:	27_Palompon

------------------------
Population pairwise FSTs
------------------------


Distance method: Pairwise differences
                     1         2         3         4         5         6         7         8         9        10        11        12        13        14        15        16        17        18        19        20
           1   0.00000
           2   0.01611   0.00000
           3  -0.01656  -0.00488   0.00000
           4  -0.05577   0.00847  -0.00218   0.00000
           5  -0.03242  -0.00212  -0.01612   0.00324   0.00000
           6  -0.02248   0.00259  -0.01640   0.01000  -0.01951   0.00000
           7  -0.05306   0.00312   0.00131  -0.01344   0.00488   0.02010   0.00000
           8   0.01152   0.03515   0.00806   0.01502   0.01995   0.02729   0.01886   0.00000
           9  -0.04451  -0.00312  -0.01182  -0.00887  -0.01147  -0.00761  -0.00283   0.00452   0.00000
          10  -0.04055   0.01441  -0.01223   0.00048  -0.00442  -0.00770   0.01280  -0.00664  -0.01816   0.00000
          11  -0.07320   0.03692   0.01679  -0.01126   0.01475   0.02435   0.00347   0.05094   0.00866   0.01952   0.00000
          12  -0.05202   0.01207  -0.00454  -0.01698   0.01032   0.01232  -0.01874   0.01637  -0.01549  -0.00553  -0.00165   0.00000
          13  -0.04240   0.00438   0.00003  -0.02211  -0.00063   0.00894  -0.01598   0.02544  -0.01606  -0.00007  -0.01591  -0.01143   0.00000
          14  -0.03086  -0.00674   0.00474  -0.00815   0.00470   0.01601  -0.01648   0.01152  -0.01295   0.00040   0.01409  -0.02123  -0.01553   0.00000
          15  -0.01453  -0.00613  -0.00629  -0.00115   0.00238   0.01097  -0.00824  -0.00588  -0.01479  -0.00974   0.02563  -0.01603  -0.00693  -0.01923   0.00000
          16  -0.04157   0.03019  -0.00635  -0.01513  -0.01198   0.00367  -0.02039   0.00475  -0.01117  -0.01362  -0.01477  -0.00947  -0.00764   0.00162  -0.00121   0.00000
          17  -0.00871   0.05347   0.01606   0.00984  -0.01384   0.01153  -0.00500   0.04396   0.00572   0.00939  -0.00631   0.03028   0.01882   0.02538   0.02661  -0.01763   0.00000
          18   0.01211   0.04484  -0.00310   0.02095  -0.02145  -0.00465   0.00550   0.02133  -0.00040  -0.00556   0.02280   0.03377   0.03439   0.02631   0.01488  -0.01511  -0.01238   0.00000
          19  -0.04717   0.00158  -0.00978  -0.00465  -0.01713  -0.00811  -0.00883   0.02278  -0.01723  -0.00658   0.00311  -0.01225  -0.00887  -0.01046  -0.00580  -0.01388  -0.00861  -0.00896   0.00000
          20  -0.04359   0.04873   0.00288   0.02691   0.00168   0.00445   0.02829   0.02905   0.00760  -0.00227   0.03289   0.01767   0.03186   0.03187   0.02533  -0.01649  -0.01290  -0.02962  -0.00178   0.00000


------------
FST P values
------------

Number of permutations : 1023

                       1                 2                 3                 4                 5                 6                 7                 8                 9                10                11                12                13                14                15                16                17                18                19                20
           1           *
           2   0.26660+-0.0135           *
           3   0.57715+-0.0150   0.48926+-0.0146           *
           4   0.92969+-0.0071   0.27344+-0.0129   0.42773+-0.0157           *
           5   0.82910+-0.0134   0.41602+-0.0158   0.83984+-0.0124   0.30957+-0.0130           *
           6   0.66895+-0.0134   0.34570+-0.0175   0.82715+-0.0100   0.23340+-0.0119   0.91797+-0.0077           *
           7   0.95605+-0.0063   0.34766+-0.0140   0.37793+-0.0147   0.80273+-0.0123   0.25000+-0.0164   0.08984+-0.0093           *
           8   0.32031+-0.0133   0.04199+-0.0052   0.19922+-0.0108   0.15820+-0.0125   0.08398+-0.0097   0.06348+-0.0079   0.12988+-0.0089           *
           9   0.92090+-0.0075   0.41309+-0.0150   0.71680+-0.0145   0.62500+-0.0152   0.68652+-0.0172   0.49707+-0.0201   0.42773+-0.0197   0.27734+-0.0158           *
          10   0.87695+-0.0096   0.16895+-0.0116   0.71875+-0.0143   0.35352+-0.0157   0.52734+-0.0164   0.58887+-0.0149   0.16504+-0.0114   0.55762+-0.0177   0.85156+-0.0112           *
          11   0.97852+-0.0038   0.03516+-0.0045   0.12109+-0.0100   0.72266+-0.0166   0.12109+-0.0120   0.06055+-0.0082   0.31055+-0.0191   0.00488+-0.0025   0.14941+-0.0126   0.08691+-0.0095           *
          12   0.84473+-0.0123   0.21094+-0.0129   0.43652+-0.0165   0.71973+-0.0139   0.19531+-0.0137   0.18164+-0.0121   0.79883+-0.0084   0.16309+-0.0106   0.68555+-0.0152   0.45117+-0.0168   0.39648+-0.0160           *
          13   0.81445+-0.0106   0.31641+-0.0133   0.34473+-0.0152   0.88867+-0.0092   0.33301+-0.0131   0.21289+-0.0107   0.81152+-0.0119   0.07617+-0.0083   0.79883+-0.0119   0.37695+-0.0143   0.80078+-0.0122   0.58398+-0.0203           *
          14   0.78418+-0.0131   0.52539+-0.0184   0.22852+-0.0106   0.54590+-0.0188   0.26953+-0.0131   0.14258+-0.0115   0.85742+-0.0114   0.20996+-0.0120   0.69922+-0.0139   0.36133+-0.0137   0.14746+-0.0102   0.78711+-0.0109   0.72559+-0.0125           *
          15   0.59766+-0.0134   0.57715+-0.0131   0.54199+-0.0183   0.44141+-0.0132   0.34570+-0.0174   0.21680+-0.0125   0.63672+-0.0134   0.56445+-0.0165   0.80176+-0.0113   0.66211+-0.0159   0.05859+-0.0078   0.73340+-0.0128   0.54590+-0.0148   0.86133+-0.0114           *
          16   0.83887+-0.0108   0.08887+-0.0072   0.52148+-0.0161   0.76074+-0.0132   0.68750+-0.0142   0.35254+-0.0148   0.87988+-0.0096   0.35449+-0.0178   0.64160+-0.0157   0.69336+-0.0179   0.74609+-0.0119   0.54492+-0.0157   0.52734+-0.0167   0.35645+-0.0140   0.45020+-0.0135           *
          17   0.44141+-0.0165   0.01953+-0.0043   0.11328+-0.0111   0.20605+-0.0128   0.69141+-0.0181   0.19727+-0.0112   0.48047+-0.0153   0.02344+-0.0044   0.20996+-0.0102   0.21875+-0.0132   0.50000+-0.0183   0.08008+-0.0092   0.11230+-0.0124   0.08594+-0.0103   0.07520+-0.0076   0.75879+-0.0122           *
          18   0.19434+-0.0150   0.01855+-0.0044   0.37695+-0.0162   0.09473+-0.0089   0.89258+-0.0106   0.41504+-0.0148   0.25000+-0.0144   0.09375+-0.0081   0.28809+-0.0153   0.46094+-0.0137   0.05664+-0.0079   0.04199+-0.0050   0.04688+-0.0058   0.06738+-0.0083   0.12793+-0.0085   0.66504+-0.0118   0.56445+-0.0146           *
          19   0.91309+-0.0089   0.34961+-0.0156   0.65625+-0.0125   0.50684+-0.0138   0.88867+-0.0102   0.61035+-0.0153   0.64258+-0.0128   0.07715+-0.0093   0.88379+-0.0103   0.54395+-0.0152   0.32520+-0.0149   0.63770+-0.0171   0.59375+-0.0155   0.62891+-0.0127   0.54785+-0.0176   0.72656+-0.0126   0.53613+-0.0114   0.52441+-0.0159           *
          20   0.90234+-0.0093   0.00977+-0.0029   0.29004+-0.0140   0.05859+-0.0074   0.29590+-0.0138   0.26367+-0.0139   0.03711+-0.0059   0.05957+-0.0062   0.18359+-0.0122   0.43359+-0.0176   0.02637+-0.0048   0.11328+-0.0092   0.03906+-0.0060   0.04492+-0.0074   0.07031+-0.0077   0.78906+-0.0124   0.67090+-0.0178   0.93359+-0.0060   0.40234+-0.0163           *


------------
Matrix of significant Fst P values
Significance Level=0.0500
------------

Number of permutations : 1023

                     1         2         3         4         5         6         7         8         9        10        11        12        13        14        15        16        17        18        19        20
           1                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           2         -                   -         -         -         -         -         +         -         -         +         -         -         -         -         -         +         +         -         +
           3         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           4         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           5         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           6         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -
           7         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         +
           8         -         +         -         -         -         -         -                   -         -         +         -         -         -         -         -         +         -         -         -
           9         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -
          10         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -
          11         -         +         -         -         -         -         -         +         -         -                   -         -         -         -         -         -         -         -         +
          12         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         +         -         -
          13         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         +         -         +
          14         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         +
          15         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         -
          16         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -
          17         -         +         -         -         -         -         -         +         -         -         -         -         -         -         -         -                   -         -         -
          18         -         +         -         -         -         -         -         -         -         -         -         +         +         -         -         -         -                   -         -
          19         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -
          20         -         +         -         -         -         -         +         -         -         -         +         -         +         +         -         -         -         -         -          


---------------------------------------
Population average pairwise differences
----------------------------------------

Above diagonal    : Average number of pairwise differences between populations (PiXY)
Diagonal elements : Average number of pairwise differences within population (PiX)
Below diagonal    : Corrected average pairwise difference (PiXY-(PiX+PiY)/2)

Distance method: Pairwise differences
                       1          2          3          4          5          6          7          8          9         10         11         12         13         14         15         16         17         18         19         20
            1    0.86813    0.95714    1.00536    0.97857    1.01128    0.96992    1.01429    1.04762    1.03214    0.99436    0.98571    0.88810    0.91429    1.04622    1.07707    0.94898    0.88690    0.88158    0.99286    0.96230
            2    0.01923    1.00769    1.08062    1.10875    1.10592    1.05987    1.13688    1.13690    1.13875    1.11579    1.16813    1.01500    1.02500    1.13382    1.14671    1.08750    1.01806    0.98487    1.10688    1.12500
            3   -0.01076   -0.00527    1.16410    1.17500    1.16776    1.11711    1.21313    1.18452    1.20625    1.16382    1.22375    1.07667    1.09875    1.22574    1.22434    1.12679    1.05903    1.01579    1.17188    1.15208
            4   -0.05101    0.00939   -0.00256    1.19103    1.20395    1.16053    1.20875    1.20655    1.22313    1.19211    1.20312    1.07667    1.08813    1.22353    1.24408    1.13036    1.06597    1.05461    1.19125    1.19444
            5   -0.02734   -0.00248   -0.01884    0.00388    1.20910    1.13573    1.24013    1.22180    1.22895    1.19529    1.24408    1.11579    1.12039    1.24845    1.25762    1.14286    1.04971    1.01939    1.18553    1.17325
            6   -0.01748    0.00268   -0.01829    0.01167   -0.02216    1.10669    1.20724    1.17857    1.18289    1.14058    1.20395    1.06579    1.07961    1.21053    1.21676    1.10902    1.02485    0.98546    1.14539    1.12500
            7   -0.04927    0.00354    0.00159   -0.01625    0.00609    0.02441    1.25897    1.24583    1.26438    1.24145    1.25500    1.10833    1.12813    1.24706    1.26908    1.15804    1.08403    1.07237    1.22000    1.23125
            8    0.02064    0.04014    0.00956    0.01812    0.02434    0.03231    0.02343    1.18583    1.23690    1.18108    1.27917    1.11111    1.13869    1.24510    1.23559    1.15051    1.10185    1.05263    1.22202    1.19444
            9   -0.04038   -0.00356   -0.01426   -0.01085   -0.01407   -0.00891   -0.00357    0.00553    1.27692    1.21250    1.27062    1.12083    1.13688    1.26029    1.26974    1.17768    1.10486    1.07500    1.21875    1.21458
           10   -0.03572    0.01593   -0.01425    0.00058   -0.00528   -0.00878    0.01594   -0.00785   -0.02198    1.19203    1.24145    1.08947    1.11250    1.23452    1.23407    1.13252    1.06579    1.02701    1.18947    1.16009
           11   -0.06951    0.04313    0.02054   -0.01354    0.01837    0.02945    0.00436    0.06510    0.01101    0.02428    1.24231    1.11917    1.12000    1.27721    1.30461    1.15625    1.07431    1.08289    1.22625    1.22847
           12   -0.04482    0.01230   -0.00424   -0.01770    0.01239    0.01360   -0.02000    0.01935   -0.01648   -0.00539   -0.00084    0.99770    1.00417    1.11373    1.13158    1.04048    0.98796    0.96754    1.08750    1.08519
           13   -0.03645    0.00449    0.00003   -0.02405   -0.00082    0.00960   -0.01803    0.02911   -0.01825   -0.00018   -0.01782   -0.01135    1.03333    1.13676    1.15855    1.05982    0.99514    0.98750    1.10813    1.11875
           14   -0.02599   -0.00817    0.00554   -0.01013    0.00575    0.01904   -0.02057    0.01404   -0.01631    0.00036    0.01791   -0.02327   -0.01805    1.27629    1.26393    1.19223    1.12582    1.10294    1.22647    1.24428
           15   -0.00707   -0.00721   -0.00778   -0.00150    0.00299    0.01335   -0.01048   -0.00740   -0.01880   -0.01202    0.03338   -0.01734   -0.00819   -0.02429    1.30014    1.20113    1.14035    1.10319    1.24408    1.24854
           16   -0.03667    0.03207   -0.00685   -0.01674   -0.01328    0.00409   -0.02304    0.00601   -0.01237   -0.01509   -0.01649   -0.00996   -0.00843    0.00249   -0.00053    1.10317    0.99306    0.97274    1.13750    1.10020
           17   -0.00668    0.05469    0.01745    0.01094   -0.01437    0.01199   -0.00498    0.04941    0.00688    0.01025   -0.00637    0.02959    0.01895    0.02815    0.03076   -0.01806    0.91905    0.88523    1.05208    1.01312
           18    0.01081    0.04432   -0.00296    0.02239   -0.02186   -0.00459    0.00618    0.02302   -0.00016   -0.00571    0.02504    0.03199    0.03413    0.02810    0.01641   -0.01554   -0.01099    0.87340    1.02895    0.97442
           19   -0.04249    0.00175   -0.01146   -0.00554   -0.02031   -0.00923   -0.01077    0.02783   -0.02099   -0.00783    0.00381   -0.01263   -0.00982   -0.01296   -0.00727   -0.01537   -0.00872   -0.00903    1.20256    1.16597
           20   -0.03843    0.05449    0.00337    0.03226    0.00203    0.00499    0.03510    0.03486    0.00946   -0.00260    0.04065    0.01967    0.03542    0.03947    0.03180   -0.01806   -0.01307   -0.02895   -0.00198    1.13333


-----------
PXY P value
-----------

                         1           2           3           4           5           6           7           8           9          10          11          12          13          14          15          16          17          18          19

             2     0.63636
             3     0.89150     0.48974
             4     0.90029     0.27370     0.42815
             5     0.92766     0.34897     0.83675     0.31476
             6     0.84164     0.32454     0.86608     0.27468     0.91887
             7     0.94233     0.34800     0.37830     0.80352     0.29130     0.12219
             8     0.80841     0.06256     0.22483     0.17498     0.09971     0.09873     0.12708
             9     0.95112     0.41349     0.71750     0.62561     0.76344     0.66276     0.42815     0.26491
            10     0.88172     0.14663     0.71652     0.38416     0.52786     0.58944     0.19941     0.60508     0.89638
            11     0.95699     0.03519     0.12121     0.72336     0.13294     0.08113     0.31085     0.00293     0.14956     0.09775
            12     0.78104     0.39198     0.77322     0.82991     0.66862     0.45650     0.90029     0.61681     0.90909     0.73998     0.79374
            13     0.77419     0.31672     0.34506     0.88954     0.26588     0.19550     0.81232     0.10068     0.79961     0.31574     0.80156     0.65982
            14     0.91887     0.23558     0.20039     0.47507     0.24145     0.10850     0.79472     0.18964     0.70283     0.33333     0.16618     0.91007     0.40860
            15     0.93255     0.45161     0.47312     0.40078     0.34604     0.21701     0.63734     0.47898     0.78495     0.66276     0.05767     0.91593     0.41056     0.87879
            16     0.84360     0.15543     0.68133     0.75073     0.80743     0.44966     0.87097     0.59629     0.82991     0.74878     0.83969     0.52590     0.48583     0.65396     0.78690
            17     0.58260     0.03226     0.28152     0.46237     0.88661     0.28250     0.84848     0.10362     0.66667     0.33822     0.86999     0.08309     0.18768     0.05865     0.13392     0.41740
            18     0.45943     0.02346     0.58260     0.15934     0.89345     0.41544     0.43500     0.27370     0.55621     0.46139     0.11730     0.04692     0.05963     0.02737     0.12805     0.31183     0.56794
            19     0.92766     0.34995     0.65689     0.50733     0.88954     0.68133     0.64321     0.07918     0.88465     0.55816     0.32551     0.84164     0.59433     0.53959     0.51124     0.75073     0.87781     0.78788
            20     0.88074     0.00880     0.34897     0.08407     0.36266     0.27566     0.07429     0.11144     0.33724     0.48485     0.05670     0.36266     0.03421     0.03812     0.09677     0.75073     0.67155     0.84946     0.53275


---------------------
Corrected PXY P value
---------------------

                           1            2            3            4            5            6            7            8            9           10           11           12           13           14           15           16           17           18           19

              2      0.24536
              3      0.50049      0.48778
              4      0.90029      0.26979      0.42815
              5      0.71457      0.42229      0.84066      0.30987
              6      0.57185      0.34506      0.82698      0.23265      0.91691
              7      0.92571      0.34213      0.37830      0.79374      0.24927      0.08993
              8      0.26002      0.04301      0.20039      0.15738      0.08309      0.06354      0.13001
              9      0.81134      0.40078      0.70870      0.60411      0.68328      0.49462      0.41251      0.27664
             10      0.81134      0.17204      0.71848      0.35484      0.52102      0.58749      0.16325      0.55718      0.85239
             11      0.97752      0.03519      0.12121      0.72043      0.12121      0.05963      0.30694      0.00489      0.14467      0.08700
             12      0.82209      0.21212      0.41642      0.70186      0.18475      0.17595      0.77322      0.16129      0.65005      0.43402      0.36559
             13      0.79179      0.31574      0.34506      0.88954      0.33529      0.21310      0.81036      0.07625      0.79668      0.38025      0.80156      0.57478
             14      0.67546      0.54154      0.22874      0.55425      0.27077      0.14761      0.85924      0.21505      0.70479      0.36755      0.14761      0.77908      0.73607
             15      0.49071      0.58456      0.54643      0.44282      0.34018      0.21408      0.63734      0.56696      0.80352      0.65689      0.05865      0.71554      0.55230      0.85924
             16      0.79472      0.09384      0.51026      0.74780      0.67644      0.35875      0.86608      0.33724      0.62463      0.68622      0.73705      0.54741      0.53861      0.35288      0.43109
             17      0.43402      0.01955      0.11339      0.20332      0.68328      0.19257      0.46334      0.02346      0.20235      0.21310      0.49169      0.08016      0.11144      0.08993      0.07527      0.76637
             18      0.19550      0.01760      0.36950      0.09384      0.88759      0.41251      0.24633      0.09189      0.28055      0.44673      0.05572      0.04301      0.04594      0.06843      0.12317      0.68426      0.56403
             19      0.85924      0.33627      0.65200      0.49071      0.88759      0.60802      0.62952      0.07722      0.86901      0.54350      0.31965      0.61877      0.59238      0.63245      0.54936      0.71457      0.52395      0.52004
             20      0.84848      0.00978      0.28739      0.05963      0.29326      0.26393      0.03715      0.05865      0.18084      0.43402      0.02542      0.11046      0.04008      0.04497      0.07038      0.78201      0.64809      0.93353      0.39980


********************
Locus by locus AMOVA:
********************

Distance method for locus-by-locus analysis: Pairwise differences

List of loci with only one allele:
----------------------------------


<A NAME="08_01_09at12_58_07Loc_by_Loc_AMOVA">&nbsp;</A>

----------------------------------------
AMOVA Results for polymorphic loci only:
----------------------------------------

            Among Groups:                                      Among Population:                                    Within Populations:                                 Fixation indices:
  Locus            SSD   d.f.             Va    % variation            SSD   d.f.             Vb    % variation            SSD   d.f.             Vc    % variation       FSC   P-value       FST   P-value       FCT   P-value
      1        0.34838      1        0.00007        0.02312        5.83331     18        0.00088        0.29793      201.14975    686        0.29322       99.67896   0.00298   0.30499   0.00321   0.29228   0.00023   0.35484
      2        0.36574      1       -0.00021       -0.05277        7.72676     18        0.00107        0.26920      241.41861    610        0.39577       99.78357   0.00269   0.26588   0.00216   0.28348  -0.00053   0.47214
      4        0.19179      1        0.00033        0.12103        2.27608     18       -0.00616       -2.26711      132.07850    476        0.27748      102.14608  -0.02270   0.99804  -0.02146   0.99511   0.00121   0.17302

====================================================
Global AMOVA results as a weighted average over loci
====================================================

----------------------------------------------------
AMOVA design and results (average over 3 loci):
----------------------------------------------------

------------------------------------------------------------------------
 Source of               Sum of           Variance            Percentage
 variation               squares          components          variation
------------------------------------------------------------------------
 Among
 groups                    0.906             0.00019             0.01948

 Among
 populations
 within
 groups                   15.836            -0.00421            -0.43789

 Within
 populations             574.647             0.96647           100.41841
------------------------------------------------------------------------
 Total                   591.389             0.96244
------------------------------------------------------------------------


Average F-Statistics over all loci
 Fixation Indices
      FST :     -0.00418
      FSC :     -0.00438
      FCT :      0.00019

------------------
Significance tests (1023 permutations)
------------------

Vc and FST : P(rand. value < obs. value)  =  0.90323
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.90323

Vb and FSC : P(rand. value > obs. value)  =  0.90518
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.90518

Va and FCT : P(rand. value > obs. value)  =  0.37634
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.37634


<A NAME="08_01_09at12_58_07_exct_tests">&nbsp;</A>

================================================================================
== Exact Test of Sample Differentiation Based on Haplotype Frequencies
================================================================================
Reference: <i>Raymond M. and F. Rousset. 1995 .</i>
           <i>Goudet, J., M. Raymond, T. de Meeüs and F. Rousset, 1996.</i>

List of labels for population samples used below:
-------------------------------------------------

Label  	Population name
-----  	---------------
  1	2_Handumon
  2	7_Santander
  3	8_Boljoon
  4	9_Argao
  5	10_Carcar
  6	11_Minglanilla
  7	13_Danao
  8	14_Sogod
  9	15_Tabogon
  10	16_Daanbantayan
  11	17_Malapascua
  12	18_Pintuyan
  13	19_PadreBurgos
  14	20_Maasin
  15	22_InopacanIsland
  16	22_InopacanMainland
  17	23_Baybay
  18	24_Albuera
  19	25_Ormoc
  20	27_Palompon


--------------------------------------------------------------------------
Global test of differentiation among sample:
--------------------------------------------

 Non-differentiation: Exact P value = 0.43016 +- 0.13274  (100000 Markov steps done)

--------------------------------------------------------------------------
Differentiation test between all pairs of samples:
--------------------------------------------------

Markov chain length : 100000 steps)

Non-differentiation exact P values :

                1                 2                 3                 4                 5                 6                 7                 8                 9                10                11                12                13                14                15                16                17                18                19

           2   1.00000+-0.0000
           3   1.00000+-0.0000   1.00000+-0.0000
           4   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000
           5   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000
           6   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000
           7   0.58795+-0.0093   0.47775+-0.0135   0.47448+-0.0220   0.49217+-0.0086   0.72030+-0.0103   0.51012+-0.0172
           8   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.47998+-0.0147
           9   0.62358+-0.0140   0.73199+-0.0152   0.47627+-0.0177   0.49199+-0.0155   0.47957+-0.0140   0.73484+-0.0133   0.26116+-0.0097   0.74186+-0.0081
          10   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.49032+-0.0147   1.00000+-0.0000   0.48163+-0.0147
          11   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.50105+-0.0162   1.00000+-0.0000   0.49307+-0.0174   1.00000+-0.0000
          12   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.49371+-0.0108   1.00000+-0.0000   0.73257+-0.0069   1.00000+-0.0000   1.00000+-0.0000
          13   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.48157+-0.0084   1.00000+-0.0000   0.72493+-0.0135   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000
          14   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.74674+-0.0105   1.00000+-0.0000   0.49618+-0.0139   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000
          15   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.47181+-0.0177   1.00000+-0.0000   0.47959+-0.0175   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000
          16   0.53225+-0.0122   0.49896+-0.0136   0.51125+-0.0104   1.00000+-0.0000   0.46039+-0.0116   0.49741+-0.0131   0.81399+-0.0068   1.00000+-0.0000   0.26114+-0.0087   0.49014+-0.0154   0.49133+-0.0113   0.50881+-0.0114   0.49033+-0.0139   0.75464+-0.0064   0.51166+-0.0132
          17   0.18419+-0.0064   0.29077+-0.0098   0.11144+-0.0074   0.34750+-0.0148   0.34012+-0.0149   0.12008+-0.0079   0.17340+-0.0152   0.39278+-0.0113   0.17030+-0.0094   0.12349+-0.0063   0.10766+-0.0065   0.11177+-0.0087   0.12199+-0.0070   0.12591+-0.0060   0.46661+-0.0138   0.32681+-0.0102
          18   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.50026+-0.0115   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.74091+-0.0102   0.11662+-0.0071
          19   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.86194+-0.0134   1.00000+-0.0000   0.47794+-0.0154   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.75614+-0.0116   0.10719+-0.0079   1.00000+-0.0000
          20   0.56706+-0.0076   0.50266+-0.0121   0.73000+-0.0114   0.48647+-0.0112   0.48682+-0.0158   0.47098+-0.0123   0.61812+-0.0086   0.48946+-0.0096   0.22303+-0.0086   0.47335+-0.0155   0.48812+-0.0155   0.48708+-0.0106   0.74330+-0.0106   0.46262+-0.0139   0.50803+-0.0084   0.23437+-0.0078   0.04796+-0.0025   0.47962+-0.0125   0.48813+-0.0211


--------------------------------
Table of significant differences (significance level=0.0500):
--------------------------------

                     1         2         3         4         5         6         7         8         9        10        11        12        13        14        15        16        17        18        19        20
           1                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           2         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           3         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           4         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           5         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           6         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -
           7         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -
           8         -         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -
           9         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -
          10         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -
          11         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -
          12         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -         -
          13         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -
          14         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         -
          15         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         -
          16         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -
          17         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         +
          18         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -
          19         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -
          20         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         +         -         -          


------------------------------------------------------------
Histogram of the number of significant different populations (significance level=0.0500):
------------------------------------------------------------

------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
|         1|         2|         3|         4|         5|         6|         7|         8|         9|        10|        11|        12|        13|        14|        15|        16|        17|        18|        19|        20|
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         1|         0|         0|         1|
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="08_01_09at12_58_07_samp0">&nbsp;</A>

===============================================================================
== Sample : 	2_Handumon
===============================================================================


===============================
== Standard diversity indices : (2_Handumon)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 14
No. of loci              : 4
No. of usable loci       : 1 loci with less than 20.00 % missing data
No. of polymorphic loci  : 1


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        14         3   0.57143   0.58242
     2         8         6   0.75000   0.92857
     4         4         3   1.00000   0.83333
-------------------------------------------------
  Mean     8.667     4.000   0.77381   0.78144
  s.d.     5.033     1.732   0.21528   0.17882
-------------------------------------------------



================================
== Molecular diversity indices : (2_Handumon)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 14.00000
No. of haplotypes                            : 13

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 1
Number of usable loci                        : 1


Average gene diversity over loci              :     0.582418 +/-     0.557654

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (2_Handumon)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -25.50310          LnLHood LE :  -30.83571
          Exact P=  0.55832  +-   0.00518 (10100 permutations done)     Chi-square test value=10.66522 (P =  0.55781,  12 d.f.)
Pair(0, 2)
          LnLHood LD :    0.00000          LnLHood LE :    0.00000
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=10.66522 (P =  0.55781,  12 d.f.)
Pair(1, 2)
          LnLHood LD :    0.00000          LnLHood LE :    0.00000
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=10.66522 (P =  0.55781,  12 d.f.)
Pair(0, 3)
          LnLHood LD :  -17.08711          LnLHood LE :  -20.63985
          Exact P=  0.28257  +-   0.00410 (10100 permutations done)     Chi-square test value= 7.10550 (P =  0.31120,  6 d.f.)
Pair(1, 3)
          LnLHood LD :  -27.93589          LnLHood LE :  -32.49258
          Exact P=  0.87139  +-   0.00338 (10100 permutations done)     Chi-square test value= 9.11338 (P =  0.95706,  18 d.f.)
Pair(2, 3)
          LnLHood LD :    0.00000          LnLHood LE :    0.00000
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value= 9.11338 (P =  0.95706,  18 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (2_Handumon)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1       7       0.57143      0.58242    1.00000  0.00000     1001000
    2       4       0.75000      0.92857    0.30523  0.00038     1001000
    3   This locus is monomorphic: no test done.
    4       2       1.00000      0.83333    1.00000  0.00000     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp1">&nbsp;</A>

===============================================================================
== Sample : 	7_Santander
===============================================================================


===============================
== Standard diversity indices : (7_Santander)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 40
No. of loci              : 4
No. of usable loci       : 2 loci with less than 20.00 % missing data
No. of polymorphic loci  : 2


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        36         4   0.55556   0.63016
     2        34         8   0.64706   0.69340
     3        14         4   0.71429   0.70330
     4         6         2   0.00000   0.53333
-------------------------------------------------
  Mean    22.500     4.500   0.47923   0.64005
  s.d.    14.821     2.517   0.32604   0.07817
-------------------------------------------------



================================
== Molecular diversity indices : (7_Santander)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 40.00000
No. of haplotypes                            : 29

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 2
Number of usable loci                        : 2


Average gene diversity over loci              :     0.503846 +/-     0.383949

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (7_Santander)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -89.56542          LnLHood LE : -105.27950
          Exact P=  0.16347  +-   0.00351 (10100 permutations done)     Chi-square test value=31.42816 (P =  0.49534,  32 d.f.)
Pair(0, 2)
          LnLHood LD :  -72.69964          LnLHood LE :  -83.58781
          Exact P=  0.04287  +-   0.00202 (10100 permutations done)     Chi-square test value=21.77634 (P =  0.15056,  16 d.f.)
Pair(1, 2)
          LnLHood LD :  -89.71997          LnLHood LE : -100.02346
          Exact P=  0.73960  +-   0.00415 (10100 permutations done)     Chi-square test value=20.60698 (P =  0.93991,  32 d.f.)
Pair(0, 3)
          LnLHood LD :  -61.72786          LnLHood LE :  -65.14937
          Exact P=  0.48267  +-   0.00551 (10100 permutations done)     Chi-square test value= 6.84301 (P =  0.55366,  8 d.f.)
Pair(1, 3)
          LnLHood LD :  -72.69761          LnLHood LE :  -81.58502
          Exact P=  0.12604  +-   0.00326 (10100 permutations done)     Chi-square test value=17.77483 (P =  0.33725,  16 d.f.)
Pair(2, 3)
          LnLHood LD :  -57.03028          LnLHood LE :  -59.89334
          Exact P=  0.50050  +-   0.00485 (10100 permutations done)     Chi-square test value= 5.72611 (P =  0.67788,  8 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           1    0    1    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   +   - 
       1 |  -   *   -   - 
       2 |  +   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (7_Santander)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      18       0.55556      0.63016    0.47132  0.00045     1001000
    2      17       0.64706      0.69340    0.59563  0.00048     1001000
    3       7       0.71429      0.70330    0.83133  0.00035     1001000
    4       3       0.00000      0.53333    0.19966  0.00041     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp2">&nbsp;</A>

===============================================================================
== Sample : 	8_Boljoon
===============================================================================


===============================
== Standard diversity indices : (8_Boljoon)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 40
No. of loci              : 4
No. of usable loci       : 2 loci with less than 20.00 % missing data
No. of polymorphic loci  : 2


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        40         5   0.75000   0.65000
     2        34         7   0.64706   0.71480
     3        18         3   0.66667   0.67974
     4        30         3   0.33333   0.54483
-------------------------------------------------
  Mean    30.500     4.500   0.59926   0.64734
  s.d.     9.292     1.915   0.18282   0.07329
-------------------------------------------------



================================
== Molecular diversity indices : (8_Boljoon)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 40.00000
No. of haplotypes                            : 34

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 2
Number of usable loci                        : 2


Average gene diversity over loci              :     0.582051 +/-     0.425325

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (8_Boljoon)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -88.68121          LnLHood LE :  -98.76725
          Exact P=  0.82891  +-   0.00385 (10100 permutations done)     Chi-square test value=20.17207 (P =  0.85811,  28 d.f.)
Pair(0, 2)
          LnLHood LD :  -74.00422          LnLHood LE :  -80.99048
          Exact P=  0.35723  +-   0.00480 (10100 permutations done)     Chi-square test value=13.97252 (P =  0.30247,  12 d.f.)
Pair(1, 2)
          LnLHood LD :  -93.40693          LnLHood LE : -102.70096
          Exact P=  0.71327  +-   0.00438 (10100 permutations done)     Chi-square test value=18.58807 (P =  0.61155,  21 d.f.)
Pair(0, 3)
          LnLHood LD :  -76.55912          LnLHood LE :  -81.88622
          Exact P=  0.63604  +-   0.00506 (10100 permutations done)     Chi-square test value=10.65419 (P =  0.55876,  12 d.f.)
Pair(1, 3)
          LnLHood LD :  -93.54782          LnLHood LE : -103.59670
          Exact P=  0.50139  +-   0.00463 (10100 permutations done)     Chi-square test value=20.09775 (P =  0.51508,  21 d.f.)
Pair(2, 3)
          LnLHood LD :  -76.98586          LnLHood LE :  -85.81992
          Exact P=  0.03832  +-   0.00206 (10100 permutations done)     Chi-square test value=17.66812 (P =  0.03923,  9 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    1    1
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   + 
       3 |  -   -   +   * 

===============================
== Hardy-Weinberg equilibrium : (8_Boljoon)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.75000      0.65000    0.49229  0.00047     1001000
    2      17       0.64706      0.71480    0.10930  0.00030     1001000
    3       9       0.66667      0.67974    1.00000  0.00000     1001000
    4      15       0.33333      0.54483    0.11452  0.00034     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp3">&nbsp;</A>

===============================================================================
== Sample : 	9_Argao
===============================================================================


===============================
== Standard diversity indices : (9_Argao)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 40
No. of loci              : 4
No. of usable loci       : 2 loci with less than 20.00 % missing data
No. of polymorphic loci  : 2


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        38         6   0.52632   0.64154
     2        34        10   0.88235   0.85205
     3        16         4   0.87500   0.71667
     4        28         2   0.71429   0.51852
-------------------------------------------------
  Mean    29.000     5.500   0.74949   0.68219
  s.d.     9.592     3.416   0.16778   0.13962
-------------------------------------------------



================================
== Molecular diversity indices : (9_Argao)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 40.00000
No. of haplotypes                            : 37

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 2
Number of usable loci                        : 2


Average gene diversity over loci              :     0.595513 +/-     0.432380

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (9_Argao)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD : -101.43789          LnLHood LE : -120.38793
          Exact P=  0.58901  +-   0.00524 (10100 permutations done)     Chi-square test value=37.90007 (P =  0.98853,  60 d.f.)
Pair(0, 2)
          LnLHood LD :  -77.89147          LnLHood LE :  -88.11835
          Exact P=  0.29703  +-   0.00482 (10100 permutations done)     Chi-square test value=20.45377 (P =  0.67070,  24 d.f.)
Pair(1, 2)
          LnLHood LD : -102.55873          LnLHood LE : -115.20049
          Exact P=  0.90713  +-   0.00298 (10100 permutations done)     Chi-square test value=25.28352 (P =  0.96628,  40 d.f.)
Pair(0, 3)
          LnLHood LD :  -76.52941          LnLHood LE :  -83.56411
          Exact P=  0.30248  +-   0.00439 (10100 permutations done)     Chi-square test value=14.06941 (P =  0.29630,  12 d.f.)
Pair(1, 3)
          LnLHood LD :  -97.21246          LnLHood LE : -110.64626
          Exact P=  0.25149  +-   0.00457 (10100 permutations done)     Chi-square test value=26.86759 (P =  0.13903,  20 d.f.)
Pair(2, 3)
          LnLHood LD :  -71.24463          LnLHood LE :  -78.37668
          Exact P=  0.08950  +-   0.00268 (10100 permutations done)     Chi-square test value=14.26410 (P =  0.07514,  8 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (9_Argao)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      19       0.52632      0.64154    0.34830  0.00048     1001000
    2      17       0.88235      0.85205    0.73573  0.00032     1001000
    3       8       0.87500      0.71667    0.88675  0.00032     1001000
    4      14       0.71429      0.51852    0.27988  0.00045     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp4">&nbsp;</A>

===============================================================================
== Sample : 	10_Carcar
===============================================================================


===============================
== Standard diversity indices : (10_Carcar)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 38
No. of loci              : 4
No. of usable loci       : 2 loci with less than 20.00 % missing data
No. of polymorphic loci  : 2


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        38         4   0.57895   0.52774
     2        36        10   0.77778   0.76032
     3        16         3   0.75000   0.70000
     4        26         3   0.46154   0.55077
-------------------------------------------------
  Mean    29.000     5.000   0.64207   0.63471
  s.d.    10.132     3.367   0.14904   0.11333
-------------------------------------------------



================================
== Molecular diversity indices : (10_Carcar)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 38.00000
No. of haplotypes                            : 30

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 2
Number of usable loci                        : 2


Average gene diversity over loci              :     0.604552 +/-     0.437678

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (10_Carcar)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -80.79627          LnLHood LE :  -89.24478
          Exact P=  0.91158  +-   0.00294 (10100 permutations done)     Chi-square test value=16.89701 (P =  0.97379,  30 d.f.)
Pair(0, 2)
          LnLHood LD :  -60.50896          LnLHood LE :  -65.58739
          Exact P=  0.29327  +-   0.00460 (10100 permutations done)     Chi-square test value=10.15685 (P =  0.33794,  9 d.f.)
Pair(1, 2)
          LnLHood LD :  -87.98623          LnLHood LE : -101.69772
          Exact P=  0.45406  +-   0.00432 (10100 permutations done)     Chi-square test value=27.42298 (P =  0.60099,  30 d.f.)
Pair(0, 3)
          LnLHood LD :  -63.81363          LnLHood LE :  -67.49408
          Exact P=  0.59485  +-   0.00553 (10100 permutations done)     Chi-square test value= 7.36090 (P =  0.59960,  9 d.f.)
Pair(1, 3)
          LnLHood LD :  -85.09942          LnLHood LE : -103.60442
          Exact P=  0.02743  +-   0.00146 (10100 permutations done)     Chi-square test value=37.01000 (P =  0.17685,  30 d.f.)
Pair(2, 3)
          LnLHood LD :  -71.11511          LnLHood LE :  -79.94703
          Exact P=  0.03752  +-   0.00207 (10100 permutations done)     Chi-square test value=17.66384 (P =  0.03928,  9 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    1    1    2
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   + 
       2 |  -   -   *   + 
       3 |  -   +   +   * 

===============================
== Hardy-Weinberg equilibrium : (10_Carcar)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      19       0.57895      0.52774    0.57686  0.00044     1001000
    2      18       0.77778      0.76032    0.76770  0.00044     1001000
    3       8       0.75000      0.70000    0.17389  0.00037     1001000
    4      13       0.46154      0.55077    0.77253  0.00042     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp5">&nbsp;</A>

===============================================================================
== Sample : 	11_Minglanilla
===============================================================================


===============================
== Standard diversity indices : (11_Minglanilla)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 38
No. of loci              : 4
No. of usable loci       : 2 loci with less than 20.00 % missing data
No. of polymorphic loci  : 2


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        38         3   0.47368   0.52774
     2        34         8   0.76471   0.72549
     3        12         4   0.66667   0.69697
     4        28         4   0.50000   0.61905
-------------------------------------------------
  Mean    28.000     4.750   0.60126   0.64231
  s.d.    11.431     2.217   0.13847   0.08865
-------------------------------------------------



================================
== Molecular diversity indices : (11_Minglanilla)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 38.00000
No. of haplotypes                            : 29

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 2
Number of usable loci                        : 2


Average gene diversity over loci              :     0.553343 +/-     0.410753

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (11_Minglanilla)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -75.52434          LnLHood LE :  -83.23905
          Exact P=  0.48208  +-   0.00480 (10100 permutations done)     Chi-square test value=15.42941 (P =  0.49344,  16 d.f.)
Pair(0, 2)
          LnLHood LD :  -55.61403          LnLHood LE :  -59.79374
          Exact P=  0.28366  +-   0.00416 (10100 permutations done)     Chi-square test value= 8.35943 (P =  0.39917,  8 d.f.)
Pair(1, 2)
          LnLHood LD :  -83.13420          LnLHood LE :  -92.74690
          Exact P=  0.68495  +-   0.00444 (10100 permutations done)     Chi-square test value=19.22539 (P =  0.96344,  32 d.f.)
Pair(0, 3)
          LnLHood LD :  -67.42742          LnLHood LE :  -71.80173
          Exact P=  0.37079  +-   0.00496 (10100 permutations done)     Chi-square test value= 8.74861 (P =  0.36395,  8 d.f.)
Pair(1, 3)
          LnLHood LD :  -87.83884          LnLHood LE : -104.75489
          Exact P=  0.16149  +-   0.00365 (10100 permutations done)     Chi-square test value=33.83210 (P =  0.37910,  32 d.f.)
Pair(2, 3)
          LnLHood LD :  -76.07305          LnLHood LE :  -81.30958
          Exact P=  0.84455  +-   0.00348 (10100 permutations done)     Chi-square test value=10.47306 (P =  0.84079,  16 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (11_Minglanilla)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      19       0.47368      0.52774    0.66348  0.00048     1001000
    2      17       0.76471      0.72549    0.74589  0.00035     1001000
    3       6       0.66667      0.69697    0.25018  0.00043     1001000
    4      14       0.50000      0.61905    0.61634  0.00044     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp6">&nbsp;</A>

===============================================================================
== Sample : 	13_Danao
===============================================================================


===============================
== Standard diversity indices : (13_Danao)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 40
No. of loci              : 4
No. of usable loci       : 2 loci with less than 20.00 % missing data
No. of polymorphic loci  : 2


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        38         5   0.63158   0.61735
     2        38         9   0.73684   0.77952
     3        12         4   0.50000   0.45455
     4        26         4   0.38462   0.58154
-------------------------------------------------
  Mean    28.500     5.500   0.56326   0.60824
  s.d.    12.369     2.380   0.15353   0.13386
-------------------------------------------------



================================
== Molecular diversity indices : (13_Danao)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 40.00000
No. of haplotypes                            : 33

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 2
Number of usable loci                        : 2


Average gene diversity over loci              :     0.629487 +/-     0.450112

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (13_Danao)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -88.61498          LnLHood LE : -104.59314
          Exact P=  0.37139  +-   0.00516 (10100 permutations done)     Chi-square test value=31.95631 (P =  0.92838,  45 d.f.)
Pair(0, 2)
          LnLHood LD :  -70.46820          LnLHood LE :  -75.25616
          Exact P=  0.89406  +-   0.00311 (10100 permutations done)     Chi-square test value= 9.57591 (P =  0.97523,  20 d.f.)
Pair(1, 2)
          LnLHood LD :  -81.39742          LnLHood LE :  -94.13496
          Exact P=  0.18564  +-   0.00379 (10100 permutations done)     Chi-square test value=25.47508 (P =  0.90431,  36 d.f.)
Pair(0, 3)
          LnLHood LD :  -82.72314          LnLHood LE :  -90.02715
          Exact P=  0.72089  +-   0.00404 (10100 permutations done)     Chi-square test value=14.60802 (P =  0.79838,  20 d.f.)
Pair(1, 3)
          LnLHood LD :  -95.62401          LnLHood LE : -108.90595
          Exact P=  0.54911  +-   0.00438 (10100 permutations done)     Chi-square test value=26.56389 (P =  0.87430,  36 d.f.)
Pair(2, 3)
          LnLHood LD :  -74.15515          LnLHood LE :  -79.56898
          Exact P=  0.59129  +-   0.00499 (10100 permutations done)     Chi-square test value=10.82764 (P =  0.82000,  16 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (13_Danao)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      19       0.63158      0.61735    0.07510  0.00023     1001000
    2      19       0.73684      0.77952    0.49235  0.00040     1001000
    3       6       0.50000      0.45455    1.00000  0.00000     1001000
    4      13       0.38462      0.58154    0.19303  0.00040     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp7">&nbsp;</A>

===============================================================================
== Sample : 	14_Sogod
===============================================================================


===============================
== Standard diversity indices : (14_Sogod)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 42
No. of loci              : 4
No. of usable loci       : 3 loci with less than 20.00 % missing data
No. of polymorphic loci  : 3


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        40         4   0.55000   0.67308
     2        36         8   0.83333   0.78730
     3        22         5   0.81818   0.73593
     4        34         4   0.70588   0.59358
-------------------------------------------------
  Mean    33.000     5.250   0.72685   0.69747
  s.d.     7.746     1.893   0.13089   0.08354
-------------------------------------------------



================================
== Molecular diversity indices : (14_Sogod)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 42.00000
No. of haplotypes                            : 39

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 3
Number of usable loci                        : 3


Average gene diversity over loci              :     0.524197 +/-     0.353477

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (14_Sogod)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -97.05725          LnLHood LE : -116.77279
          Exact P=  0.03248  +-   0.00171 (10100 permutations done)     Chi-square test value=39.43107 (P =  0.17173,  32 d.f.)
Pair(0, 2)
          LnLHood LD :  -89.21005          LnLHood LE : -100.37883
          Exact P=  0.21099  +-   0.00373 (10100 permutations done)     Chi-square test value=22.33755 (P =  0.32248,  20 d.f.)
Pair(1, 2)
          LnLHood LD : -106.44050          LnLHood LE : -121.89522
          Exact P=  0.64604  +-   0.00438 (10100 permutations done)     Chi-square test value=30.90944 (P =  0.84835,  40 d.f.)
Pair(0, 3)
          LnLHood LD :  -84.47429          LnLHood LE :  -93.15930
          Exact P=  0.28545  +-   0.00403 (10100 permutations done)     Chi-square test value=17.37002 (P =  0.36205,  16 d.f.)
Pair(1, 3)
          LnLHood LD : -103.83781          LnLHood LE : -114.67569
          Exact P=  0.91495  +-   0.00281 (10100 permutations done)     Chi-square test value=21.67576 (P =  0.91579,  32 d.f.)
Pair(2, 3)
          LnLHood LD :  -90.46720          LnLHood LE :  -98.28173
          Exact P=  0.67911  +-   0.00467 (10100 permutations done)     Chi-square test value=15.62907 (P =  0.73935,  20 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           1    1    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   +   -   - 
       1 |  +   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (14_Sogod)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.55000      0.67308    0.50386  0.00045     1001000
    2      18       0.83333      0.78730    0.83722  0.00035     1001000
    3      11       0.81818      0.73593    0.69325  0.00046     1001000
    4      17       0.70588      0.59358    0.07042  0.00027     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp8">&nbsp;</A>

===============================================================================
== Sample : 	15_Tabogon
===============================================================================


===============================
== Standard diversity indices : (15_Tabogon)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 40
No. of loci              : 4
No. of usable loci       : 3 loci with less than 20.00 % missing data
No. of polymorphic loci  : 3


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        40         4   0.75000   0.62179
     2        36         8   0.83333   0.81111
     3        16         3   0.50000   0.49167
     4        38         4   0.31579   0.55192
-------------------------------------------------
  Mean    32.500     4.750   0.59978   0.61912
  s.d.    11.121     2.217   0.23645   0.13860
-------------------------------------------------



================================
== Molecular diversity indices : (15_Tabogon)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 40.00000
No. of haplotypes                            : 33

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 3
Number of usable loci                        : 3


Average gene diversity over loci              :     0.591453 +/-     0.388040

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (15_Tabogon)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -89.20169          LnLHood LE : -100.83746
          Exact P=  0.53099  +-   0.00541 (10100 permutations done)     Chi-square test value=23.27155 (P =  0.50382,  24 d.f.)
Pair(0, 2)
          LnLHood LD :  -66.47900          LnLHood LE :  -69.53446
          Exact P=  0.79426  +-   0.00408 (10100 permutations done)     Chi-square test value= 6.11091 (P =  0.72877,  9 d.f.)
Pair(1, 2)
          LnLHood LD :  -94.79322          LnLHood LE : -104.47755
          Exact P=  0.62871  +-   0.00529 (10100 permutations done)     Chi-square test value=19.36865 (P =  0.73205,  24 d.f.)
Pair(0, 3)
          LnLHood LD :  -66.54625          LnLHood LE :  -70.40146
          Exact P=  0.78208  +-   0.00397 (10100 permutations done)     Chi-square test value= 7.71042 (P =  0.80733,  12 d.f.)
Pair(1, 3)
          LnLHood LD :  -87.91742          LnLHood LE : -105.34455
          Exact P=  0.02149  +-   0.00132 (10100 permutations done)     Chi-square test value=34.85427 (P =  0.33376,  32 d.f.)
Pair(2, 3)
          LnLHood LD :  -69.47547          LnLHood LE :  -74.04155
          Exact P=  0.55772  +-   0.00515 (10100 permutations done)     Chi-square test value= 9.13215 (P =  0.69160,  12 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    1    0    1
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   + 
       2 |  -   -   *   - 
       3 |  -   +   -   * 

===============================
== Hardy-Weinberg equilibrium : (15_Tabogon)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.75000      0.62179    0.73512  0.00041     1001000
    2      18       0.83333      0.81111    0.62067  0.00046     1001000
    3       8       0.50000      0.49167    0.28298  0.00043     1001000
    4      19       0.31579      0.55192    0.04774  0.00022     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp9">&nbsp;</A>

===============================================================================
== Sample : 	16_Daanbantayan
===============================================================================


===============================
== Standard diversity indices : (16_Daanbantayan)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 38
No. of loci              : 4
No. of usable loci       : 2 loci with less than 20.00 % missing data
No. of polymorphic loci  : 2


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        36         3   0.61111   0.60317
     2        34         9   0.76471   0.81640
     3        12         3   0.33333   0.43939
     4        24         3   0.50000   0.58333
-------------------------------------------------
  Mean    26.500     4.500   0.55229   0.61058
  s.d.    11.000     3.000   0.18189   0.15542
-------------------------------------------------



================================
== Molecular diversity indices : (16_Daanbantayan)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 38.00000
No. of haplotypes                            : 30

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 2
Number of usable loci                        : 2


Average gene diversity over loci              :     0.596017 +/-     0.433208

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (16_Daanbantayan)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -86.27287          LnLHood LE : -102.55772
          Exact P=  0.10119  +-   0.00328 (10100 permutations done)     Chi-square test value=32.56971 (P =  0.21164,  27 d.f.)
Pair(0, 2)
          LnLHood LD :  -59.15829          LnLHood LE :  -66.49646
          Exact P=  0.04089  +-   0.00205 (10100 permutations done)     Chi-square test value=14.67634 (P =  0.10022,  9 d.f.)
Pair(1, 2)
          LnLHood LD :  -83.79589          LnLHood LE :  -98.00157
          Exact P=  0.04842  +-   0.00197 (10100 permutations done)     Chi-square test value=28.41135 (P =  0.39001,  27 d.f.)
Pair(0, 3)
          LnLHood LD :  -72.37009          LnLHood LE :  -78.14324
          Exact P=  0.31317  +-   0.00477 (10100 permutations done)     Chi-square test value=11.54629 (P =  0.24011,  9 d.f.)
Pair(1, 3)
          LnLHood LD :  -93.36647          LnLHood LE : -109.64835
          Exact P=  0.16554  +-   0.00367 (10100 permutations done)     Chi-square test value=32.56375 (P =  0.21185,  27 d.f.)
Pair(2, 3)
          LnLHood LD :  -67.78429          LnLHood LE :  -73.58709
          Exact P=  0.18287  +-   0.00323 (10100 permutations done)     Chi-square test value=11.60560 (P =  0.23647,  9 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           1    1    2    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   +   - 
       1 |  -   *   +   - 
       2 |  +   +   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (16_Daanbantayan)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      18       0.61111      0.60317    0.75285  0.00044     1001000
    2      17       0.76471      0.81640    0.77830  0.00030     1001000
    3       6       0.33333      0.43939    0.27462  0.00044     1001000
    4      12       0.50000      0.58333    0.79267  0.00040     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp10">&nbsp;</A>

===============================================================================
== Sample : 	17_Malapascua
===============================================================================


===============================
== Standard diversity indices : (17_Malapascua)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 40
No. of loci              : 4
No. of usable loci       : 3 loci with less than 20.00 % missing data
No. of polymorphic loci  : 3


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        40         4   0.60000   0.55769
     2        36        11   0.83333   0.84762
     3        22         6   0.72727   0.67532
     4        38         4   0.68421   0.59602
-------------------------------------------------
  Mean    34.000     6.250   0.71120   0.66916
  s.d.     8.165     3.304   0.09707   0.12866
-------------------------------------------------



================================
== Molecular diversity indices : (17_Malapascua)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 40.00000
No. of haplotypes                            : 33

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 3
Number of usable loci                        : 3


Average gene diversity over loci              :     0.593162 +/-     0.388905

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (17_Malapascua)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -95.65969          LnLHood LE : -103.86522
          Exact P=  0.98307  +-   0.00134 (10100 permutations done)     Chi-square test value=16.41104 (P =  0.99298,  33 d.f.)
Pair(0, 2)
          LnLHood LD :  -69.68844          LnLHood LE :  -80.11136
          Exact P=  0.05168  +-   0.00221 (10100 permutations done)     Chi-square test value=20.84585 (P =  0.28726,  18 d.f.)
Pair(1, 2)
          LnLHood LD : -108.21569          LnLHood LE : -126.40068
          Exact P=  0.90139  +-   0.00292 (10100 permutations done)     Chi-square test value=36.36999 (P =  0.99887,  66 d.f.)
Pair(0, 3)
          LnLHood LD :  -61.82430          LnLHood LE :  -65.78863
          Exact P=  0.68901  +-   0.00473 (10100 permutations done)     Chi-square test value= 7.92866 (P =  0.79068,  12 d.f.)
Pair(1, 3)
          LnLHood LD : -100.65606          LnLHood LE : -112.07795
          Exact P=  0.98772  +-   0.00108 (10100 permutations done)     Chi-square test value=22.84378 (P =  0.99651,  44 d.f.)
Pair(2, 3)
          LnLHood LD :  -76.92774          LnLHood LE :  -88.32410
          Exact P=  0.16525  +-   0.00381 (10100 permutations done)     Chi-square test value=22.79272 (P =  0.53206,  24 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (17_Malapascua)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.60000      0.55769    1.00000  0.00000     1001000
    2      18       0.83333      0.84762    0.57399  0.00031     1001000
    3      11       0.72727      0.67532    0.30748  0.00044     1001000
    4      19       0.68421      0.59602    0.37922  0.00049     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp11">&nbsp;</A>

===============================================================================
== Sample : 	18_Pintuyan
===============================================================================


===============================
== Standard diversity indices : (18_Pintuyan)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 30
No. of loci              : 4
No. of usable loci       : 2 loci with less than 20.00 % missing data
No. of polymorphic loci  : 2


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        26         4   0.61538   0.64000
     2        24         6   0.83333   0.81884
     3        14         3   0.42857   0.64835
     4        16         2   0.87500   0.52500
-------------------------------------------------
  Mean    20.000     3.750   0.68807   0.65805
  s.d.     5.888     1.708   0.20710   0.12107
-------------------------------------------------



================================
== Molecular diversity indices : (18_Pintuyan)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 30.00000
No. of haplotypes                            : 25

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 2
Number of usable loci                        : 2


Average gene diversity over loci              :     0.498851 +/-     0.384454

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (18_Pintuyan)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -67.88826          LnLHood LE :  -81.78301
          Exact P=  0.16634  +-   0.00364 (10100 permutations done)     Chi-square test value=27.78950 (P =  0.26902,  24 d.f.)
Pair(0, 2)
          LnLHood LD :  -58.41171          LnLHood LE :  -67.07057
          Exact P=  0.13010  +-   0.00370 (10100 permutations done)     Chi-square test value=17.31773 (P =  0.13803,  12 d.f.)
Pair(1, 2)
          LnLHood LD :  -69.45800          LnLHood LE :  -79.79285
          Exact P=  0.35386  +-   0.00458 (10100 permutations done)     Chi-square test value=20.66969 (P =  0.29640,  18 d.f.)
Pair(0, 3)
          LnLHood LD :  -54.98904          LnLHood LE :  -61.37106
          Exact P=  0.16277  +-   0.00337 (10100 permutations done)     Chi-square test value=12.76405 (P =  0.12023,  8 d.f.)
Pair(1, 3)
          LnLHood LD :  -65.84027          LnLHood LE :  -74.09334
          Exact P=  0.29653  +-   0.00476 (10100 permutations done)     Chi-square test value=16.50613 (P =  0.16914,  12 d.f.)
Pair(2, 3)
          LnLHood LD :  -54.39657          LnLHood LE :  -59.38090
          Exact P=  0.16257  +-   0.00378 (10100 permutations done)     Chi-square test value= 9.96866 (P =  0.12598,  6 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (18_Pintuyan)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      13       0.61538      0.64000    0.21175  0.00038     1001000
    2      12       0.83333      0.81884    0.73261  0.00052     1001000
    3       7       0.42857      0.64835    0.25512  0.00042     1001000
    4       8       0.87500      0.52500    0.13812  0.00036     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp12">&nbsp;</A>

===============================================================================
== Sample : 	19_PadreBurgos
===============================================================================


===============================
== Standard diversity indices : (19_PadreBurgos)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 40
No. of loci              : 4
No. of usable loci       : 2 loci with less than 20.00 % missing data
No. of polymorphic loci  : 2


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        36         5   0.55556   0.61111
     2        32        10   0.87500   0.84879
     3        18         5   0.77778   0.73203
     4        28         2   0.35714   0.49471
-------------------------------------------------
  Mean    28.500     5.500   0.64137   0.67166
  s.d.     7.724     3.317   0.23190   0.15275
-------------------------------------------------



================================
== Molecular diversity indices : (19_PadreBurgos)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 40.00000
No. of haplotypes                            : 36

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 2
Number of usable loci                        : 2


Average gene diversity over loci              :     0.516667 +/-     0.390782

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (19_PadreBurgos)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -99.09300          LnLHood LE : -118.04230
          Exact P=  0.30545  +-   0.00444 (10100 permutations done)     Chi-square test value=37.89861 (P =  0.89548,  50 d.f.)
Pair(0, 2)
          LnLHood LD :  -84.60803          LnLHood LE :  -91.44040
          Exact P=  0.91050  +-   0.00272 (10100 permutations done)     Chi-square test value=13.66473 (P =  0.96727,  25 d.f.)
Pair(1, 2)
          LnLHood LD : -103.94503          LnLHood LE : -120.04522
          Exact P=  0.74990  +-   0.00426 (10100 permutations done)     Chi-square test value=32.20038 (P =  0.97621,  50 d.f.)
Pair(0, 3)
          LnLHood LD :  -76.63887          LnLHood LE :  -84.44783
          Exact P=  0.10307  +-   0.00319 (10100 permutations done)     Chi-square test value=15.61792 (P =  0.11110,  10 d.f.)
Pair(1, 3)
          LnLHood LD : -102.60285          LnLHood LE : -113.05265
          Exact P=  0.69040  +-   0.00447 (10100 permutations done)     Chi-square test value=20.89959 (P =  0.40306,  20 d.f.)
Pair(2, 3)
          LnLHood LD :  -78.69884          LnLHood LE :  -86.45074
          Exact P=  0.13792  +-   0.00297 (10100 permutations done)     Chi-square test value=15.50380 (P =  0.11475,  10 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (19_PadreBurgos)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      18       0.55556      0.61111    0.31145  0.00047     1001000
    2      16       0.87500      0.84879    0.48000  0.00056     1001000
    3       9       0.77778      0.73203    0.84041  0.00033     1001000
    4      14       0.35714      0.49471    0.57084  0.00052     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp13">&nbsp;</A>

===============================================================================
== Sample : 	20_Maasin
===============================================================================


===============================
== Standard diversity indices : (20_Maasin)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 34
No. of loci              : 4
No. of usable loci       : 2 loci with less than 20.00 % missing data
No. of polymorphic loci  : 2


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        34         4   0.58824   0.65775
     2        30         9   0.86667   0.79770
     3        14         5   0.71429   0.72527
     4        26         3   0.30769   0.52000
-------------------------------------------------
  Mean    26.000     5.250   0.61922   0.67518
  s.d.     8.641     2.630   0.23684   0.11819
-------------------------------------------------



================================
== Molecular diversity indices : (20_Maasin)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 34.00000
No. of haplotypes                            : 29

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 2
Number of usable loci                        : 2


Average gene diversity over loci              :     0.638146 +/-     0.456608

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (20_Maasin)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -70.91594          LnLHood LE :  -88.45217
          Exact P=  0.01564  +-   0.00126 (10100 permutations done)     Chi-square test value=35.07247 (P =  0.13704,  27 d.f.)
Pair(0, 2)
          LnLHood LD :  -64.74703          LnLHood LE :  -70.40519
          Exact P=  0.76941  +-   0.00434 (10100 permutations done)     Chi-square test value=11.31632 (P =  0.72987,  15 d.f.)
Pair(1, 2)
          LnLHood LD :  -76.66846          LnLHood LE :  -94.47174
          Exact P=  0.12554  +-   0.00339 (10100 permutations done)     Chi-square test value=35.60655 (P =  0.84068,  45 d.f.)
Pair(0, 3)
          LnLHood LD :  -64.43093          LnLHood LE :  -68.54451
          Exact P=  0.60475  +-   0.00530 (10100 permutations done)     Chi-square test value= 8.22715 (P =  0.51143,  9 d.f.)
Pair(1, 3)
          LnLHood LD :  -80.05574          LnLHood LE :  -92.61106
          Exact P=  0.44485  +-   0.00434 (10100 permutations done)     Chi-square test value=25.11063 (P =  0.56826,  27 d.f.)
Pair(2, 3)
          LnLHood LD :  -65.96057          LnLHood LE :  -74.56407
          Exact P=  0.19663  +-   0.00390 (10100 permutations done)     Chi-square test value=17.20699 (P =  0.30664,  15 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           1    1    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   +   -   - 
       1 |  +   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (20_Maasin)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      17       0.58824      0.65775    0.36674  0.00046     1001000
    2      15       0.86667      0.79770    0.53747  0.00051     1001000
    3       7       0.71429      0.72527    0.88662  0.00032     1001000
    4      13       0.30769      0.52000    0.20753  0.00040     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp14">&nbsp;</A>

===============================================================================
== Sample : 	22_InopacanIsland
===============================================================================


===============================
== Standard diversity indices : (22_InopacanIsland)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 38
No. of loci              : 4
No. of usable loci       : 2 loci with less than 20.00 % missing data
No. of polymorphic loci  : 2


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        38         5   0.63158   0.70270
     2        34         6   0.76471   0.74866
     3        14         2   0.42857   0.36264
     4        24         3   0.50000   0.51812
-------------------------------------------------
  Mean    27.500     4.000   0.58121   0.58303
  s.d.    10.755     1.826   0.14844   0.17752
-------------------------------------------------



================================
== Molecular diversity indices : (22_InopacanIsland)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 38.00000
No. of haplotypes                            : 30

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 2
Number of usable loci                        : 2


Average gene diversity over loci              :     0.650071 +/-     0.461407

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (22_InopacanIsland)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -84.60444          LnLHood LE :  -96.63708
          Exact P=  0.43525  +-   0.00456 (10100 permutations done)     Chi-square test value=24.06529 (P =  0.45787,  24 d.f.)
Pair(0, 2)
          LnLHood LD :  -68.19821          LnLHood LE :  -71.52550
          Exact P=  0.69089  +-   0.00486 (10100 permutations done)     Chi-square test value= 6.65457 (P =  0.57432,  8 d.f.)
Pair(1, 2)
          LnLHood LD :  -83.06498          LnLHood LE :  -85.51730
          Exact P=  0.99703  +-   0.00050 (10100 permutations done)     Chi-square test value= 4.90464 (P =  0.96109,  12 d.f.)
Pair(0, 3)
          LnLHood LD :  -71.92284          LnLHood LE :  -81.18894
          Exact P=  0.06010  +-   0.00255 (10100 permutations done)     Chi-square test value=18.53221 (P =  0.10046,  12 d.f.)
Pair(1, 3)
          LnLHood LD :  -82.30940          LnLHood LE :  -95.18075
          Exact P=  0.06050  +-   0.00229 (10100 permutations done)     Chi-square test value=25.74269 (P =  0.10579,  18 d.f.)
Pair(2, 3)
          LnLHood LD :  -68.31189          LnLHood LE :  -70.06916
          Exact P=  0.83267  +-   0.00349 (10100 permutations done)     Chi-square test value= 3.51454 (P =  0.74203,  6 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (22_InopacanIsland)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      19       0.63158      0.70270    0.53928  0.00049     1001000
    2      17       0.76471      0.74866    0.91421  0.00025     1001000
    3       7       0.42857      0.36264    1.00000  0.00000     1001000
    4      12       0.50000      0.51812    0.47393  0.00052     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp15">&nbsp;</A>

===============================================================================
== Sample : 	22_InopacanMainland
===============================================================================


===============================
== Standard diversity indices : (22_InopacanMainland)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 28
No. of loci              : 4
No. of usable loci       : 1 loci with less than 20.00 % missing data
No. of polymorphic loci  : 1


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        28         4   0.50000   0.58466
     2        22         9   0.72727   0.84848
     3        12         4   0.66667   0.74242
     4        18         2   0.77778   0.52941
-------------------------------------------------
  Mean    20.000     4.750   0.66793   0.67624
  s.d.     6.733     2.986   0.12082   0.14605
-------------------------------------------------



================================
== Molecular diversity indices : (22_InopacanMainland)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 28.00000
No. of haplotypes                            : 24

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 1
Number of usable loci                        : 1


Average gene diversity over loci              :     0.584656 +/-     0.539122

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (22_InopacanMainland)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -63.32097          LnLHood LE :  -74.64464
          Exact P=  0.50069  +-   0.00519 (10100 permutations done)     Chi-square test value=22.64733 (P =  0.70381,  27 d.f.)
Pair(0, 2)
          LnLHood LD :  -49.93600          LnLHood LE :  -55.17839
          Exact P=  0.54663  +-   0.00505 (10100 permutations done)     Chi-square test value=10.48479 (P =  0.57351,  12 d.f.)
Pair(1, 2)
          LnLHood LD :  -65.15327          LnLHood LE :  -81.84436
          Exact P=  0.10624  +-   0.00293 (10100 permutations done)     Chi-square test value=33.38218 (P =  0.59374,  36 d.f.)
Pair(0, 3)
          LnLHood LD :  -47.78203          LnLHood LE :  -49.86314
          Exact P=  0.85406  +-   0.00358 (10100 permutations done)     Chi-square test value= 4.16221 (P =  0.65474,  6 d.f.)
Pair(1, 3)
          LnLHood LD :  -67.82518          LnLHood LE :  -76.52910
          Exact P=  0.80792  +-   0.00409 (10100 permutations done)     Chi-square test value=17.40785 (P =  0.49525,  18 d.f.)
Pair(2, 3)
          LnLHood LD :  -50.14560          LnLHood LE :  -57.06286
          Exact P=  0.08822  +-   0.00290 (10100 permutations done)     Chi-square test value=13.83451 (P =  0.08618,  8 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (22_InopacanMainland)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      14       0.50000      0.58466    0.42076  0.00055     1001000
    2      11       0.72727      0.84848    0.41236  0.00033     1001000
    3       6       0.66667      0.74242    0.12301  0.00034     1001000
    4       9       0.77778      0.52941    0.22589  0.00042     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp16">&nbsp;</A>

===============================================================================
== Sample : 	23_Baybay
===============================================================================


===============================
== Standard diversity indices : (23_Baybay)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 36
No. of loci              : 4
No. of usable loci       : 1 loci with less than 20.00 % missing data
No. of polymorphic loci  : 1


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        36         3   0.44444   0.43810
     2        28         8   0.85714   0.80159
     3        12         5   0.83333   0.83333
     4        22         2   0.54545   0.51948
-------------------------------------------------
  Mean    24.500     4.500   0.67009   0.64812
  s.d.    10.116     2.646   0.20663   0.19876
-------------------------------------------------



================================
== Molecular diversity indices : (23_Baybay)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 36.00000
No. of haplotypes                            : 25

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 1
Number of usable loci                        : 1


Average gene diversity over loci              :     0.438095 +/-     0.446463

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (23_Baybay)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -73.33135          LnLHood LE :  -79.29431
          Exact P=  0.78545  +-   0.00384 (10100 permutations done)     Chi-square test value=11.92592 (P =  0.74906,  16 d.f.)
Pair(0, 2)
          LnLHood LD :  -55.87521          LnLHood LE :  -58.38322
          Exact P=  0.92337  +-   0.00271 (10100 permutations done)     Chi-square test value= 5.01602 (P =  0.89011,  10 d.f.)
Pair(1, 2)
          LnLHood LD :  -77.85659          LnLHood LE :  -96.21920
          Exact P=  0.03069  +-   0.00181 (10100 permutations done)     Chi-square test value=36.72522 (P =  0.61848,  40 d.f.)
Pair(0, 3)
          LnLHood LD :  -53.19439          LnLHood LE :  -55.78543
          Exact P=  0.38000  +-   0.00492 (10100 permutations done)     Chi-square test value= 5.18208 (P =  0.26912,  4 d.f.)
Pair(1, 3)
          LnLHood LD :  -79.45779          LnLHood LE :  -93.62141
          Exact P=  0.02574  +-   0.00166 (10100 permutations done)     Chi-square test value=28.32725 (P =  0.02889,  16 d.f.)
Pair(2, 3)
          LnLHood LD :  -65.04259          LnLHood LE :  -72.71032
          Exact P=  0.12881  +-   0.00360 (10100 permutations done)     Chi-square test value=15.33545 (P =  0.12030,  10 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    2    1    1
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   +   + 
       2 |  -   +   *   - 
       3 |  -   +   -   * 

===============================
== Hardy-Weinberg equilibrium : (23_Baybay)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      18       0.44444      0.43810    1.00000  0.00000     1001000
    2      14       0.85714      0.80159    0.58171  0.00041     1001000
    3       6       0.83333      0.83333    0.55815  0.00046     1001000
    4      11       0.54545      0.51948    1.00000  0.00000     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp17">&nbsp;</A>

===============================================================================
== Sample : 	24_Albuera
===============================================================================


===============================
== Standard diversity indices : (24_Albuera)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 38
No. of loci              : 4
No. of usable loci       : 1 loci with less than 20.00 % missing data
No. of polymorphic loci  : 1


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        36         3   0.55556   0.45079
     2        30         8   0.86667   0.75862
     3        20         5   0.90000   0.72105
     4        26         2   0.46154   0.51692
-------------------------------------------------
  Mean    28.000     4.500   0.69594   0.61185
  s.d.     6.733     2.646   0.22018   0.15102
-------------------------------------------------



================================
== Molecular diversity indices : (24_Albuera)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 38.00000
No. of haplotypes                            : 33

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 1
Number of usable loci                        : 1


Average gene diversity over loci              :     0.403983 +/-     0.424329

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (24_Albuera)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -79.44599          LnLHood LE :  -89.62985
          Exact P=  0.61614  +-   0.00454 (10100 permutations done)     Chi-square test value=20.36771 (P =  0.67568,  24 d.f.)
Pair(0, 2)
          LnLHood LD :  -69.31148          LnLHood LE :  -75.56935
          Exact P=  0.60792  +-   0.00429 (10100 permutations done)     Chi-square test value=12.51575 (P =  0.63965,  15 d.f.)
Pair(1, 2)
          LnLHood LD :  -92.39779          LnLHood LE : -107.39068
          Exact P=  0.63168  +-   0.00430 (10100 permutations done)     Chi-square test value=29.98578 (P =  0.87562,  40 d.f.)
Pair(0, 3)
          LnLHood LD :  -60.37469          LnLHood LE :  -66.38909
          Exact P=  0.08782  +-   0.00303 (10100 permutations done)     Chi-square test value=12.02881 (P =  0.06133,  6 d.f.)
Pair(1, 3)
          LnLHood LD :  -89.61165          LnLHood LE :  -98.21041
          Exact P=  0.67158  +-   0.00419 (10100 permutations done)     Chi-square test value=17.19753 (P =  0.37293,  16 d.f.)
Pair(2, 3)
          LnLHood LD :  -77.48835          LnLHood LE :  -84.14992
          Exact P=  0.30871  +-   0.00458 (10100 permutations done)     Chi-square test value=13.32314 (P =  0.20616,  10 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   -   - 
       1 |  -   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (24_Albuera)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      18       0.55556      0.45079    0.76607  0.00042     1001000
    2      15       0.86667      0.75862    0.77789  0.00029     1001000
    3      10       0.90000      0.72105    0.94946  0.00023     1001000
    4      13       0.46154      0.51692    1.00000  0.00000     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp18">&nbsp;</A>

===============================================================================
== Sample : 	25_Ormoc
===============================================================================


===============================
== Standard diversity indices : (25_Ormoc)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 40
No. of loci              : 4
No. of usable loci       : 2 loci with less than 20.00 % missing data
No. of polymorphic loci  : 2


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        40         4   0.60000   0.55769
     2        36         7   0.77778   0.79841
     3        12         5   0.83333   0.83333
     4        26         3   0.38462   0.55692
-------------------------------------------------
  Mean    28.500     4.750   0.64893   0.68659
  s.d.    12.477     1.708   0.20237   0.14996
-------------------------------------------------



================================
== Molecular diversity indices : (25_Ormoc)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 40.00000
No. of haplotypes                            : 30

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 2
Number of usable loci                        : 2


Average gene diversity over loci              :     0.601282 +/-     0.435399

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (25_Ormoc)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -85.44163          LnLHood LE :  -94.52661
          Exact P=  0.55050  +-   0.00545 (10100 permutations done)     Chi-square test value=18.16997 (P =  0.63822,  21 d.f.)
Pair(0, 2)
          LnLHood LD :  -59.27449          LnLHood LE :  -69.02352
          Exact P=  0.03723  +-   0.00195 (10100 permutations done)     Chi-square test value=19.49805 (P =  0.19205,  15 d.f.)
Pair(1, 2)
          LnLHood LD :  -85.66808          LnLHood LE : -103.85132
          Exact P=  0.02287  +-   0.00137 (10100 permutations done)     Chi-square test value=36.36649 (P =  0.40486,  35 d.f.)
Pair(0, 3)
          LnLHood LD :  -69.64855          LnLHood LE :  -73.82882
          Exact P=  0.48208  +-   0.00516 (10100 permutations done)     Chi-square test value= 8.36055 (P =  0.49826,  9 d.f.)
Pair(1, 3)
          LnLHood LD :  -99.45047          LnLHood LE : -108.65663
          Exact P=  0.71356  +-   0.00432 (10100 permutations done)     Chi-square test value=18.41232 (P =  0.62278,  21 d.f.)
Pair(2, 3)
          LnLHood LD :  -76.12037          LnLHood LE :  -83.15353
          Exact P=  0.54604  +-   0.00496 (10100 permutations done)     Chi-square test value=14.06633 (P =  0.52050,  15 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           1    1    2    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   -   +   - 
       1 |  -   *   +   - 
       2 |  +   +   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (25_Ormoc)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.60000      0.55769    0.74390  0.00047     1001000
    2      18       0.77778      0.79841    0.78786  0.00038     1001000
    3       6       0.83333      0.83333    0.55373  0.00048     1001000
    4      13       0.38462      0.55692    0.26072  0.00043     1001000
------------------------------------------------------------------------


<A NAME="08_01_09at12_58_07_samp19">&nbsp;</A>

===============================================================================
== Sample : 	27_Palompon
===============================================================================


===============================
== Standard diversity indices : (27_Palompon)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 36
No. of loci              : 4
No. of usable loci       : 2 loci with less than 20.00 % missing data
No. of polymorphic loci  : 2


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        34         4   0.52941   0.44207
     2        34         9   0.82353   0.83066
     3        16         5   0.50000   0.75000
     4        28         4   0.42857   0.58466
-------------------------------------------------
  Mean    28.000     5.500   0.57038   0.65185
  s.d.     8.485     2.380   0.17400   0.17333
-------------------------------------------------



================================
== Molecular diversity indices : (27_Palompon)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 36.00000
No. of haplotypes                            : 29

Allowed level of missing data                : 20.00000 %
Number of polymorphic loci                   : 2
Number of usable loci                        : 2


Average gene diversity over loci              :     0.566667 +/-     0.418391

(Standard deviations are for both sampling and stochastic processes)

====================================
== Pairwise linkage disequilibrium : (27_Palompon)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -71.44836          LnLHood LE :  -92.00459
          Exact P=  0.00287  +-   0.00051 (10100 permutations done)     Chi-square test value=41.11245 (P =  0.25658,  36 d.f.)
Pair(0, 2)
          LnLHood LD :  -62.44940          LnLHood LE :  -73.42420
          Exact P=  0.07594  +-   0.00258 (10100 permutations done)     Chi-square test value=21.94961 (P =  0.34325,  20 d.f.)
Pair(1, 2)
          LnLHood LD :  -88.55147          LnLHood LE : -106.29451
          Exact P=  0.34376  +-   0.00485 (10100 permutations done)     Chi-square test value=35.48608 (P =  0.84426,  45 d.f.)
Pair(0, 3)
          LnLHood LD :  -64.37742          LnLHood LE :  -71.01372
          Exact P=  0.59634  +-   0.00451 (10100 permutations done)     Chi-square test value=13.27262 (P =  0.65273,  16 d.f.)
Pair(1, 3)
          LnLHood LD :  -85.74608          LnLHood LE : -103.88403
          Exact P=  0.10069  +-   0.00288 (10100 permutations done)     Chi-square test value=36.27591 (P =  0.45579,  36 d.f.)
Pair(2, 3)
          LnLHood LD :  -75.60725          LnLHood LE :  -85.30365
          Exact P=  0.27921  +-   0.00493 (10100 permutations done)     Chi-square test value=19.39280 (P =  0.49644,  20 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3
---------------------------------------------------------------------
           1    1    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ----------------
Locus  # |  0|  1|  2|  3|
          ----------------
       0 |  *   +   -   - 
       1 |  +   *   -   - 
       2 |  -   -   *   - 
       3 |  -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (27_Palompon)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      17       0.52941      0.44207    1.00000  0.00000     1001000
    2      17       0.82353      0.83066    0.43159  0.00032     1001000
    3       8       0.50000      0.75000    0.08923  0.00029     1001000
    4      14       0.42857      0.58466    0.23468  0.00045     1001000
------------------------------------------------------------------------




================================================================================
== Summary of computations done within populations
================================================================================

<A NAME="08_01_09at12_58_07_comp_sum_Basic">&nbsp;</A>

----------------
Basic properties
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
          Statistics           2_Handumon          7_Santander            8_Boljoon              9_Argao            10_Carcar       11_Minglanilla             13_Danao             14_Sogod           15_Tabogon      16_Daanbantayan        17_Malapascua          18_Pintuyan       19_PadreBurgos            20_Maasin    22_InopacanIsland  22_InopacanMainland            23_Baybay           24_Albuera             25_Ormoc          27_Palompon        Mean        s.d.
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  No. of gene copies                   14                   40                   40                   40                   38                   38                   40                   42                   40                   38                   40                   30                   40                   34                   38                   28                   36                   38                   40                   36      36.500       6.354
         No. of loci                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4                    4       4.000       0.000
  No. of usable loci                    1                    2                    2                    2                    2                    2                    2                    3                    3                    2                    3                    2                    2                    2                    2                    1                    1                    1                    2                    2       1.950       0.605
  No. of polym. loci                    1                    2                    2                    2                    2                    2                    2                    3                    3                    2                    3                    2                    2                    2                    2                    1                    1                    1                    2                    2       1.950       0.605
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
<A NAME="08_01_09at12_58_07_comp_sum_het">&nbsp;</A>

-----------------------
Expected heterozygosity
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Locus#           2_Handumon          7_Santander            8_Boljoon              9_Argao            10_Carcar       11_Minglanilla             13_Danao             14_Sogod           15_Tabogon      16_Daanbantayan        17_Malapascua          18_Pintuyan       19_PadreBurgos            20_Maasin    22_InopacanIsland  22_InopacanMainland            23_Baybay           24_Albuera             25_Ormoc          27_Palompon        Mean        s.d.   Tot. Het.
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
     1              0.58242              0.63016              0.65000              0.64154              0.52774              0.52774              0.61735              0.67308              0.62179              0.60317              0.55769              0.64000              0.61111              0.65775              0.70270              0.58466              0.43810              0.45079              0.55769              0.44207     0.58588     0.07639     0.58817
     2              0.92857              0.69340              0.71480              0.85205              0.76032              0.72549              0.77952              0.78730              0.81111              0.81640              0.84762              0.81884              0.84879              0.79770              0.74866              0.84848              0.80159              0.75862              0.79841              0.83066     0.79842     0.05552     0.79336
     3              0.00000              0.70330              0.67974              0.71667              0.70000              0.69697              0.45455              0.73593              0.49167              0.43939              0.67532              0.64835              0.73203              0.72527              0.36264              0.74242              0.83333              0.72105              0.83333              0.75000     0.63210     0.19607     0.67507
     4              0.83333              0.53333              0.54483              0.51852              0.55077              0.61905              0.58154              0.59358              0.55192              0.58333              0.59602              0.52500              0.49471              0.52000              0.51812              0.52941              0.51948              0.51692              0.55692              0.58466     0.56357     0.07185     0.54362
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Mean              0.58608              0.64005              0.64734              0.68219              0.63471              0.64231              0.60824              0.69747              0.61912              0.61058              0.66916              0.65805              0.67166              0.67518              0.58303              0.67624              0.64812              0.61185              0.68659              0.65185     0.64499     0.03327     0.65006
  s.d.              0.41711              0.07817              0.07329              0.13962              0.11333              0.08865              0.13386              0.08354              0.13860              0.15542              0.12866              0.12107              0.15275              0.11819              0.17752              0.14605              0.19876              0.15102              0.14996              0.17333     0.14694     0.07192     0.09529
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
<A NAME="08_01_09at12_58_07_comp_sum_thetaH">&nbsp;</A>

----------------------------------------
Theta(H) under the infinite-allele model
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Locus#           2_Handumon          7_Santander            8_Boljoon              9_Argao            10_Carcar       11_Minglanilla             13_Danao             14_Sogod           15_Tabogon      16_Daanbantayan        17_Malapascua          18_Pintuyan       19_PadreBurgos            20_Maasin    22_InopacanIsland  22_InopacanMainland            23_Baybay           24_Albuera             25_Ormoc          27_Palompon        Mean        s.d.
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
     1              1.39474              1.70386              1.85714              1.78968              1.11747              1.11747              1.61338              2.05882              1.64407              1.52000              1.26087              1.77778              1.57143              1.92187              2.36364              1.40764              0.77966              0.82081              1.26087              0.79233     1.48868     0.41896
     2             13.00000              2.26163              2.50625              5.75904              3.17219              2.64286              3.53548              3.70149              4.29412              4.44660              5.56250              4.52000              5.61333              3.94318              2.97872              5.60000              4.04000              3.14286              3.96063              4.90526     4.47931     2.21753
     3              0.00000              2.37037              2.12245              2.52941              2.33333              2.30000              0.83333              2.78689              0.96721              0.78378              2.08000              1.84375              2.73171              2.64000              0.56897              2.88235              5.00000              2.58491              5.00000              3.00000     2.26792     1.24234
     4              5.00000              1.14286              1.19697              1.07692              1.22603              1.62500              1.38971              1.46053              1.23175              1.40000              1.47535              1.10526              0.97906              1.08333              1.07519              1.12500              1.08108              1.07006              1.25694              1.40764     1.42043     0.83841
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Mean              1.41593              1.77815              1.83559              2.14656              1.73752              1.79573              1.55257              2.30549              1.62552              1.56789              2.02264              1.92439              2.04562              2.07865              1.39825              2.08875              1.84191              1.57631              2.19071              1.87229     1.84002     0.26155
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
<A NAME="08_01_09at12_58_07_comp_sum_numAll">&nbsp;</A>

-----------------
Number of alleles
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Locus#           2_Handumon          7_Santander            8_Boljoon              9_Argao            10_Carcar       11_Minglanilla             13_Danao             14_Sogod           15_Tabogon      16_Daanbantayan        17_Malapascua          18_Pintuyan       19_PadreBurgos            20_Maasin    22_InopacanIsland  22_InopacanMainland            23_Baybay           24_Albuera             25_Ormoc          27_Palompon        Mean        s.d. Tot. number
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
     1                    3                    4                    5                    6                    4                    3                    5                    4                    4                    3                    4                    4                    5                    4                    5                    4                    3                    3                    4                    4       4.050       0.826           7
     2                    6                    8                    7                   10                   10                    8                    9                    8                    8                    9                   11                    6                   10                    9                    6                    9                    8                    8                    7                    9       8.300       1.418          14
     3                    0                    4                    3                    4                    3                    4                    4                    5                    3                    3                    6                    3                    5                    5                    2                    4                    5                    5                    5                    5       3.900       1.373           8
     4                    3                    2                    3                    2                    3                    4                    4                    4                    4                    3                    4                    2                    2                    3                    3                    2                    2                    2                    3                    4       2.950       0.826           4
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Mean                3.000                4.500                4.500                5.500                5.000                4.750                5.500                5.250                4.750                4.500                6.250                3.750                5.500                5.250                4.000                4.750                4.500                4.500                4.750                5.500       4.800       0.724       8.250
  s.d.                2.449                2.517                1.915                3.416                3.367                2.217                2.380                1.893                2.217                3.000                3.304                1.708                3.317                2.630                1.826                2.986                2.646                2.646                1.708                2.380       2.526       0.564       3.631
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 1 (08/01/09 at 13:09:58))
Total computing time for this run : 0h 11m 51s 360 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

