
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>
<A NAME="08_02_09at22_38_27">&nbsp;</A>
////////////////////////////////////////////////////////////////////
RUN NUMBER 1 (08/02/09 at 22:38:27)
////////////////////////////////////////////////////////////////////

<A NAME="08_02_09at22_38_27_run_information">&nbsp;</A>

Project information:
--------------------
	NbSamples      =  1
	DataType       =  MICROSAT
	GenotypicData  =  1
	GameticPhase   =  0
	RecessiveData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Infer haplotype definition from distance matrix
        Alllowed level of missing data = 0.2

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :No. of different alleles (FST)
        GammaA Value       = 0.632
        Theta estimators   : 

    Linkage disequilibrium test:
    ----------------------------
        No. of permutations          = 10000
        No. of Initial conditions    = 2
        Print Histogram and table : No

    Hardy-Weinberg equilibrium test:
    --------------------------------
        No. of steps in Markov chain      = 1000000
        No. of Dememorisation Steps       = 100000
        Required precision on Probability = 0
        Test association at the Locus level

    Population pairwise Fst values:
    -------------------------------
        Compute pairwise differences
        No. of permutations for significance = 1000
        No. of permutations for Mantel test  = 500

       Distance matrix:
            Compute distance matrix
            Molecular distance : No. of different alleles (FST)
            Gamma a value      = 0


    Correlation between distance matrices.
    --------------------------------------------------------

        No. of Permutations for significance = 500
<A NAME="08_02_09at22_38_27_gen_struct">&nbsp;</A>
===============================================================================
== GENETIC STRUCTURE ANALYSIS
===============================================================================


Number of usable loci for distance computation : 6
Allowed level of missing data                  : 0.2

List of usable loci :
---------------------
   1     2     3     4     5     6  

List of loci with too much missing data :
-----------------------------------------

NONE

<A NAME="08_02_09at22_38_27_pairw_diff">&nbsp;</A>

======================================================================================
== Comparisons of pairs of population samples
======================================================================================

List of labels for population samples used below:
-------------------------------------------------

Label  	Population name
-----  	---------------
  1:	All


---------------------------------------
Population average pairwise differences
----------------------------------------

Above diagonal    : Average number of pairwise differences between populations (PiXY)
Diagonal elements : Average number of pairwise differences within population (PiX)
Below diagonal    : Corrected average pairwise difference (PiXY-(PiX+PiY)/2)

Distance method: No. of different alleles (FST)
                       1
            1    0.00000


-----------
PXY P value
-----------

                         1



---------------------
Corrected PXY P value
---------------------

                           1



==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="08_02_09at22_38_27_samp0">&nbsp;</A>

===============================================================================
== Sample : 	All
===============================================================================


===============================
== Standard diversity indices : (All)
===============================

Reference: <i>Nei, M., 1987.</i>
Reference: <i>Garza, J.C., and Williamson, E.G. 2001.</i>
No. of gene copies       : 756
No. of loci              : 6
No. of usable loci       : 6 loci with less than 20.00 % missing data
No. of polymorphic loci  : 6


Results are only shown for polymorphic loci
---------------------------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.   Allelic        G-W
Locus#    copies   alleles      Het.       Het     range       stat.
---------------------------------------------------------------------
     1       744         7   0.58333   0.60216        11     0.58333
     2       726        14   0.74105   0.80214        16     0.82353
     3       742         8   0.71429   0.69968         7     1.00000
     4       748         4   0.52406   0.55585         3     1.00000
     5       644         5   0.38509   0.42690         4     1.00000
     6       686         9   0.64140   0.67650         9     0.90000
---------------------------------------------------------------------
  Mean   715.000     7.833   0.59820   0.62720     8.333     0.88448
  s.d.    41.622     3.545   0.13182   0.12971     4.384     0.14982
---------------------------------------------------------------------



================================
== Molecular diversity indices : (All)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                         : 756.00000
No. of haplotypes                   : 555

Allowed level of missing data       : 20.00000 %
Number of usable loci               : 6


Distance method                     : No. of different alleles
Average gene diversity over loci    :     0.567943 +/-     0.321724

(Standard deviations are for both the sampling and the stochastic processes)

====================================
== Pairwise linkage disequilibrium : (All)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD : -2061.21982          LnLHood LE : -2107.29318
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=92.14673 (P =  0.64759,  98 d.f.)
Pair(0, 2)
          LnLHood LD : -1000.00000          LnLHood LE : -1710.25764
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=1420.51528 (P =  0.00000,  56 d.f.)
Pair(1, 2)
          LnLHood LD : -1000.00000          LnLHood LE : -2293.77469
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=2587.54937 (P =  0.00000,  112 d.f.)
Pair(0, 3)
          LnLHood LD : -1000.00000          LnLHood LE : -1361.29040
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=722.58080 (P =  0.00000,  28 d.f.)
Pair(1, 3)
          LnLHood LD : -1000.00000          LnLHood LE : -1944.80744
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=1889.61489 (P =  0.00000,  56 d.f.)
Pair(2, 3)
          LnLHood LD : -1000.00000          LnLHood LE : -1547.77190
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=1095.54380 (P =  0.00000,  32 d.f.)
Pair(0, 4)
          LnLHood LD : -1000.00000          LnLHood LE : -1560.22936
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=1120.45873 (P =  0.00000,  35 d.f.)
Pair(1, 4)
          LnLHood LD : -1000.00000          LnLHood LE : -2143.74641
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=2287.49282 (P =  0.00000,  70 d.f.)
Pair(2, 4)
          LnLHood LD : -1000.00000          LnLHood LE : -1746.71087
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=1493.42173 (P =  0.00000,  40 d.f.)
Pair(3, 4)
          LnLHood LD : -1000.00000          LnLHood LE : -1397.74363
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=795.48725 (P =  0.00000,  20 d.f.)
Pair(0, 5)
          LnLHood LD : -1000.00000          LnLHood LE : -1770.40812
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=1540.81624 (P =  0.00000,  63 d.f.)
Pair(1, 5)
          LnLHood LD : -1000.00000          LnLHood LE : -2353.92517
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=2707.85034 (P =  0.00000,  126 d.f.)
Pair(2, 5)
          LnLHood LD : -1000.00000          LnLHood LE : -1956.88962
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=1913.77925 (P =  0.00000,  72 d.f.)
Pair(3, 5)
          LnLHood LD : -1000.00000          LnLHood LE : -1607.92238
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=1215.84477 (P =  0.00000,  36 d.f.)
Pair(4, 5)
          LnLHood LD : -1000.00000          LnLHood LE : -1806.86135
          Exact P=  1.00000  +-   0.00000 (0 permutations done)     Chi-square test value=1613.72270 (P =  0.00000,  45 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3    4    5
---------------------------------------------------------------------
           0    0    0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ------------------------
Locus  # |  0|  1|  2|  3|  4|  5|
          ------------------------
       0 |  *   -   -   -   -   - 
       1 |  -   *   -   -   -   - 
       2 |  -   -   *   -   -   - 
       3 |  -   -   -   *   -   - 
       4 |  -   -   -   -   *   - 
       5 |  -   -   -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (All)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>


================================================================================
== Summary of computations done within populations
================================================================================

<A NAME="08_02_09at22_38_27_comp_sum_Basic">&nbsp;</A>

----------------
Basic properties
--------------------------------------------------------
          Statistics         All        Mean        s.d.
--------------------------------------------------------
  No. of gene copies         756     756.000       0.000
         No. of loci           6       6.000       0.000
  No. of usable loci           6       6.000       0.000
  No. of polym. loci           6       6.000       0.000
--------------------------------------------------------
<A NAME="08_02_09at22_38_27_comp_sum_het">&nbsp;</A>

-----------------------
Expected heterozygosity
------------------------------------------------------
Locus#         All        Mean        s.d.   Tot. Het.
------------------------------------------------------
     1     0.60216     0.60216     0.00000     0.60216
     2     0.80214     0.80214     0.00000     0.80214
     3     0.69968     0.69968     0.00000     0.69968
     4     0.55585     0.55585     0.00000     0.55585
     5     0.42690     0.42690     0.00000     0.42690
     6     0.67650     0.67650     0.00000     0.67650
------------------------------------------------------
  Mean     0.62720     0.62720     0.00000     0.62720
  s.d.     0.12971     0.12971     0.00000     0.11841
------------------------------------------------------
<A NAME="08_02_09at22_38_27_comp_sum_thetaH">&nbsp;</A>

------------------------------------------
Theta(H) under the stepwise mutation model
------------------------------------------
Locus#         All        Mean        s.d.
------------------------------------------
     1     1.58713     1.58713     0.00000
     2     2.65032     2.65032     0.00000
     3     1.87949     1.87949     0.00000
     4     1.52526     1.52526     0.00000
     5     1.54369     1.54369     0.00000
     6     1.78471     1.78471     0.00000
------------------------------------------
  Mean     1.63840     1.63840     0.00000
------------------------------------------
<A NAME="08_02_09at22_38_27_comp_sum_numAll">&nbsp;</A>

-----------------
Number of alleles
------------------------------------------------------
Locus#         All        Mean        s.d. Tot. number
------------------------------------------------------
     1           7       7.000       0.000           7
     2          14      14.000       0.000          14
     3           8       8.000       0.000           8
     4           4       4.000       0.000           4
     5           5       5.000       0.000           5
     6           9       9.000       0.000           9
------------------------------------------------------
  Mean       7.833       7.833       0.000       7.833
  s.d.       3.545       3.545       0.000       3.236
------------------------------------------------------
<A NAME="08_02_09at22_38_27_comp_sum_allRange">&nbsp;</A>

------------------
Allelic size range
------------------------------------------------------
Locus#         All        Mean        s.d.  Tot. range
------------------------------------------------------
     1          11      11.000       0.000          11
     2          16      16.000       0.000          16
     3           7       7.000       0.000           7
     4           3       3.000       0.000           3
     5           4       4.000       0.000           4
     6           9       9.000       0.000           9
------------------------------------------------------
  Mean       8.333       8.333       0.000       8.333
  s.d.       4.384       4.384       0.000       4.384
------------------------------------------------------
<A NAME="08_02_09at22_38_27_comp_sum_GW">&nbsp;</A>

----------------------
Garza-Williamson index
------------------------------------------
Locus#         All        Mean        s.d.
------------------------------------------
     1     0.58333     0.58333     0.00000
     2     0.82353     0.82353     0.00000
     3     1.00000     1.00000     0.00000
     4     1.00000     1.00000     0.00000
     5     1.00000     1.00000     0.00000
     6     0.90000     0.90000     0.00000
------------------------------------------
  Mean     0.88448     0.88448     0.00000
  s.d.     0.14982     0.14982     0.14982
------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 1 (08/02/09 at 22:42:42))
Total computing time for this run : 0h 4m 14s 359 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

