
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>
<A NAME="18_09_09at08_37_00">&nbsp;</A>
////////////////////////////////////////////////////////////////////
RUN NUMBER 1 (18/09/09 at 08:37:00)
////////////////////////////////////////////////////////////////////

<A NAME="18_09_09at08_37_00_run_information">&nbsp;</A>

Project information:
--------------------
	NbSamples      =  12
	DataType       =  STANDARD
	GenotypicData  =  1
	GameticPhase   =  0
	RecessiveData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Infer haplotype definition from distance matrix
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :Pairwise differences
        GammaA Value       = 0
        Theta estimators   : 

    Analysis of Molecular Variance:
    -------------------------------
        No. of Permutations = 1000

    Population pairwise Fst values:
    -------------------------------
        Compute pairwise differences
        Compute coancestry coefficients
        Compute Slatkin's distances
        No. of permutations for significance = 100
        No. of permutations for Mantel test  = 1000

       Distance matrix:
            Compute distance matrix
            Molecular distance : Pairwise differences
            Gamma a value      = 0

<A NAME="18_09_09at08_37_00_gen_struct">&nbsp;</A>
===============================================================================
== GENETIC STRUCTURE ANALYSIS
===============================================================================


Number of usable loci for distance computation : 13
Allowed level of missing data                  : 0.05

List of usable loci :
---------------------
   1     3     4     5     6     7     8     9    10    12    13    14    15  

List of loci with too much missing data :
-----------------------------------------

   2    11    16  

<A NAME="18_09_09at08_37_00_amova">&nbsp;</A>

=================================================================
  AMOVA ANALYSIS
=================================================================


---------------------------
Genetic structure to test :
---------------------------

No. of Groups = 2


	[[Structure]]

		StructureName = "New Edited Structure"
		NbGroups = 2
		IndividualLevel = 0
		DistMatLabel = ""
		Group={
		"1"
		"2"
		"7"
		"8"
		"9"
		"10"
		"11"
		"13"
		"14"
		"15"
		}

		Group={
		"19"
		"22"
		}

---------------------------


Distance method: Pairwise differences

--------------------------
AMOVA design and results :
--------------------------

Reference: <i>Weir, B.S. and Cockerham, C.C. 1984.</i>
           <i>Excoffier, L., Smouse, P., and Quattro, J. 1992.</i>
           <i>Weir, B. S., 1996.</i>
----------------------------------------------------------------------
 Source of                  Sum of      Variance         Percentage
 variation      d.f.        squares     components       of variation
----------------------------------------------------------------------
 Among
 groups           1          5.359        0.02096 Va             0.54

 Among
 populations
 within
 groups          10         38.216       -0.00407 Vb            -0.11

 Among
 individuals
 within
 populations    146        572.441        0.08225 Vc             2.13
 Within
 individuals    158        593.500        3.75633 Vd            97.43
----------------------------------------------------------------------
 Total          315       1209.516        3.85546
----------------------------------------------------------------------
 Fixation Indices
      FIS :      0.02143
      FSC :     -0.00106
      FCT :      0.00544
      FIT :      0.02571
----------------------------------------------------------------------

Significance tests (1023 permutations)
------------------

Vd and FIT : P(rand. value < obs. value)  =  0.08016
             P(rand. value = obs. value)  =  0.00880
                                  P-value =  0.08895+-0.00963

Vc and FIS : P(rand. value > obs. value)  =  0.10850
             P(rand. value = obs. value)  =  0.01075
                                  P-value =  0.11926+-0.01106

Vb and FSC : P(rand. value > obs. value)  =  0.63245
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.63245+-0.01649

Va and FCT : P(rand. value > obs. value)  =  0.06354
             P(rand. value = obs. value)  =  0.01369
                                  P-value =  0.07722+-0.00793

<A NAME="18_09_09at08_37_00_pairw_diff">&nbsp;</A>

======================================================================================
== Comparisons of pairs of population samples
======================================================================================

List of labels for population samples used below:
-------------------------------------------------

Label  	Population name
-----  	---------------
  1:	1
  2:	2
  3:	7
  4:	8
  5:	9
  6:	10
  7:	11
  8:	13
  9:	14
  10:	15
  11:	19
  12:	22

------------------------
Population pairwise FSTs
------------------------


Distance method: Pairwise differences
                     1         2         3         4         5         6         7         8         9        10        11        12
           1   0.00000
           2   0.00125   0.00000
           3   0.01029   0.00960   0.00000
           4   0.01182   0.00015  -0.00355   0.00000
           5   0.01668   0.00740   0.00474  -0.00222   0.00000
           6  -0.00029   0.00165   0.00276  -0.00110  -0.00225   0.00000
           7   0.00618   0.00104   0.00053  -0.00740   0.00718  -0.00468   0.00000
           8   0.00534  -0.01110  -0.03185  -0.02499  -0.01527  -0.02564  -0.02834   0.00000
           9   0.01556  -0.00168  -0.01803   0.00309  -0.03458  -0.01804  -0.03161  -0.07113   0.00000
          10  -0.00492  -0.04705  -0.02922  -0.03787  -0.03670  -0.03568  -0.04681  -0.05785  -0.04348   0.00000
          11   0.01248   0.01053  -0.00357  -0.00834   0.00138  -0.00221   0.00768  -0.03003  -0.01580   0.00319   0.00000
          12   0.02364   0.05345   0.00420   0.00810   0.02843   0.01032   0.03243  -0.00369   0.03955   0.09595  -0.02041   0.00000


------------
FST P values
------------

Number of permutations : 110

                       1                 2                 3                 4                 5                 6                 7                 8                 9                10                11                12
           1           *
           2   0.36036+-0.0407           *
           3   0.19820+-0.0264   0.14414+-0.0309           *
           4   0.06306+-0.0237   0.45946+-0.0370   0.64865+-0.0446           *
           5   0.05405+-0.0242   0.18919+-0.0344   0.31532+-0.0280   0.52252+-0.0402           *
           6   0.54054+-0.0332   0.27928+-0.0316   0.36937+-0.0344   0.45045+-0.0559   0.59459+-0.0454           *
           7   0.20721+-0.0385   0.39640+-0.0433   0.49550+-0.0512   0.82883+-0.0327   0.24324+-0.0592   0.81081+-0.0359           *
           8   0.48649+-0.0493   0.70270+-0.0298   0.97297+-0.0125   0.80180+-0.0287   0.79279+-0.0298   0.90991+-0.0253   0.90090+-0.0384           *
           9   0.47748+-0.0490   0.46847+-0.0354   0.87387+-0.0306   0.54054+-0.0303   0.76577+-0.0340   0.59459+-0.0364   0.73874+-0.0466   0.93694+-0.0151           *
          10   0.99099+-0.0030   0.99099+-0.0030   0.99099+-0.0030   0.99099+-0.0030   0.99099+-0.0030   0.99099+-0.0030   0.99099+-0.0030   0.99099+-0.0030   0.99099+-0.0030           *
          11   0.03604+-0.0148   0.03604+-0.0148   0.65766+-0.0430   0.89189+-0.0287   0.43243+-0.0402   0.54054+-0.0470   0.15315+-0.0305   0.96396+-0.0142   0.70270+-0.0425   0.58559+-0.0413           *
          12   0.34234+-0.0466   0.06306+-0.0139   0.36937+-0.0438   0.40541+-0.0311   0.25225+-0.0379   0.27928+-0.0370   0.22523+-0.0339   0.37838+-0.0424   0.99099+-0.0030   0.37838+-0.0297   0.66667+-0.0364           *


------------
Matrix of significant Fst P values
Significance Level=0.0500
------------

Number of permutations : 110

                     1         2         3         4         5         6         7         8         9        10        11        12
           1                   -         -         -         -         -         -         -         -         -         +         -
           2         -                   -         -         -         -         -         -         -         -         +         -
           3         -         -                   -         -         -         -         -         -         -         -         -
           4         -         -         -                   -         -         -         -         -         -         -         -
           5         -         -         -         -                   -         -         -         -         -         -         -
           6         -         -         -         -         -                   -         -         -         -         -         -
           7         -         -         -         -         -         -                   -         -         -         -         -
           8         -         -         -         -         -         -         -                   -         -         -         -
           9         -         -         -         -         -         -         -         -                   -         -         -
          10         -         -         -         -         -         -         -         -         -                   -         -
          11         +         +         -         -         -         -         -         -         -         -                   -
          12         -         -         -         -         -         -         -         -         -         -         -          


---------------------------------------
Population average pairwise differences
----------------------------------------

Above diagonal    : Average number of pairwise differences between populations (PiXY)
Diagonal elements : Average number of pairwise differences within population (PiX)
Below diagonal    : Corrected average pairwise difference (PiXY-(PiX+PiY)/2)

Distance method: Pairwise differences
                       1          2          3          4          5          6          7          8          9         10         11         12
            1    7.53718    7.62202    7.56250    7.68359    7.94853    7.73810    7.49931    7.85000    7.70625    7.36250    7.68438    7.31875
            2    0.00965    7.68757    7.63393    7.66964    7.95028    7.82823    7.53638    7.80655    7.66667    7.16667    7.74524    7.66071
            3    0.07818    0.07442    7.43145    7.51367    7.80239    7.71057    7.40365    7.51953    7.39062    7.10938    7.50938    7.10156
            4    0.09041    0.00126   -0.02665    7.64919    7.85662    7.78869    7.45312    7.68359    7.67969    7.20312    7.58125    7.26562
            5    0.13092    0.05747    0.03764   -0.01700    8.09804    8.00210    7.78840    7.99632    7.66176    7.52941    7.87868    7.71324
            6   -0.00204    0.01290    0.02329   -0.00745   -0.01846    7.94309    7.62103    7.83333    7.69643    7.42857    7.77440    7.47024
            7    0.04659    0.00846    0.00379   -0.05560    0.05525   -0.03464    7.36825    7.51042    7.25694    6.94444    7.56250    7.27778
            8    0.00998   -0.10867   -0.26762   -0.21243   -0.12412   -0.20964   -0.24514    8.14286    7.40625    7.37500    7.64062    7.40625
            9    0.10433   -0.01045   -0.15843    0.02176   -0.22059   -0.10845   -0.26052   -0.49851    7.66667    7.12500    7.53125    7.37500
           10    0.09391   -0.17712   -0.10635   -0.12147   -0.01961   -0.04297   -0.23968   -0.19643   -0.20833    7.00000    7.50000    7.37500
           11    0.09591    0.08158   -0.02622   -0.06322    0.00979   -0.01701    0.05850   -0.25068   -0.12196    0.18013    7.63974    7.05000
           12    0.30016    0.56693    0.13584    0.19103    0.41422    0.24869    0.34365    0.08482    0.29167    0.62500   -0.01987    6.50000


-----------
PXY P value
-----------

                         1           2           3           4           5           6           7           8           9          10          11

             2     0.39091
             3     0.25455     0.23636
             4     0.08182     0.41818     0.65455
             5     0.00909     0.06364     0.43636     0.63636
             6     0.62727     0.28182     0.76364     0.72727     0.50000
             7     0.27273     0.56364     0.46364     0.73636     0.16364     0.95455
             8     0.19091     0.49091     0.33636     0.40909     0.75455     0.71818     0.30909
             9     0.31818     0.19091     0.44545     0.20000     0.54545     0.76364     0.66364     1.00000
            10     1.00000     1.00000     1.00000     1.00000     1.00000     1.00000     1.00000     1.00000     1.00000
            11     0.03636     0.03636     0.74545     0.89091     0.25455     0.64545     0.21818     0.52727     0.68182     0.67273
            12     0.66364     0.61818     0.71818     0.77273     0.85455     0.87273     0.47273     0.85455     1.00000     0.38182     0.92727


---------------------
Corrected PXY P value
---------------------

                           1            2            3            4            5            6            7            8            9           10           11

              2      0.36364
              3      0.20909      0.14545
              4      0.06364      0.46364      0.65455
              5      0.05455      0.19091      0.31818      0.51818
              6      0.54545      0.28182      0.36364      0.41818      0.60000
              7      0.21818      0.39091      0.50000      0.82727      0.24545      0.81818
              8      0.56364      0.76364      0.99091      0.83636      0.74545      0.88182      0.94545
              9      0.53636      0.47273      0.90909      0.59091      0.78182      0.58182      0.65455      0.94545
             10      1.00000      0.00000      1.00000      1.00000      1.00000      1.00000      0.00000      1.00000      1.00000
             11      0.03636      0.03636      0.66364      0.90000      0.45455      0.55455      0.15455      0.97273      0.67273      0.53636
             12      0.30000      0.04545      0.30909      0.37273      0.20909      0.14545      0.19091      0.38182      1.00000      0.38182      0.53636

---------------------------------------------------
Matrix of coancestry coefficients as t/M=-ln(1-FST)
(M=N for haploid data, M=2N for diploid data)
---------------------------------------------------

Reference: <i>Reynolds, J., Weir, B.S., and Cockerham, C.C. 1983.</i>
                           1            2            3            4            5            6            7            8            9           10           11           12
              1     -0.00000
              2      0.00125     -0.00000
              3      0.01034      0.00965     -0.00000
              4      0.01189      0.00015            0     -0.00000
              5      0.01682      0.00742      0.00475            0     -0.00000
              6            0      0.00165      0.00276            0            0     -0.00000
              7      0.00620      0.00104      0.00053            0      0.00721            0     -0.00000
              8      0.00535            0            0            0            0            0            0     -0.00000
              9      0.01568            0            0      0.00310            0            0            0            0     -0.00000
             10            0            0            0            0            0            0            0            0            0     -0.00000
             11      0.01256      0.01058            0            0      0.00138            0      0.00771            0            0      0.00319     -0.00000
             12      0.02392      0.05494      0.00420      0.00813      0.02884      0.01037      0.03296            0      0.04035      0.10087            0     -0.00000


----------------------------------------------------
Matrix of Slatkin linearized FSTs as t/M=FST/(1-FST)
(M=N for haploid data, M=2N for diploid data)
----------------------------------------------------

Reference: <i>Slatkin, M., 1995. </i>
                           1            2            3            4            5            6            7            8            9           10           11           12
              1      0.00000
              2      0.00126      0.00000
              3      0.01039      0.00969      0.00000
              4      0.01196      0.00015            0      0.00000
              5      0.01696      0.00745      0.00476            0      0.00000
              6            0      0.00165      0.00277            0            0      0.00000
              7      0.00622      0.00104      0.00053            0      0.00723            0      0.00000
              8      0.00536            0            0            0            0            0            0      0.00000
              9      0.01580            0            0      0.00310            0            0            0            0      0.00000
             10            0            0            0            0            0            0            0            0            0      0.00000
             11      0.01264      0.01064            0            0      0.00138            0      0.00774            0            0      0.00320      0.00000
             12      0.02421      0.05647      0.00421      0.00817      0.02926      0.01042      0.03351            0      0.04118      0.10613            0      0.00000


------------------------------------------------------------------
Matrix of M values (M=Nm for haploid data, M=2Nm for diploid data)
------------------------------------------------------------------

                           1            2            3            4            5            6            7            8            9           10           11           12
              1
              2    398.35464
              3     48.10804     51.58105
              4     41.79482   3439.85260          inf
              5     29.48467     67.11020    105.00717          inf
              6          inf    302.91318    180.73399          inf          inf
              7     80.44353    482.10089    947.37906          inf     69.12653          inf
              8     93.21381          inf          inf          inf          inf          inf          inf
              9     31.64358          inf          inf    161.23971          inf          inf          inf          inf
             10          inf          inf          inf          inf          inf          inf          inf          inf          inf
             11     39.55890     46.99047          inf          inf    363.07844          inf     64.62025          inf          inf    156.28443
             12     20.65058      8.85381    118.67904     61.22647     17.08720     47.96361     14.91932          inf     12.14286      4.71111          inf



==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="18_09_09at08_37_00_samp0">&nbsp;</A>

===============================================================================
== Sample : 	1
===============================================================================


===============================
== Standard diversity indices : (1)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 40
No. of loci              : 16
No. of usable loci       : 16 loci with less than  5.00 % missing data
No. of polymorphic loci  : 16


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        40         3   0.55000   0.58846
     2        40         2   0.40000   0.46667
     3        38         7   0.52632   0.76529
     4        40         2   0.30000   0.32821
     5        40        10   0.95000   0.88462
     6        38         5   0.78947   0.77240
     7        40         9   0.55000   0.77949
     8        40         9   0.55000   0.70128
     9        40         4   0.45000   0.41923
    10        40         7   0.80000   0.65769
    11        40        13   0.95000   0.90641
    12        40         2   0.45000   0.45000
    13        40         3   0.55000   0.68333
    14        40         2   0.05000   0.05000
    15        40         4   0.40000   0.60897
    16        40         6   0.75000   0.77692
-------------------------------------------------
  Mean    39.750     5.500   0.56349   0.61494
  s.d.     0.661     3.279   0.23117   0.21823
-------------------------------------------------



================================
== Molecular diversity indices : (1)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 40.00000
No. of haplotypes                            : 40

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 16
Number of usable loci                        : 16


Average gene diversity over loci              :     0.605449 +/-     0.314087

(Standard deviations are for both sampling and stochastic processes)

Unable to compute theta(Hom) and theta(k) when all gene copies are different
<A NAME="18_09_09at08_37_00_samp1">&nbsp;</A>

===============================================================================
== Sample : 	2
===============================================================================


===============================
== Standard diversity indices : (2)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 42
No. of loci              : 16
No. of usable loci       : 16 loci with less than  5.00 % missing data
No. of polymorphic loci  : 16


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        42         3   0.57143   0.53891
     2        42         2   0.42857   0.49361
     3        42         7   0.85714   0.78746
     4        40         2   0.55000   0.48077
     5        42        11   0.85714   0.88386
     6        42         4   0.66667   0.73055
     7        42         7   0.66667   0.72125
     8        42         8   0.61905   0.61905
     9        42         3   0.33333   0.40767
    10        42         5   0.57143   0.64228
    11        42        13   0.90476   0.89895
    12        42         2   0.52381   0.43786
    13        42         4   0.80952   0.69803
    14        42         2   0.23810   0.21487
    15        42         4   0.57143   0.57027
    16        42         6   0.80952   0.72822
-------------------------------------------------
  Mean    41.875     5.188   0.62366   0.61585
  s.d.     0.484     3.206   0.18632   0.17727
-------------------------------------------------



================================
== Molecular diversity indices : (2)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 42.00000
No. of haplotypes                            : 42

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 16
Number of usable loci                        : 16


Average gene diversity over loci              :     0.613023 +/-     0.317383

(Standard deviations are for both sampling and stochastic processes)

Unable to compute theta(Hom) and theta(k) when all gene copies are different
<A NAME="18_09_09at08_37_00_samp2">&nbsp;</A>

===============================================================================
== Sample : 	7
===============================================================================


===============================
== Standard diversity indices : (7)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 32
No. of loci              : 16
No. of usable loci       : 12 loci with less than  5.00 % missing data
No. of polymorphic loci  : 12


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        32         2   0.31250   0.27218
     2        30         2   0.53333   0.51494
     3        32         7   0.75000   0.72177
     4        32         2   0.31250   0.46573
     5        32         9   0.87500   0.82258
     6        32         6   0.62500   0.77016
     7        32         8   0.68750   0.84879
     8        32         9   0.62500   0.66734
     9        32         4   0.56250   0.48992
    10        30         5   0.86667   0.66437
    11        24        13   0.91667   0.93116
    12        32         2   0.31250   0.41734
    13        32         4   0.68750   0.65726
    14        32         2   0.12500   0.12097
    15        32         4   0.43750   0.59476
    16        30         5   0.73333   0.80690
-------------------------------------------------
  Mean    31.125     5.250   0.58516   0.61038
  s.d.     1.996     3.132   0.22447   0.21296
-------------------------------------------------



================================
== Molecular diversity indices : (7)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 32.00000
No. of haplotypes                            : 32

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 12
Number of usable loci                        : 12


Average gene diversity over loci              :     0.570733 +/-     0.306094

(Standard deviations are for both sampling and stochastic processes)

Unable to compute theta(Hom) and theta(k) when all gene copies are different
<A NAME="18_09_09at08_37_00_samp3">&nbsp;</A>

===============================================================================
== Sample : 	8
===============================================================================


===============================
== Standard diversity indices : (8)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 32
No. of loci              : 16
No. of usable loci       : 8 loci with less than  5.00 % missing data
No. of polymorphic loci  : 8


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        30         3   0.60000   0.47586
     2        30         2   0.66667   0.51494
     3        30         6   0.73333   0.70115
     4        32         3   0.50000   0.52419
     5        32         9   1.00000   0.81250
     6        30         5   0.86667   0.77011
     7        32         6   0.43750   0.80040
     8        32         6   0.68750   0.61895
     9        32         4   0.56250   0.60081
    10        30         5   0.73333   0.66207
    11        28        11   0.78571   0.87831
    12        32         2   0.62500   0.51613
    13        32         3   0.87500   0.65524
    14        30         2   0.26667   0.33103
    15        32         4   0.37500   0.54234
    16        24         5   0.66667   0.63768
-------------------------------------------------
  Mean    30.500     4.750   0.64885   0.62761
  s.d.     2.062     2.437   0.18630   0.13909
-------------------------------------------------



================================
== Molecular diversity indices : (8)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 32.00000
No. of haplotypes                            : 32

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 8
Number of usable loci                        : 8


Average gene diversity over loci              :     0.633821 +/-     0.350502

(Standard deviations are for both sampling and stochastic processes)

Unable to compute theta(Hom) and theta(k) when all gene copies are different
<A NAME="18_09_09at08_37_00_samp4">&nbsp;</A>

===============================================================================
== Sample : 	9
===============================================================================


===============================
== Standard diversity indices : (9)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 34
No. of loci              : 16
No. of usable loci       : 10 loci with less than  5.00 % missing data
No. of polymorphic loci  : 10


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        32         3   0.25000   0.53831
     2        28         2   0.42857   0.50794
     3        32         6   0.56250   0.66331
     4        30         2   0.66667   0.49655
     5        34         8   0.88235   0.84848
     6        34         5   0.76471   0.75758
     7        34         7   0.64706   0.83779
     8        34        10   0.64706   0.78966
     9        34         3   0.41176   0.50089
    10        34         6   0.82353   0.75223
    11        32        10   0.81250   0.86895
    12        34         2   0.41176   0.48663
    13        34         4   0.94118   0.67558
    14        34         2   0.29412   0.33690
    15        34         3   0.76471   0.66488
    16        30         5   0.86667   0.73333
-------------------------------------------------
  Mean    32.750     4.875   0.63595   0.65369
  s.d.     1.854     2.666   0.21251   0.15311
-------------------------------------------------



================================
== Molecular diversity indices : (9)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 34.00000
No. of haplotypes                            : 34

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 10
Number of usable loci                        : 10


Average gene diversity over loci              :     0.665062 +/-     0.356990

(Standard deviations are for both sampling and stochastic processes)

Unable to compute theta(Hom) and theta(k) when all gene copies are different
<A NAME="18_09_09at08_37_00_samp5">&nbsp;</A>

===============================================================================
== Sample : 	10
===============================================================================


===============================
== Standard diversity indices : (10)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 42
No. of loci              : 16
No. of usable loci       : 14 loci with less than  5.00 % missing data
No. of polymorphic loci  : 14


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        42         3   0.47619   0.51568
     2        40         2   0.55000   0.50128
     3        42         8   0.76190   0.77120
     4        40         2   0.45000   0.45000
     5        42         9   0.90476   0.80372
     6        42         6   0.61905   0.76190
     7        42         9   0.85714   0.86992
     8        42        11   0.76190   0.73403
     9        42         4   0.33333   0.43554
    10        38         6   0.78947   0.69132
    11        42        12   0.95238   0.88966
    12        42         2   0.57143   0.48316
    13        42         5   0.61905   0.71661
    14        42         2   0.14286   0.21487
    15        42         4   0.66667   0.66434
    16        38         5   0.68421   0.67425
-------------------------------------------------
  Mean    41.250     5.625   0.63377   0.63609
  s.d.     1.392     3.199   0.20718   0.17869
-------------------------------------------------



================================
== Molecular diversity indices : (10)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 42.00000
No. of haplotypes                            : 42

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 14
Number of usable loci                        : 14


Average gene diversity over loci              :     0.623030 +/-     0.325223

(Standard deviations are for both sampling and stochastic processes)

Unable to compute theta(Hom) and theta(k) when all gene copies are different
<A NAME="18_09_09at08_37_00_samp6">&nbsp;</A>

===============================================================================
== Sample : 	11
===============================================================================


===============================
== Standard diversity indices : (11)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 36
No. of loci              : 16
No. of usable loci       : 11 loci with less than  5.00 % missing data
No. of polymorphic loci  : 11


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        36         3   0.50000   0.40794
     2        32         2   0.31250   0.51411
     3        36         7   0.77778   0.73810
     4        36         3   0.38889   0.45238
     5        36         9   0.88889   0.78254
     6        36         5   0.88889   0.77460
     7        36         7   0.61111   0.83333
     8        36         6   0.50000   0.46984
     9        34         3   0.23529   0.31551
    10        34         6   0.64706   0.71123
    11        34        13   0.94118   0.90909
    12        36         2   0.38889   0.43651
    13        36         3   0.44444   0.62698
    14        36         2   0.33333   0.28571
    15        36         5   0.61111   0.64603
    16        34         6   0.52941   0.72549
-------------------------------------------------
  Mean    35.250     5.125   0.56242   0.60184
  s.d.     1.199     2.891   0.21211   0.18595
-------------------------------------------------



================================
== Molecular diversity indices : (11)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 36.00000
No. of haplotypes                            : 36

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 11
Number of usable loci                        : 11


Average gene diversity over loci              :     0.586724 +/-     0.315315

(Standard deviations are for both sampling and stochastic processes)

Unable to compute theta(Hom) and theta(k) when all gene copies are different
<A NAME="18_09_09at08_37_00_samp7">&nbsp;</A>

===============================================================================
== Sample : 	13
===============================================================================


===============================
== Standard diversity indices : (13)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 8
No. of loci              : 16
No. of usable loci       : 16 loci with less than  5.00 % missing data
No. of polymorphic loci  : 16


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1         8         2   0.25000   0.25000
     2         8         2   0.75000   0.53571
     3         8         3   1.00000   0.67857
     4         8         2   0.75000   0.53571
     5         8         6   0.75000   0.89286
     6         8         2   0.75000   0.53571
     7         8         6   0.75000   0.92857
     8         8         3   0.25000   0.60714
     9         8         2   0.75000   0.53571
    10         8         5   0.75000   0.78571
    11         8         5   1.00000   0.89286
    12         8         2   0.50000   0.42857
    13         8         4   0.75000   0.82143
    14         8         2   0.50000   0.42857
    15         8         3   0.75000   0.71429
    16         8         4   0.75000   0.78571
-------------------------------------------------
  Mean     8.000     3.312   0.68750   0.64732
  s.d.     0.000     1.446   0.20729   0.19019
-------------------------------------------------



================================
== Molecular diversity indices : (13)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 8.00000
No. of haplotypes                            : 8

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 16
Number of usable loci                        : 16


Average gene diversity over loci              :     0.647321 +/-     0.372570

(Standard deviations are for both sampling and stochastic processes)

Unable to compute theta(Hom) and theta(k) when all gene copies are different
<A NAME="18_09_09at08_37_00_samp8">&nbsp;</A>

===============================================================================
== Sample : 	14
===============================================================================


===============================
== Standard diversity indices : (14)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 4
No. of loci              : 16
No. of usable loci       : 15 loci with less than  5.00 % missing data
No. of polymorphic loci  : 13


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     2         4         2   0.50000   0.50000
     3         4         3   0.50000   0.83333
     4         4         2   0.50000   0.50000
     5         4         3   1.00000   0.83333
     6         4         4   1.00000   1.00000
     7         4         3   0.50000   0.83333
     8         4         2   0.50000   0.50000
     9         4         2   0.50000   0.50000
    10         4         3   0.50000   0.83333
    11         4         2   0.50000   0.50000
    12         4         2   0.50000   0.50000
    13         4         2   0.00000   0.66667
    15         4         2   0.00000   0.66667
    16         2         2   1.00000   1.00000
-------------------------------------------------
  Mean     3.875     2.250   0.46875   0.60417
  s.d.     0.484     0.750   0.32924   0.28792
-------------------------------------------------



================================
== Molecular diversity indices : (14)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 4.00000
No. of haplotypes                            : 4

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 13
Number of usable loci                        : 15


Average gene diversity over loci              :     0.577778 +/-     0.394442

(Standard deviations are for both sampling and stochastic processes)

Unable to compute theta(Hom) and theta(k) when all gene copies are different
<A NAME="18_09_09at08_37_00_samp9">&nbsp;</A>

===============================================================================
== Sample : 	15
===============================================================================


===============================
== Standard diversity indices : (15)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 2
No. of loci              : 16
No. of usable loci       : 16 loci with less than  5.00 % missing data
No. of polymorphic loci  : 9


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1         2         2   1.00000   1.00000
     3         2         2   1.00000   1.00000
     4         2         2   1.00000   1.00000
     5         2         2   1.00000   1.00000
     6         2         2   1.00000   1.00000
     8         2         2   1.00000   1.00000
    11         2         2   1.00000   1.00000
    13         2         2   1.00000   1.00000
    16         2         2   1.00000   1.00000
-------------------------------------------------
  Mean     2.000     1.562   0.56250   0.56250
  s.d.     0.000     0.496   0.49608   0.49608
-------------------------------------------------



================================
== Molecular diversity indices : (15)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 2.00000
No. of haplotypes                            : 2

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 9
Number of usable loci                        : 16


Average gene diversity over loci              :     0.562500 +/-     0.557267

(Standard deviations are for both sampling and stochastic processes)

Unable to compute theta(Hom) and theta(k) when all gene copies are different
<A NAME="18_09_09at08_37_00_samp10">&nbsp;</A>

===============================================================================
== Sample : 	19
===============================================================================


===============================
== Standard diversity indices : (19)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 40
No. of loci              : 16
No. of usable loci       : 15 loci with less than  5.00 % missing data
No. of polymorphic loci  : 15


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1        40         3   0.40000   0.45641
     2        38         2   0.78947   0.50782
     3        40         6   0.60000   0.71282
     4        40         2   0.40000   0.46667
     5        38        11   0.89474   0.87482
     6        40         5   0.60000   0.62436
     7        40         9   0.70000   0.85513
     8        40         6   0.80000   0.67308
     9        40         4   0.60000   0.50641
    10        38         5   0.68421   0.61451
    11        40        11   1.00000   0.86538
    12        40         2   0.35000   0.51154
    13        40         4   0.75000   0.62179
    14        40         2   0.20000   0.18462
    15        40         4   0.55000   0.68462
    16        36         6   0.77778   0.72063
-------------------------------------------------
  Mean    39.375     5.125   0.63101   0.61754
  s.d.     1.166     2.891   0.20670   0.17378
-------------------------------------------------



================================
== Molecular diversity indices : (19)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 40.00000
No. of haplotypes                            : 40

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 15
Number of usable loci                        : 15


Average gene diversity over loci              :     0.597521 +/-     0.311654

(Standard deviations are for both sampling and stochastic processes)

Unable to compute theta(Hom) and theta(k) when all gene copies are different
<A NAME="18_09_09at08_37_00_samp11">&nbsp;</A>

===============================================================================
== Sample : 	22
===============================================================================


===============================
== Standard diversity indices : (22)
===============================

Reference: <i>Nei, M., 1987.</i>
No. of gene copies       : 4
No. of loci              : 16
No. of usable loci       : 16 loci with less than  5.00 % missing data
No. of polymorphic loci  : 14


Results are only shown for polymorphic loci
-------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.
Locus#    copies   alleles      Het.       Het
-------------------------------------------------
     1         4         2   0.50000   0.50000
     2         4         2   0.50000   0.50000
     3         4         3   1.00000   0.83333
     5         4         2   1.00000   0.66667
     6         4         2   0.50000   0.50000
     7         4         3   1.00000   0.83333
     8         4         2   0.00000   0.66667
     9         4         2   0.50000   0.50000
    10         4         2   0.50000   0.50000
    11         4         4   1.00000   1.00000
    12         4         2   0.50000   0.50000
    13         4         2   0.50000   0.50000
    15         4         2   0.50000   0.50000
    16         4         4   1.00000   1.00000
-------------------------------------------------
  Mean     4.000     2.250   0.56250   0.56250
  s.d.     0.000     0.829   0.34799   0.27560
-------------------------------------------------



================================
== Molecular diversity indices : (22)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                                  : 4.00000
No. of haplotypes                            : 4

Allowed level of missing data                : 5.00000 %
Number of polymorphic loci                   : 14
Number of usable loci                        : 16


Average gene diversity over loci              :     0.562500 +/-     0.383033

(Standard deviations are for both sampling and stochastic processes)

Unable to compute theta(Hom) and theta(k) when all gene copies are different


================================================================================
== Summary of computations done within populations
================================================================================

<A NAME="18_09_09at08_37_00_comp_sum_Basic">&nbsp;</A>

----------------
Basic properties
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
          Statistics           1           2           7           8           9          10          11          13          14          15          19          22        Mean        s.d.
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  No. of gene copies          40          42          32          32          34          42          36           8           4           2          40           4      26.333      15.829
         No. of loci          16          16          16          16          16          16          16          16          16          16          16          16      16.000       0.000
  No. of usable loci          16          16          12           8          10          14          11          16          15          16          15          16      13.750       2.681
  No. of polym. loci          16          16          12           8          10          14          11          16          13           9          15          14      12.833       2.703
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
<A NAME="18_09_09at08_37_00_comp_sum_het">&nbsp;</A>

-----------------------
Expected heterozygosity
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Locus#           1           2           7           8           9          10          11          13          14          15          19          22        Mean        s.d.   Tot. Het.
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
     1     0.58846     0.53891     0.27218     0.47586     0.53831     0.51568     0.40794     0.25000     0.00000     1.00000     0.45641     0.50000     0.46198     0.22713     0.47525
     2     0.46667     0.49361     0.51494     0.51494     0.50794     0.50128     0.51411     0.53571     0.50000     0.00000     0.50782     0.50000     0.46309     0.14049     0.50159
     3     0.76529     0.78746     0.72177     0.70115     0.66331     0.77120     0.73810     0.67857     0.83333     1.00000     0.71282     0.83333     0.76719     0.08768     0.73093
     4     0.32821     0.48077     0.46573     0.52419     0.49655     0.45000     0.45238     0.53571     0.50000     1.00000     0.46667     0.00000     0.47502     0.21042     0.44843
     5     0.88462     0.88386     0.82258     0.81250     0.84848     0.80372     0.78254     0.89286     0.83333     1.00000     0.87482     0.66667     0.84216     0.07599     0.83773
     6     0.77240     0.73055     0.77016     0.77011     0.75758     0.76190     0.77460     0.53571     1.00000     1.00000     0.62436     0.50000     0.74978     0.14483     0.74656
     7     0.77949     0.72125     0.84879     0.80040     0.83779     0.86992     0.83333     0.92857     0.83333     0.00000     0.85513     0.83333     0.76178     0.23464     0.82469
     8     0.70128     0.61905     0.66734     0.61895     0.78966     0.73403     0.46984     0.60714     0.50000     1.00000     0.67308     0.66667     0.67059     0.13116     0.65797
     9     0.41923     0.40767     0.48992     0.60081     0.50089     0.43554     0.31551     0.53571     0.50000     
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>
<A NAME="18_09_09at08_41_33">&nbsp;</A>
////////////////////////////////////////////////////////////////////
RUN NUMBER 1 (18/09/09 at 08:41:33)
////////////////////////////////////////////////////////////////////

<A NAME="18_09_09at08_41_33_run_information">&nbsp;</A>

Project information:
--------------------
	NbSamples      =  12
	DataType       =  STANDARD
	GenotypicData  =  1
	GameticPhase   =  0
	RecessiveData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Infer haplotype definition from distance matrix
        Alllowed level of missing data = 0.05

 Active Tasks:
 -------------

    Population pairwise Fst values:
    -------------------------------
        Compute pairwise differences
        Compute coancestry coefficients
        Compute Slatkin's distances
        No. of permutations for significance = 100
        No. of permutations for Mantel test  = 1000

       Distance matrix:
            Compute F-statistics on haplotype frequencies only



The computation of AMOVA with the option "CONVENTIONAL FST" is
not possible with unknown gametic phase data.
SKIPPING COMPUTATION OF GENETIC STRUCTURE

 #[WARNING # 1] : unable to compute Genetic Structure with current settings

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 1 (18/09/09 at 08:41:33))
Total computing time for this run : 0h 0m 0s 62 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

