
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>
<A NAME="08_02_09at22_46_50">&nbsp;</A>
////////////////////////////////////////////////////////////////////
RUN NUMBER 1 (08/02/09 at 22:46:50)
////////////////////////////////////////////////////////////////////

<A NAME="08_02_09at22_46_50_run_information">&nbsp;</A>

Project information:
--------------------
	NbSamples      =  19
	DataType       =  MICROSAT
	GenotypicData  =  1
	GameticPhase   =  0
	RecessiveData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Infer haplotype definition from distance matrix
        Alllowed level of missing data = 0.2

 Active Tasks:
 -------------

    Analysis of Molecular Variance:
    -------------------------------
        No. of Permutations = 1000

    Population pairwise Fst values:
    -------------------------------
        Compute pairwise differences
        No. of permutations for significance = 1000
        No. of permutations for Mantel test  = 500

       Distance matrix:
            Compute distance matrix
            Molecular distance : Sum of squared size difference (RST)
            Gamma a value      = 0


    Correlation between distance matrices.
    --------------------------------------------------------

        No. of Permutations for significance = 500
<A NAME="08_02_09at22_46_50_gen_struct">&nbsp;</A>
===============================================================================
== GENETIC STRUCTURE ANALYSIS
===============================================================================


Number of usable loci for distance computation : 1
Allowed level of missing data                  : 0.2

List of usable loci :
---------------------
   1  

List of loci with too much missing data :
-----------------------------------------

NONE

<A NAME="08_02_09at22_46_50_amova">&nbsp;</A>

=================================================================
  AMOVA ANALYSIS
=================================================================


---------------------------
Genetic structure to test :
---------------------------

No. of Groups = 1


	[[Structure]]

		StructureName = "One Group"
		NbGroups = 1
		Group={
		"Handumon"
		"Santander"
		"Boljoon"
		"Argao"
		"Carcar"
		"Minglanilla"
		"Danao"
		"Sogod"
		"Tabogon"
		"Daanbantayan"
		"Malapascua"
		"Pintuyan"
		"Padre Burgos"
		"Maasin"
		"Inopacan"
		"Baybay"
		"Albuera"
		"Ormoc"
		"Palompon"
		}

---------------------------


Distance method: Sum of squared size difference (RST)

--------------------------
AMOVA design and results :
--------------------------

Reference: <i>Weir, B.S. and Cockerham, C.C. 1984.</i>
           <i>Excoffier, L., Smouse, P., and Quattro, J. 1992.</i>
           <i>Weir, B. S., 1996.</i>
----------------------------------------------------------------------
 Source of                  Sum of      Variance         Percentage
 variation      d.f.        squares     components       of variation
----------------------------------------------------------------------
 Among
 populations     18         27.122        0.00589 Va             0.46

 Within
 populations    725        925.747        1.27689 Vb            99.54
----------------------------------------------------------------------
 Total          743        952.870        1.28278
----------------------------------------------------------------------
 Fixation Index      FST :      0.00459
----------------------------------------------------------------------

Significance tests (1023 permutations)
------------------

Va and FST : P(rand. value > obs. value)  =  0.37928
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.37928+-0.01240

<A NAME="08_02_09at22_46_50_pairw_diff">&nbsp;</A>

======================================================================================
== Comparisons of pairs of population samples
======================================================================================

List of labels for population samples used below:
-------------------------------------------------

Label  	Population name
-----  	---------------
  1:	Handumon
  2:	Santander
  3:	Boljoon
  4:	Argao
  5:	Carcar
  6:	Minglanilla
  7:	Danao
  8:	Sogod
  9:	Tabogon
  10:	Daanbantayan
  11:	Malapascua
  12:	Pintuyan
  13:	Padre Burgos
  14:	Maasin
  15:	Inopacan
  16:	Baybay
  17:	Albuera
  18:	Ormoc
  19:	Palompon

------------------------
Population pairwise FSTs
------------------------


Distance method: Sum of squared size difference (RST)
                     1         2         3         4         5         6         7         8         9        10        11        12        13        14        15        16        17        18        19
           1   0.00000
           2   0.00023   0.00000
           3  -0.02017  -0.02050   0.00000
           4  -0.05042   0.01145  -0.00839   0.00000
           5  -0.05037   0.03780   0.01188  -0.02484   0.00000
           6  -0.05065   0.03507   0.00902  -0.02543  -0.02524   0.00000
           7  -0.03424  -0.00834  -0.02329  -0.01880  -0.00532  -0.00817   0.00000
           8   0.06067  -0.02160  -0.00339   0.04665   0.09864   0.09803   0.02208   0.00000
           9  -0.04305   0.01114  -0.01114  -0.02376  -0.01612  -0.01868  -0.02244   0.05946   0.00000
          10  -0.03300  -0.00786  -0.02315  -0.01858  -0.00430  -0.00719  -0.02703   0.02368  -0.02224   0.00000
          11  -0.05021   0.04562   0.01866  -0.02472  -0.02564  -0.02510  -0.00055   0.11777  -0.01315   0.00100   0.00000
          12  -0.04834  -0.00777  -0.02232  -0.02559  -0.02050  -0.02232  -0.02777   0.01828  -0.02766  -0.02774  -0.01940   0.00000
          13  -0.01549  -0.01725  -0.02678  -0.00945   0.01419   0.01144  -0.02444   0.00579  -0.01180  -0.02433   0.02369  -0.02349   0.00000
          14  -0.01085  -0.01310  -0.02590  -0.01120   0.01503   0.01251  -0.02565   0.01685  -0.01338  -0.02558   0.02767  -0.02485  -0.02667   0.00000
          15  -0.00876  -0.01467  -0.02114   0.00135   0.02173   0.01804  -0.01632   0.00275  -0.00319  -0.01619   0.02724  -0.01453  -0.02067  -0.01951   0.00000
          16  -0.03858   0.07088   0.04469  -0.01829  -0.01927  -0.01375   0.02415   0.15168   0.01058   0.02699  -0.01669  -0.00673   0.05474   0.06389   0.05490   0.00000
          17  -0.04454   0.05875   0.03192  -0.02056  -0.02235  -0.01904   0.01228   0.12939  -0.00091   0.01416  -0.02138  -0.01148   0.03767   0.04181   0.04273  -0.02580   0.00000
          18  -0.04869   0.05150   0.02447  -0.02361  -0.02520  -0.02358   0.00474   0.12525  -0.00844   0.00654  -0.02503  -0.01668   0.03040   0.03522   0.03346  -0.02104  -0.02381   0.00000
          19  -0.04945   0.04142   0.01547  -0.02392  -0.02531  -0.02417  -0.00194   0.10154  -0.01311  -0.00084  -0.02523  -0.01841   0.01792   0.01881   0.02605  -0.02266  -0.02429  -0.02564   0.00000


------------
FST P values
------------

Number of permutations : 1023

                       1                 2                 3                 4                 5                 6                 7                 8                 9                10                11                12                13                14                15                16                17                18                19
           1           *
           2   0.40234+-0.0153           *
           3   0.41016+-0.0151   0.67480+-0.0159           *
           4   0.99902+-0.0002   0.36914+-0.0126   0.50977+-0.0143           *
           5   0.99902+-0.0002   0.19727+-0.0126   0.26758+-0.0139   0.94629+-0.0064           *
           6   0.99902+-0.0002   0.18848+-0.0110   0.20898+-0.0137   0.99902+-0.0002   0.99902+-0.0002           *
           7   0.71191+-0.0121   0.49512+-0.0180   0.78223+-0.0117   0.66797+-0.0132   0.48535+-0.0153   0.47168+-0.0147           *
           8   0.22168+-0.0128   0.79883+-0.0125   0.41504+-0.0144   0.12402+-0.0100   0.05078+-0.0066   0.03516+-0.0055   0.24219+-0.0125           *
           9   0.70215+-0.0169   0.32031+-0.0159   0.45898+-0.0158   0.87793+-0.0078   0.68555+-0.0139   0.67871+-0.0122   0.75586+-0.0097   0.07520+-0.0089           *
          10   0.60645+-0.0151   0.45410+-0.0164   0.74316+-0.0134   0.64746+-0.0128   0.40332+-0.0132   0.34473+-0.0162   0.99902+-0.0002   0.19824+-0.0130   0.68848+-0.0126           *
          11   0.99902+-0.0002   0.13965+-0.0109   0.20312+-0.0143   0.92969+-0.0075   0.99902+-0.0002   0.99902+-0.0002   0.40625+-0.0156   0.01660+-0.0042   0.54395+-0.0181   0.28711+-0.0149           *
          12   0.86133+-0.0106   0.47363+-0.0132   0.70117+-0.0144   0.83496+-0.0130   0.72266+-0.0142   0.73438+-0.0144   0.85449+-0.0119   0.25781+-0.0151   0.92285+-0.0081   0.87891+-0.0123   0.67773+-0.0134           *
          13   0.48926+-0.0176   0.60547+-0.0109   0.99902+-0.0002   0.53027+-0.0129   0.28516+-0.0125   0.29297+-0.0157   0.86816+-0.0121   0.35938+-0.0161   0.51953+-0.0168   0.81152+-0.0120   0.20312+-0.0127   0.78711+-0.0133           *
          14   0.52051+-0.0143   0.56055+-0.0146   0.92773+-0.0070   0.54590+-0.0138   0.31641+-0.0149   0.23047+-0.0142   0.91016+-0.0087   0.26367+-0.0140   0.52832+-0.0143   0.90723+-0.0098   0.17871+-0.0138   0.78711+-0.0091   0.99902+-0.0002           *
          15   0.40918+-0.0164   0.66895+-0.0130   0.99902+-0.0002   0.37402+-0.0177   0.20117+-0.0133   0.21973+-0.0126   0.67871+-0.0118   0.35449+-0.0160   0.38574+-0.0180   0.66992+-0.0154   0.17773+-0.0111   0.61328+-0.0161   0.93262+-0.0069   0.79199+-0.0127           *
          16   0.66992+-0.0176   0.09082+-0.0092   0.12891+-0.0107   0.69238+-0.0131   0.73730+-0.0106   0.56934+-0.0142   0.25684+-0.0124   0.01172+-0.0033   0.27930+-0.0135   0.15039+-0.0106   0.65820+-0.0170   0.44336+-0.0137   0.14258+-0.0105   0.09863+-0.0107   0.09570+-0.0093           *
          17   0.85938+-0.0118   0.09277+-0.0089   0.10938+-0.0109   0.72656+-0.0135   0.83398+-0.0156   0.64258+-0.0141   0.26855+-0.0150   0.01758+-0.0042   0.31641+-0.0120   0.15820+-0.0117   0.76074+-0.0135   0.47754+-0.0166   0.14258+-0.0103   0.12207+-0.0115   0.08984+-0.0105   0.87988+-0.0105           *
          18   0.99902+-0.0002   0.15137+-0.0112   0.16699+-0.0122   0.86621+-0.0103   0.99902+-0.0002   0.89258+-0.0097   0.33789+-0.0111   0.01367+-0.0042   0.49414+-0.0169   0.22949+-0.0140   0.99902+-0.0002   0.59668+-0.0108   0.19336+-0.0117   0.15430+-0.0130   0.12793+-0.0086   0.66406+-0.0135   0.87793+-0.0103           *
          19   0.99902+-0.0002   0.15918+-0.0118   0.20117+-0.0117   0.86230+-0.0126   0.99902+-0.0002   0.90234+-0.0072   0.39746+-0.0169   0.03320+-0.0049   0.50488+-0.0178   0.28809+-0.0116   0.99902+-0.0002   0.63672+-0.0163   0.23047+-0.0114   0.20020+-0.0111   0.16699+-0.0120   0.68848+-0.0161   0.90430+-0.0120   0.99902+-0.0002           *


------------
Matrix of significant Fst P values
Significance Level=0.0500
------------

Number of permutations : 1023

                     1         2         3         4         5         6         7         8         9        10        11        12        13        14        15        16        17        18        19
           1                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           2         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           3         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           4         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           5         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -
           6         -         -         -         -         -                   -         +         -         -         -         -         -         -         -         -         -         -         -
           7         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -
           8         -         -         -         -         -         +         -                   -         -         +         -         -         -         -         +         +         +         +
           9         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -
          10         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -
          11         -         -         -         -         -         -         -         +         -         -                   -         -         -         -         -         -         -         -
          12         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -
          13         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         -
          14         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -         -
          15         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -
          16         -         -         -         -         -         -         -         +         -         -         -         -         -         -         -                   -         -         -
          17         -         -         -         -         -         -         -         +         -         -         -         -         -         -         -         -                   -         -
          18         -         -         -         -         -         -         -         +         -         -         -         -         -         -         -         -         -                   -
          19         -         -         -         -         -         -         -         +         -         -         -         -         -         -         -         -         -         -          


---------------------------------------
Population average pairwise differences
----------------------------------------

Above diagonal    : Average number of pairwise differences between populations (PiXY)
Diagonal elements : Average number of pairwise differences within population (PiX)
Below diagonal    : Corrected average pairwise difference (PiXY-(PiX+PiY)/2)

Distance method: Sum of squared size difference (RST)
                       1          2          3          4          5          6          7          8          9         10         11         12         13         14         15         16         17         18         19
            1    1.45055    3.40357    2.50752    2.96429    1.57143    1.36786    1.99624    2.64626    1.45357    1.89098    1.17143    3.25210    1.93233    1.51504    2.23810    1.30451    1.53214    1.22500    1.72500
            2    0.08919    5.17821    4.29474    4.99500    3.64750    3.42125    3.87895    4.18452    3.41625    3.77368    3.24750    5.18676    3.74342    3.35000    4.01591    3.47368    3.67625    3.32375    3.82375
            3   -0.01079   -0.08740    3.58606    4.10789    2.74474    2.52632    3.04294    3.46742    2.55263    2.93767    2.34474    4.33282    2.93213    2.53047    3.21611    2.53878    2.75000    2.41316    2.91316
            4   -0.10971    0.05718   -0.03385    4.69744    3.19000    2.98500    3.60263    4.23095    3.06500    3.49737    2.79000    4.86176    3.53421    3.11842    3.83788    2.92895    3.15500    2.84500    3.34500
            5   -0.07436    0.13788    0.03119   -0.07923    1.84103    1.59250    2.22895    2.89524    1.68250    2.12368    1.39500    3.48235    2.16842    1.75000    2.47576    1.52368    1.75250    1.44750    1.94750
            6   -0.06960    0.11997    0.02111   -0.07590   -0.04019    1.42436    2.01579    2.66310    1.47375    1.91053    1.19250    3.27206    1.95132    1.53421    2.25682    1.32632    1.55375    1.24625    1.74625
            7   -0.04980   -0.03092   -0.07086   -0.06685   -0.01233   -0.01716    2.64154    3.07644    2.06316    2.46676    1.82895    3.84985    2.47784    2.07064    2.76954    2.00139    2.21842    1.89211    2.39211
            8    0.23457   -0.09099   -0.01202    0.19582    0.28831    0.26450    0.06926    3.37282    2.63452    2.97118    2.49524    4.39776    2.92231    2.53509    3.18615    2.74561    2.94167    2.57738    3.07738
            9   -0.06081    0.03804   -0.02950   -0.07282   -0.02712   -0.02753   -0.04671    0.15901    1.57821    1.95789    1.28250    3.33088    1.98289    1.57105    2.28106    1.43684    1.65875    1.34125    1.84125
           10   -0.04696   -0.02808   -0.06802   -0.06401   -0.00949   -0.01431   -0.06667    0.07211   -0.04387    2.42532    1.72368    3.74458    2.37258    1.96537    2.66427    1.89612    2.11316    1.78684    2.28684
           11   -0.06410    0.14814    0.04145   -0.06897   -0.03577   -0.02994   -0.00208    0.29857   -0.01686    0.00077    1.02051    3.08235    1.76842    1.35000    2.07576    1.12368    1.35250    1.04750    1.54750
           12   -0.11132   -0.04048   -0.09836   -0.12510   -0.07631   -0.07827   -0.10907    0.07320   -0.09637   -0.10622   -0.06605    5.27629    3.76471    3.35449    4.06061    3.24303    3.46324    3.14265    3.64265
           13   -0.01058   -0.06332   -0.07854   -0.03215    0.03027    0.02150   -0.06057    0.01826   -0.02385   -0.05772    0.04053   -0.09108    2.43528    1.96814    2.65710    1.95706    2.16974    1.83553    2.33553
           14   -0.01322   -0.04209   -0.06555   -0.03328    0.02650    0.01904   -0.05312    0.04569   -0.02104   -0.05027    0.03676   -0.08664   -0.05248    1.60597    2.25598    1.53324    1.74737    1.41579    1.91579
           15    0.02005   -0.06597   -0.06970   -0.00361    0.06247    0.05186   -0.04400    0.00696   -0.00082   -0.04116    0.07273   -0.07031   -0.05332   -0.03978    2.98555    2.27193    2.48258    2.14470    2.64470
           16   -0.05305    0.25229    0.11346   -0.05206   -0.02912   -0.01815    0.04833    0.42691    0.01545    0.05117   -0.01886   -0.02740    0.10713    0.09796    0.14687    1.26458    1.46579    1.17105    1.67105
           17   -0.06429    0.21599    0.08582   -0.06487   -0.03917   -0.02958    0.02650    0.38410   -0.00151    0.02934   -0.02891   -0.04606    0.08094    0.07323    0.11865   -0.03765    1.74231    1.40125    1.90125
           18   -0.06374    0.17119    0.05667   -0.06718   -0.03647   -0.02939    0.00788    0.32751   -0.01131    0.01072   -0.02622   -0.05896    0.05443    0.04934    0.08846   -0.02470   -0.03337    1.12692    1.59875
           19   -0.07656    0.15837    0.04385   -0.08000   -0.04929   -0.04221   -0.00494    0.31469   -0.02413   -0.00210   -0.03904   -0.07178    0.04161    0.03652    0.07564   -0.03752   -0.04619   -0.04099    2.15256


-----------
PXY P value
-----------

                         1           2           3           4           5           6           7           8           9          10          11          12          13          14          15          16          17          18

             2     0.78104
             3     0.89736     0.92864
             4     0.72434     0.36950     0.70186
             5     0.70381     0.19746     0.17693     0.94721
             6     0.64418     0.18866     0.11046     1.00000     1.00000
             7     0.72532     0.83578     0.78299     0.77908     0.37439     0.31281
             8     0.93646     0.54545     0.37146     0.11046     0.05670     0.05963     0.31574
             9     0.63441     0.32063     0.24633     0.87879     0.68622     0.67937     0.49658     0.11046
            10     0.88465     0.83675     0.74389     0.80645     0.33920     0.22385     1.00000     0.30792     0.39785
            11     0.46041     0.13978     0.09580     0.93060     1.00000     1.00000     0.23460     0.02835     0.54448     0.13881
            12     0.77322     0.60215     0.58358     0.60117     0.28152     0.18280     0.52786     0.27077     0.24731     0.46628     0.10948
            13     0.63050     0.97067     1.00000     0.73705     0.24829     0.20528     0.86901     0.52786     0.37537     0.81232     0.13881     0.49071
            14     0.58651     0.98436     0.92864     0.78299     0.33040     0.21408     0.91105     0.67351     0.54545     0.90811     0.13099     0.32942     1.00000
            15     0.91007     0.16422     0.35777     0.23265     0.83773     0.94526     0.67058     0.33431     0.98534     0.77615     0.99707     0.25024     0.72825     0.99413
            16     0.43402     0.16813     0.12903     0.87195     0.87586     0.63832     0.25709     0.03030     0.30010     0.15054     0.50440     0.13001     0.14272     0.09873     0.92180
            17     0.68035     0.09286     0.07625     0.72727     0.83480     0.64321     0.19648     0.02444     0.31672     0.12903     0.76149     0.13392     0.11730     0.12317     0.73118     0.98436
            18     0.43695     0.15152     0.08993     0.86706     1.00000     0.89345     0.22092     0.01857     0.49462     0.12708     1.00000     0.12023     0.13099     0.12805     0.97947     0.67644     0.87879
            19     0.86510     0.15934     0.15152     0.86315     1.00000     0.90323     0.34995     0.03910     0.50538     0.26491     1.00000     0.29619     0.20821     0.24340     0.67155     0.97752     0.90518     1.00000


---------------------
Corrected PXY P value
---------------------

                           1            2            3            4            5            6            7            8            9           10           11           12           13           14           15           16           17           18

              2      0.34213
              3      0.32747      0.63441
              4      0.82502      0.36950      0.49267
              5      0.85728      0.19746      0.25709      0.90811
              6      0.92278      0.18866      0.20332      0.95601      0.99413
              7      0.57185      0.46334      0.78299      0.63832      0.46530      0.45943
              8      0.15054      0.79863      0.41056      0.12121      0.04399      0.02835      0.23851
              9      0.66862      0.32063      0.45650      0.84751      0.67351      0.67351      0.74878      0.06647
             10      0.40567      0.41251      0.74389      0.61290      0.36950      0.34311      1.00000      0.18280      0.68719
             11      0.94428      0.13978      0.20235      0.89443      0.98827      1.00000      0.39980      0.01466      0.52786      0.28641
             12      0.68133      0.45161      0.64516      0.82405      0.74878      0.82502      0.91007      0.24731      0.98045      0.93842      0.72630
             13      0.41642      0.56989      0.94917      0.47898      0.26295      0.27273      0.86901      0.29814      0.49560      0.81232      0.19453      0.76149
             14      0.42717      0.50244      0.84360      0.49951      0.27761      0.20626      0.91105      0.22483      0.47996      0.90811      0.17107      0.84164      0.91887
             15      0.32160      0.78592      0.95894      0.36852      0.17889      0.18866      0.62561      0.34409      0.34995      0.60215      0.14858      0.78788      0.83187      0.65103
             16      0.71750      0.08113      0.12903      0.60997      0.60508      0.50440      0.25709      0.00978      0.27566      0.15054      0.61290      0.45161      0.12121      0.09873      0.06745
             17      0.75758      0.09286      0.09677      0.72727      0.78592      0.64027      0.23656      0.01760      0.28837      0.13490      0.74096      0.48974      0.12610      0.08993      0.08211      0.80645
             18      0.92473      0.15152      0.16422      0.86706      0.89638      0.88563      0.32649      0.01075      0.42717      0.22678      0.87781      0.61975      0.17986      0.14076      0.10264      0.67449      0.70479
             19      0.71359      0.15934      0.18573      0.85337      0.89541      0.89834      0.35973      0.03324      0.43402      0.25220      0.86217      0.65005      0.19746      0.16618      0.15543      0.61486      0.82991      0.89736


////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 1 (08/02/09 at 22:46:51))
Total computing time for this run : 0h 0m 1s 281 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>


<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>
<A NAME="08_02_09at23_54_34">&nbsp;</A>
////////////////////////////////////////////////////////////////////
RUN NUMBER 1 (08/02/09 at 23:54:34)
////////////////////////////////////////////////////////////////////

<A NAME="08_02_09at23_54_34_run_information">&nbsp;</A>

Project information:
--------------------
	NbSamples      =  19
	DataType       =  MICROSAT
	GenotypicData  =  1
	GameticPhase   =  0
	RecessiveData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Infer haplotype definition from distance matrix
        Alllowed level of missing data = 0.2

 Active Tasks:
 -------------

    Analysis of Molecular Variance:
    -------------------------------
        No. of Permutations = 1000

    Population pairwise Fst values:
    -------------------------------
        Compute pairwise differences
        No. of permutations for significance = 1000
        No. of permutations for Mantel test  = 500

       Distance matrix:
            Compute distance matrix
            Molecular distance : No. of different alleles (FST)
            Gamma a value      = 0


    Correlation between distance matrices.
    --------------------------------------------------------

        No. of Permutations for significance = 500
<A NAME="08_02_09at23_54_34_gen_struct">&nbsp;</A>
===============================================================================
== GENETIC STRUCTURE ANALYSIS
===============================================================================


Number of usable loci for distance computation : 1
Allowed level of missing data                  : 0.2

List of usable loci :
---------------------
   1  

List of loci with too much missing data :
-----------------------------------------

NONE

<A NAME="08_02_09at23_54_34_amova">&nbsp;</A>

=================================================================
  AMOVA ANALYSIS
=================================================================


---------------------------
Genetic structure to test :
---------------------------

No. of Groups = 1


	[[Structure]]

		StructureName = "One Group"
		NbGroups = 1
		Group={
		"Handumon"
		"Santander"
		"Boljoon"
		"Argao"
		"Carcar"
		"Minglanilla"
		"Danao"
		"Sogod"
		"Tabogon"
		"Daanbantayan"
		"Malapascua"
		"Pintuyan"
		"Padre Burgos"
		"Maasin"
		"Inopacan"
		"Baybay"
		"Albuera"
		"Ormoc"
		"Palompon"
		}

---------------------------


Distance method: No. of different alleles (FST)

--------------------------
AMOVA design and results :
--------------------------

Reference: <i>Weir, B.S. and Cockerham, C.C. 1984.</i>
           <i>Excoffier, L., Smouse, P., and Quattro, J. 1992.</i>
           <i>Weir, B. S., 1996.</i>
----------------------------------------------------------------------
 Source of                  Sum of      Variance         Percentage
 variation      d.f.        squares     components       of variation
----------------------------------------------------------------------
 Among
 populations     18          7.239        0.00265 Va             0.88

 Within
 populations    725        216.464        0.29857 Vb            99.12
----------------------------------------------------------------------
 Total          743        223.703        0.30122
----------------------------------------------------------------------
 Fixation Index      FST :      0.00880
----------------------------------------------------------------------

Significance tests (1023 permutations)
------------------

Va and FST : P(rand. value > obs. value)  =  0.08993
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.08993+-0.00872

<A NAME="08_02_09at23_54_34_pairw_diff">&nbsp;</A>

======================================================================================
== Comparisons of pairs of population samples
======================================================================================

List of labels for population samples used below:
-------------------------------------------------

Label  	Population name
-----  	---------------
  1:	Handumon
  2:	Santander
  3:	Boljoon
  4:	Argao
  5:	Carcar
  6:	Minglanilla
  7:	Danao
  8:	Sogod
  9:	Tabogon
  10:	Daanbantayan
  11:	Malapascua
  12:	Pintuyan
  13:	Padre Burgos
  14:	Maasin
  15:	Inopacan
  16:	Baybay
  17:	Albuera
  18:	Ormoc
  19:	Palompon

------------------------
Population pairwise FSTs
------------------------


Distance method: No. of different alleles (FST)
                     1         2         3         4         5         6         7         8         9        10        11        12        13        14        15        16        17        18        19
           1   0.00000
           2  -0.02743   0.00000
           3  -0.03832  -0.00778   0.00000
           4  -0.04445  -0.01228  -0.01606   0.00000
           5  -0.03982   0.00092  -0.02180  -0.01533   0.00000
           6  -0.05058  -0.00174  -0.01437  -0.01906  -0.01573   0.00000
           7  -0.04376  -0.00709  -0.02586  -0.01979  -0.02344  -0.01950   0.00000
           8   0.01766   0.06633   0.01816   0.02995   0.02102   0.04061   0.02037   0.00000
           9  -0.04635  -0.00684  -0.01698  -0.02353  -0.01825  -0.02114  -0.02193   0.03106   0.00000
          10  -0.03070   0.02242  -0.01760  -0.00369  -0.01594  -0.00804  -0.01751  -0.00186  -0.00654   0.00000
          11  -0.04544  -0.00833  -0.01055  -0.01913  -0.01260  -0.01982  -0.01670   0.05865  -0.02230   0.00701   0.00000
          12  -0.01001   0.00962   0.03114  -0.00434   0.02969   0.01750   0.02367   0.06764   0.00426   0.05065   0.01047   0.00000
          13  -0.03303  -0.00680   0.00331  -0.01935   0.00265  -0.00651  -0.00304   0.04771  -0.01595   0.02053  -0.01175  -0.02178   0.00000
          14  -0.02333   0.00258   0.00930  -0.01382   0.00666   0.00343   0.00209   0.04821  -0.01253   0.02726  -0.00721  -0.01893  -0.02362   0.00000
          15  -0.02301   0.01699  -0.01415  -0.00481  -0.01275   0.00020  -0.01369  -0.00224  -0.00598  -0.01647   0.00817   0.03657   0.01247   0.01476   0.00000
          16  -0.02167   0.03168  -0.00649   0.01715  -0.00504  -0.00078  -0.00625   0.05755   0.00998  -0.00591   0.01064   0.09570   0.05262   0.06335   0.01270   0.00000
          17   0.01172   0.07003   0.01474   0.04987   0.01783   0.02795   0.01818   0.06010   0.04305   0.00242   0.05011   0.14229   0.09460   0.10780   0.02703  -0.01687   0.00000
          18  -0.03917   0.00138  -0.01772  -0.00938  -0.01793  -0.01512  -0.02057   0.05098  -0.01560  -0.00451  -0.01753   0.04439   0.01121   0.01526   0.00182  -0.01220   0.01709   0.00000
          19  -0.00337   0.05432   0.00163   0.03176   0.00247   0.01581   0.00480   0.03239   0.02676  -0.01184   0.03732   0.11322   0.07119   0.08274   0.00870  -0.01538  -0.02153   0.01001   0.00000


------------
FST P values
------------

Number of permutations : 1023

                       1                 2                 3                 4                 5                 6                 7                 8                 9                10                11                12                13                14                15                16                17                18                19
           1           *
           2   0.72656+-0.0155           *
           3   0.89062+-0.0092   0.54688+-0.0160           *
           4   0.99902+-0.0002   0.71777+-0.0143   0.75293+-0.0128           *
           5   0.96973+-0.0052   0.44922+-0.0115   0.97656+-0.0049   0.85254+-0.0147           *
           6   0.99902+-0.0002   0.44531+-0.0148   0.67480+-0.0134   0.87793+-0.0101   0.81543+-0.0167           *
           7   0.97266+-0.0052   0.51953+-0.0160   0.99902+-0.0002   0.89258+-0.0107   0.99902+-0.0002   0.84961+-0.0110           *
           8   0.29199+-0.0129   0.02930+-0.0046   0.16992+-0.0121   0.10645+-0.0048   0.20410+-0.0124   0.09473+-0.0092   0.17969+-0.0121           *
           9   0.99902+-0.0002   0.51270+-0.0135   0.68555+-0.0149   0.99902+-0.0002   0.90332+-0.0094   0.96387+-0.0055   0.93359+-0.0104   0.08887+-0.0089           *
          10   0.69434+-0.0172   0.12598+-0.0092   0.80957+-0.0115   0.43359+-0.0139   0.76953+-0.0143   0.47852+-0.0100   0.79980+-0.0112   0.44336+-0.0165   0.37793+-0.0166           *
          11   0.99902+-0.0002   0.49121+-0.0159   0.52344+-0.0139   0.87598+-0.0089   0.74414+-0.0127   0.84961+-0.0109   0.71582+-0.0164   0.02148+-0.0038   0.93164+-0.0061   0.18555+-0.0123           *
          12   0.51758+-0.0168   0.28711+-0.0128   0.08691+-0.0077   0.49512+-0.0146   0.14355+-0.0108   0.23535+-0.0124   0.17383+-0.0153   0.05273+-0.0056   0.29004+-0.0147   0.06641+-0.0062   0.24316+-0.0141           *
          13   0.76465+-0.0122   0.51465+-0.0176   0.35156+-0.0140   0.83301+-0.0117   0.39648+-0.0148   0.51270+-0.0180   0.43066+-0.0154   0.07617+-0.0083   0.68164+-0.0160   0.17188+-0.0132   0.54492+-0.0166   0.88965+-0.0101           *
          14   0.63086+-0.0153   0.35352+-0.0142   0.25391+-0.0116   0.67871+-0.0172   0.32324+-0.0169   0.36328+-0.0163   0.35840+-0.0187   0.08203+-0.0086   0.60840+-0.0173   0.12891+-0.0129   0.46777+-0.0182   0.82324+-0.0114   0.97852+-0.0043           *
          15   0.70898+-0.0148   0.16113+-0.0123   0.81348+-0.0128   0.52930+-0.0110   0.82910+-0.0128   0.39258+-0.0150   0.80371+-0.0155   0.48535+-0.0144   0.51660+-0.0148   0.87402+-0.0095   0.24316+-0.0102   0.08594+-0.0080   0.24219+-0.0141   0.18359+-0.0153           *
          16   0.52148+-0.0167   0.12695+-0.0104   0.46875+-0.0153   0.20801+-0.0153   0.49023+-0.0159   0.41309+-0.0126   0.49121+-0.0164   0.05371+-0.0059   0.20605+-0.0118   0.43555+-0.0154   0.22852+-0.0124   0.02246+-0.0047   0.09082+-0.0074   0.06348+-0.0086   0.20312+-0.0100           *
          17   0.23438+-0.0107   0.01758+-0.0042   0.12988+-0.0101   0.06152+-0.0076   0.14941+-0.0114   0.14551+-0.0125   0.13965+-0.0101   0.02344+-0.0039   0.03613+-0.0050   0.25000+-0.0166   0.03223+-0.0046   0.00488+-0.0020   0.01074+-0.0036   0.01074+-0.0033   0.08984+-0.0104   0.61816+-0.0123           *
          18   0.83691+-0.0117   0.34375+-0.0178   0.76855+-0.0106   0.52637+-0.0150   0.91699+-0.0107   0.67285+-0.0140   0.89453+-0.0094   0.05176+-0.0068   0.68262+-0.0130   0.38770+-0.0144   0.79297+-0.0108   0.08398+-0.0086   0.23438+-0.0143   0.21680+-0.0122   0.33691+-0.0154   0.54785+-0.0178   0.15527+-0.0110           *
          19   0.40137+-0.0145   0.02832+-0.0042   0.24023+-0.0130   0.10742+-0.0111   0.33008+-0.0145   0.20508+-0.0119   0.28320+-0.0135   0.10352+-0.0105   0.08398+-0.0094   0.52637+-0.0211   0.04883+-0.0078   0.01172+-0.0027   0.03711+-0.0052   0.02344+-0.0050   0.22168+-0.0164   0.64355+-0.0156   0.91602+-0.0074   0.19043+-0.0132           *


------------
Matrix of significant Fst P values
Significance Level=0.0500
------------

Number of permutations : 1023

                     1         2         3         4         5         6         7         8         9        10        11        12        13        14        15        16        17        18        19
           1                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           2         -                   -         -         -         -         -         +         -         -         -         -         -         -         -         -         +         -         +
           3         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           4         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -         -
           5         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -         -
           6         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -         -
           7         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -         -         -         -
           8         -         +         -         -         -         -         -                   -         -         +         -         -         -         -         -         +         -         -
           9         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -         +         -         -
          10         -         -         -         -         -         -         -         -         -                   -         -         -         -         -         -         -         -         -
          11         -         -         -         -         -         -         -         +         -         -                   -         -         -         -         -         +         -         +
          12         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         +         +         -         +
          13         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         +         -         +
          14         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         +         -         +
          15         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -         -         -         -
          16         -         -         -         -         -         -         -         -         -         -         -         +         -         -         -                   -         -         -
          17         -         +         -         -         -         -         -         +         +         -         +         +         +         +         -         -                   -         -
          18         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -         -                   -
          19         -         +         -         -         -         -         -         -         -         -         +         +         +         +         -         -         -         -          


---------------------------------------
Population average pairwise differences
----------------------------------------

Above diagonal    : Average number of pairwise differences between populations (PiXY)
Diagonal elements : Average number of pairwise differences within population (PiX)
Below diagonal    : Corrected average pairwise difference (PiXY-(PiX+PiY)/2)

Distance method: No. of different alleles (FST)
                       1          2          3          4          5          6          7          8          9         10         11         12         13         14         15         16         17         18         19
            1    0.58242    0.58929    0.58459    0.58393    0.58214    0.54107    0.57519    0.63946    0.56964    0.57331    0.55000    0.60714    0.58835    0.60338    0.60823    0.52068    0.51607    0.54821    0.54286
            2   -0.01538    0.62692    0.62368    0.62375    0.62750    0.59000    0.61776    0.69405    0.61375    0.62697    0.59250    0.64044    0.62500    0.64013    0.65417    0.57434    0.57625    0.59313    0.60125
            3   -0.02170   -0.00486    0.63016    0.62303    0.61513    0.58421    0.60803    0.66165    0.60921    0.60388    0.59276    0.65635    0.63296    0.64612    0.63557    0.55402    0.54539    0.58355    0.57105
            4   -0.02523   -0.00766   -0.01000    0.63590    0.62187    0.58437    0.61447    0.67262    0.60813    0.61513    0.59062    0.63603    0.62171    0.63421    0.64432    0.57039    0.56875    0.59125    0.59188
            5   -0.02253    0.00058   -0.01341   -0.00954    0.62692    0.58188    0.60789    0.66190    0.60688    0.60329    0.59000    0.65368    0.63092    0.64276    0.63485    0.55329    0.54563    0.58188    0.57000
            6   -0.02770   -0.00103   -0.00843   -0.01114   -0.00915    0.55513    0.57500    0.63810    0.57000    0.57237    0.55063    0.60882    0.58947    0.60461    0.60758    0.51974    0.51438    0.54812    0.54125
            7   -0.02470   -0.00438   -0.01572   -0.01215   -0.01424   -0.01124    0.61735    0.65664    0.60000    0.59765    0.58289    0.64474    0.62258    0.63504    0.62959    0.54778    0.54079    0.57566    0.56645
            8    0.01375    0.04609    0.01208    0.02018    0.01395    0.02604    0.01347    0.66899    0.65952    0.63283    0.65714    0.70308    0.68296    0.69298    0.66234    0.61278    0.59286    0.64643    0.60952
            9   -0.02605   -0.00420   -0.01035   -0.01431   -0.01107   -0.01205   -0.01316    0.02054    0.60897    0.60000    0.57563    0.62794    0.61053    0.62171    0.63030    0.55263    0.55063    0.57437    0.57500
           10   -0.01733    0.01408   -0.01063   -0.00225   -0.00960   -0.00463   -0.01046   -0.00109   -0.00392    0.59886    0.58750    0.65325    0.62812    0.64197    0.61882    0.53878    0.52303    0.57566    0.54803
           11   -0.02518   -0.00494   -0.00629   -0.01130   -0.00744   -0.01091   -0.00976    0.03867   -0.01284    0.00409    0.56795    0.61103    0.59276    0.60461    0.61894    0.53224    0.53312    0.55312    0.56000
           12   -0.00492    0.00612    0.02041   -0.00277    0.01936    0.01040    0.01520    0.04773    0.00260    0.03296    0.00620    0.64171    0.62307    0.63390    0.67513    0.62229    0.63162    0.62721    0.64853
           13   -0.01865   -0.00425    0.00210   -0.01203    0.00167   -0.00388   -0.00189    0.03267   -0.00975    0.01290   -0.00700   -0.01358    0.63158    0.62604    0.65351    0.58934    0.59408    0.60132    0.61447
           14   -0.01286    0.00163    0.00601   -0.00877    0.00427    0.00201    0.00133    0.03345   -0.00781    0.01750   -0.00440   -0.01199   -0.01479    0.65007    0.66427    0.60596    0.61316    0.61316    0.63224
           15   -0.01235    0.01133   -0.00888   -0.00300   -0.00798    0.00064   -0.00846   -0.00153   -0.00355   -0.00998    0.00559    0.02491    0.00835    0.00987    0.65874    0.58054    0.56894    0.60985    0.59053
           16   -0.01306    0.01835   -0.00359    0.00991   -0.00270   -0.00036   -0.00343    0.03576    0.00561   -0.00318    0.00573    0.05890    0.03101    0.03839    0.00864    0.48506    0.45724    0.51513    0.49013
           17    0.00243    0.04035    0.00788    0.02837    0.00973    0.01438    0.00968    0.03593    0.02370    0.00116    0.02671    0.08833    0.05585    0.06569    0.01713   -0.00773    0.44487    0.51000    0.46750
           18   -0.02184    0.00082   -0.01037   -0.00554   -0.01043   -0.00829   -0.01187    0.03309   -0.00896   -0.00262   -0.00970    0.02750    0.00668    0.00928    0.00163   -0.00625    0.00872    0.55769    0.53938
           19   -0.00348    0.03266    0.00085    0.01880    0.00141    0.00856    0.00264    0.01990    0.01538   -0.00653    0.02090    0.07255    0.04356    0.05207    0.00603   -0.00753   -0.01006    0.00540    0.51026


-----------
PXY P value
-----------

                         1           2           3           4           5           6           7           8           9          10          11          12          13          14          15          16          17          18

             2     0.78104
             3     0.84164     0.56891
             4     0.88465     0.71848     0.77322
             5     0.75953     0.44966     0.98827     0.85337
             6     0.63832     0.44575     0.64907     0.87879     0.81623
             7     0.73607     0.57967     1.00000     0.93646     1.00000     0.79374
             8     0.63441     0.03226     0.21896     0.12219     0.22483     0.10948     0.22581
             9     0.84262     0.51320     0.71065     1.00000     0.90420     0.96481     0.93548     0.11144
            10     0.72727     0.14467     0.81036     0.48778     0.85044     0.51124     0.80059     0.58260     0.46041
            11     0.72141     0.49169     0.47019     0.87683     0.74487     0.85044     0.68524     0.02835     0.93255     0.18475
            12     0.72043     0.30303     0.09286     0.50440     0.15347     0.19648     0.16618     0.07234     0.27370     0.06549     0.22874
            13     0.91789     0.53177     0.35191     0.86413     0.42033     0.44770     0.43109     0.08113     0.71750     0.17204     0.50440     0.86315
            14     0.90714     0.35582     0.25415     0.70968     0.32649     0.34311     0.35875     0.09189     0.57674     0.12903     0.43988     0.86804     0.97947
            15     0.81916     0.35484     0.76637     0.63539     0.76149     0.95112     0.81232     0.39492     0.78592     0.93842     0.81329     0.20626     0.39980     0.28739
            16     0.30792     0.16520     0.46921     0.28837     0.66960     0.49658     0.49169     0.08309     0.30010     0.43597     0.25709     0.01466     0.09091     0.06354     0.99609
            17     0.18280     0.01760     0.08700     0.06158     0.14956     0.14565     0.10068     0.03324     0.03617     0.18866     0.03226     0.00293     0.00978     0.00880     0.99902     0.62659
            18     0.54545     0.34409     0.66667     0.52688     0.91789     0.67351     0.84751     0.05963     0.68328     0.37341     0.79374     0.06745     0.21310     0.19844     0.90323     0.71750     0.15543
            19     0.35484     0.02835     0.19648     0.10753     0.33040     0.20528     0.23460     0.14565     0.08407     0.45650     0.04888     0.01173     0.03519     0.01760     0.98240     0.71163     0.91691     0.19062


---------------------
Corrected PXY P value
---------------------

                           1            2            3            4            5            6            7            8            9           10           11           12           13           14           15           16           17           18

              2      0.72141
              3      0.81916      0.54252
              4      0.93939      0.68035      0.74194
              5      0.92962      0.43500      0.97752      0.82209
              6      1.00000      0.44575      0.67546      0.87879      0.81623
              7      0.95112      0.51711      1.00000      0.88759      1.00000      0.85044
              8      0.29228      0.03128      0.17400      0.10655      0.20430      0.09384      0.17693
              9      0.95112      0.49365      0.68622      0.95308      0.86217      0.93451      0.93451      0.08895
             10      0.57771      0.12414      0.73803      0.43011      0.77028      0.47898      0.80059      0.44086      0.37830
             11      0.95699      0.49169      0.52395      0.84848      0.71261      0.85044      0.71652      0.02151      0.93255      0.18573
             12      0.51808      0.28837      0.08602      0.48974      0.14565      0.23460      0.17400      0.05279      0.29032      0.06647      0.24340
             13      0.74682      0.51613      0.33040      0.82796      0.39687      0.51320      0.43011      0.07527      0.68231      0.16618      0.54545      0.89541
             14      0.60411      0.35386      0.23949      0.67644      0.32356      0.36266      0.35875      0.08211      0.60899      0.12610      0.48094      0.81232      0.95503
             15      0.65982      0.15934      0.79374      0.53470      0.81036      0.37439      0.79472      0.48289      0.50733      0.85533      0.24047      0.08504      0.23558      0.17889
             16      0.54057      0.12708      0.43597      0.20821      0.47214      0.41349      0.48192      0.05376      0.20626      0.43597      0.22874      0.02248      0.08993      0.06354      0.18377
             17      0.23460      0.01760      0.12903      0.04594      0.13783      0.14565      0.13294      0.02346      0.03324      0.23558      0.02444      0.00391      0.00978      0.01075      0.08309      0.61877
             18      0.83773      0.33920      0.76931      0.51222      0.88368      0.67351      0.89541      0.05181      0.66373      0.37146      0.79374      0.08407      0.23460      0.21896      0.32747      0.54350      0.13099
             19      0.42815      0.02737      0.24438      0.09580      0.31867      0.20528      0.28543      0.10264      0.07625      0.52688      0.04790      0.01173      0.03715      0.02346      0.20919      0.64418      0.77810      0.18573


////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 1 (08/02/09 at 23:54:35))
Total computing time for this run : 0h 0m 1s 328 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

